{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "provenance": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/mriared/Entropy-of-abstracts-and-the-citations-relationship/blob/main/Datasets_prepocessing_IT.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "TmPNso_gpUhG",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2620cf37-534a-4380-cdce-3905f7e679d1"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
            "[nltk_data]   Package punkt is already up-to-date!\n"
          ]
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "True"
            ]
          },
          "metadata": {},
          "execution_count": 107
        }
      ],
      "source": [
        "import pandas as pd\n",
        "import numpy as np\n",
        "import nltk\n",
        "nltk.download('punkt')"
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##Loading data sets with Nature abstracts from years 2016-2019"
      ],
      "metadata": {
        "id": "_Iuxff3Y0eWl"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "#df_20 = pd.read_csv('/content/Abstracts-Nature-2020.csv')\n",
        "df_19 = pd.read_csv('/content/Abstract-Nature-all-2019.csv')\n",
        "df_18 = pd.read_csv('/content/Abstracts-nature-2018.csv')\n",
        "df_17 = pd.read_csv('/content/Abstracts-Nature-2017.csv')\n",
        "df_16 = pd.read_csv('/content/Abstracts-Nature-2016.csv')"
      ],
      "metadata": {
        "id": "E6P6WVNZm3KI"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "print(f\"Columns 2016: {df_16.columns}, 2017: {df_17.columns}, 2018: {df_18.columns}, 2019: {df_19.columns}\") #, 2020: {df_20.columns}\")"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "WX4splcK3iFj",
        "outputId": "0b436a3c-c210-442e-ee49-390126cce1e3"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Columns 2016: Index(['web-scraper-order', 'web-scraper-start-url', 'URL', 'URL-href',\n",
            "       'abstract', 'Citations'],\n",
            "      dtype='object'), 2017: Index(['web-scraper-order', 'web-scraper-start-url', 'URL', 'URL-href',\n",
            "       'abstract', 'citations'],\n",
            "      dtype='object'), 2018: Index(['web-scraper-order', 'web-scraper-start-url', 'URL', 'URL-href',\n",
            "       'abstract', 'citations'],\n",
            "      dtype='object'), 2019: Index(['web-scraper-order', 'web-scraper-start-url', 'URL', 'URL-href',\n",
            "       'abstracts', 'citations'],\n",
            "      dtype='object')\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "# Prepering for concatenation - renaming columns\n",
        "df_19 = df_19.rename(columns={'abstracts': 'abstract'})\n",
        "df_16 = df_16.rename(columns={'Citations': 'citations'})\n",
        "#df_20 = df_20.rename(columns={'title': 'URL'})"
      ],
      "metadata": {
        "id": "7PnsgVR8oI52"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#Getting subsets for concatination\n",
        "#df_20_subset = df_20[['URL', 'abstract', \"citations\"]]\n",
        "df_19_subset = df_19[['URL', 'abstract', \"citations\"]]\n",
        "df_18_subset = df_18[['URL', 'abstract', \"citations\"]]\n",
        "df_17_subset = df_17[['URL', 'abstract', \"citations\"]]\n",
        "df_16_subset = df_16[['URL', 'abstract', \"citations\"]]"
      ],
      "metadata": {
        "id": "zmy_T7FNpl_T"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#Concatinating all Nature abstracts\n",
        "nature_df_16to19 = pd.concat([df_19_subset, df_18_subset, df_17_subset, df_16_subset], ignore_index=True)"
      ],
      "metadata": {
        "id": "UHqjjlsM2Qkm"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "7q2qK9cR5SXu",
        "outputId": "fa52bfbf-73e5-4c11-da61-a97311baac5d"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                                             URL  \\\n",
              "0            Brain regulatory T cells suppress astrogliosis and potentiate neurological recovery   \n",
              "1                                             Genomic encoding of transcriptional burst kinetics   \n",
              "2                        Cryo-EM structure of the human α1β3γ2 GABAA receptor in a lipid bilayer   \n",
              "3                       A wireless closed-loop system for optogenetic peripheral neuromodulation   \n",
              "4                                               An ultrafast symmetry switch in a Weyl semimetal   \n",
              "...                                                                                          ...   \n",
              "3514    Genetic variants regulating expression levels and isoform diversity during embryogenesis   \n",
              "3515  Variable chromatin structure revealed by in situ spatially correlated DNA cleavage mapping   \n",
              "3516      mTORC1 and muscle regeneration are regulated by the LINC00961-encoded SPAR polypeptide   \n",
              "3517                                     The role of fatty acid β-oxidation in lymphangiogenesis   \n",
              "3518                                 Genome sequence and genetic diversity of European ash trees   \n",
              "\n",
              "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             abstract  \\\n",
              "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              In addition to maintaining immune tolerance, FOXP3+ regulatory T (Treg) cells perform specialized functions in tissue homeostasis and remodelling1,2. However, the characteristics and functions of brain Treg cells are not well understood because there is a low number of Treg cells in the brain under normal conditions. Here we show that there is massive accumulation of Treg cells in the mouse brain after ischaemic stroke, and this potentiates neurological recovery during the chronic phase of ischaemic brain injury. Although brain Treg cells are similar to Treg cells in other tissues such as visceral adipose tissue and muscle3,4,5, they are apparently distinct and express unique genes related to the nervous system including Htr7, which encodes the serotonin receptor 5-HT7. The amplification of brain Treg cells is dependent on interleukin (IL)-2, IL-33, serotonin and T cell receptor recognition, and infiltration into the brain is driven by the chemokines CCL1 and CCL20. Brain Treg cells suppress neurotoxic astrogliosis by producing amphiregulin, a low-affinity epidermal growth factor receptor (EGFR) ligand. Stroke is a leading cause of neurological disability, and there are currently few effective recovery methods other than rehabilitation during the chronic phase. Our findings suggest that Treg cells and their products may provide therapeutic opportunities for neuronal protection against stroke and neuroinflammatory diseases.   \n",
              "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   Mammalian gene expression is inherently stochastic1,2, and results in discrete bursts of RNA molecules that are synthesized from each allele3,4,5,6,7. Although transcription is known to be regulated by promoters and enhancers, it is unclear how cis-regulatory sequences encode transcriptional burst kinetics. Characterization of transcriptional bursting, including the burst size and frequency, has mainly relied on live-cell4,6,8 or single-molecule RNA fluorescence in situ hybridization3,5,8,9 recordings of selected loci. Here we determine transcriptome-wide burst frequencies and sizes for endogenous mouse and human genes using allele-sensitive single-cell RNA sequencing. We show that core promoter elements affect burst size and uncover synergistic effects between TATA and initiator elements, which were masked at mean expression levels. Notably, we provide transcriptome-wide evidence that enhancers control burst frequencies, and demonstrate that cell-type-specific gene expression is primarily shaped by changes in burst frequencies. Together, our data show that burst frequency is primarily encoded in enhancers and burst size in core promoters, and that allelic single-cell RNA sequencing is a powerful model for investigating transcriptional kinetics.   \n",
              "2     Type A γ-aminobutyric acid (GABAA) receptors are pentameric ligand-gated ion channels and the main drivers of fast inhibitory neurotransmission in the vertebrate nervous system1,2. Their dysfunction is implicated in a range of neurological disorders, including depression, epilepsy and schizophrenia3,4. Among the numerous assemblies that are theoretically possible, the most prevalent in the brain are the α1β2/3γ2 GABAA receptors5. The β3 subunit has an important role in maintaining inhibitory tone, and the expression of this subunit alone is sufficient to rescue inhibitory synaptic transmission in β1–β3 triple knockout neurons6. So far, efforts to generate accurate structural models for heteromeric GABAA receptors have been hampered by the use of engineered receptors and the presence of detergents7,8,9. Notably, some recent cryo-electron microscopy reconstructions have reported ‘collapsed’ conformations8,9; however, these disagree with the structure of the prototypical pentameric ligand-gated ion channel the Torpedo nicotinic acetylcholine receptor10,11, the large body of structural work on homologous homopentameric receptor variants12 and the logic of an ion-channel architecture. Here we present a high-resolution cryo-electron microscopy structure of the full-length human α1β3γ2L—a major synaptic GABAA receptor isoform—that is functionally reconstituted in lipid nanodiscs. The receptor is bound to a positive allosteric modulator ‘megabody’ and is in a desensitized conformation. Each GABAA receptor pentamer contains two phosphatidylinositol-4,5-bisphosphate molecules, the head groups of which occupy positively charged pockets in the intracellular juxtamembrane regions of α1 subunits. Beyond this level, the intracellular M3–M4 loops are largely disordered, possibly because interacting post-synaptic proteins are not present. This structure illustrates the molecular principles of heteromeric GABAA receptor organization and provides a reference framework for future mechanistic investigations of GABAergic signalling and pharmacology.   \n",
              "3                                                                                                                                                                                                                                                                                        The fast-growing field of bioelectronic medicine aims to develop engineered systems that can relieve clinical conditions by stimulating the peripheral nervous system1,2,3,4,5. This type of technology relies largely on electrical stimulation to provide neuromodulation of organ function or pain. One example is sacral nerve stimulation to treat overactive bladder, urinary incontinence and interstitial cystitis (also known as bladder pain syndrome)4,6,7. Conventional, continuous stimulation protocols, however, can cause discomfort and pain, particularly when treating symptoms that can be intermittent (for example, sudden urinary urgency)8. Direct physical coupling of electrodes to the nerve can lead to injury and inflammation9,10,11. Furthermore, typical therapeutic stimulators target large nerve bundles that innervate multiple structures, resulting in a lack of organ specificity. Here we introduce a miniaturized bio-optoelectronic implant that avoids these limitations by using (1) an optical stimulation interface that exploits microscale inorganic light-emitting diodes to activate opsins; (2) a soft, high-precision biophysical sensor system that allows continuous measurements of organ function; and (3) a control module and data analytics approach that enables coordinated, closed-loop operation of the system to eliminate pathological behaviours as they occur in real-time. In the example reported here, a soft strain gauge yields real-time information on bladder function in a rat model. Data algorithms identify pathological behaviour, and automated, closed-loop optogenetic neuromodulation of bladder sensory afferents normalizes bladder function. This all-optical scheme for neuromodulation offers chronic stability and the potential to stimulate specific cell types.   \n",
              "4                                                                                                                                                                                                                                                                              Topological quantum materials exhibit fascinating properties1,2,3, with important applications for dissipationless electronics and fault-tolerant quantum computers4,5. Manipulating the topological invariants in these materials would allow the development of topological switching applications analogous to switching of transistors6. Lattice strain provides the most natural means of tuning these topological invariants because it directly modifies the electron–ion interactions and potentially alters the underlying crystalline symmetry on which the topological properties depend7,8,9. However, conventional means of applying strain through heteroepitaxial lattice mismatch10 and dislocations11 are not extendable to controllable time-varying protocols, which are required in transistors. Integration into a functional device requires the ability to go beyond the robust, topologically protected properties of materials and to manipulate the topology at high speeds. Here we use crystallographic measurements by relativistic electron diffraction to demonstrate that terahertz light pulses can be used to induce terahertz-frequency interlayer shear strain with large strain amplitude in the Weyl semimetal WTe2, leading to a topologically distinct metastable phase. Separate nonlinear optical measurements indicate that this transition is associated with a symmetry change to a centrosymmetric, topologically trivial phase. We further show that such shear strain provides an ultrafast, energy-efficient way of inducing robust, well separated Weyl points or of annihilating all Weyl points of opposite chirality. This work demonstrates possibilities for ultrafast manipulation of the topological properties of solids and for the development of a topological switch operating at terahertz frequencies.   \n",
              "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ...   \n",
              "3514                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Embryonic development is driven by tightly regulated patterns of gene expression, despite extensive genetic variation among individuals. Studies of expression quantitative trait loci1,2,3,4 (eQTL) indicate that genetic variation frequently alters gene expression in cell-culture models and differentiated tissues5,6. However, the extent and types of genetic variation impacting embryonic gene expression, and their interactions with developmental programs, remain largely unknown. Here we assessed the effect of genetic variation on transcriptional (expression levels) and post-transcriptional (3′ RNA processing) regulation across multiple stages of metazoan development, using 80 inbred Drosophila wild isolates7, identifying thousands of developmental-stage-specific and shared QTL. Given the small blocks of linkage disequilibrium in Drosophila7,8,9, we obtain near base-pair resolution, resolving causal mutations in developmental enhancers, validated transcription-factor-binding sites and RNA motifs. This fine-grain mapping uncovered extensive allelic interactions within enhancers that have opposite effects, thereby buffering their impact on enhancer activity. QTL affecting 3′ RNA processing identify new functional motifs leading to transcript isoform diversity and changes in the lengths of 3′ untranslated regions. These results highlight how developmental stage influences the effects of genetic variation and uncover multiple mechanisms that regulate and buffer expression variation during embryogenesis.   \n",
              "3515                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Chromatin structure at the length scale encompassing local nucleosome–nucleosome interactions is thought to play a crucial role in regulating transcription and access to DNA1,2,3. However, this secondary structure of chromatin remains poorly understood compared with the primary structure of single nucleosomes or the tertiary structure of long-range looping interactions4. Here we report the first genome-wide map of chromatin conformation in human cells at the 1–3 nucleosome (50–500 bp) scale, obtained using ionizing radiation-induced spatially correlated cleavage of DNA with sequencing (RICC-seq) to identify DNA–DNA contacts that are spatially proximal. Unbiased analysis of RICC-seq signal reveals regional enrichment of DNA fragments characteristic of alternating rather than adjacent nucleosome interactions in tri-nucleosome units, particularly in H3K9me3-marked heterochromatin. We infer differences in the likelihood of nucleosome–nucleosome contacts among open chromatin, H3K27me3-marked, and H3K9me3-marked repressed chromatin regions. After calibrating RICC-seq signal to three-dimensional distances, we show that compact two-start helical fibre structures with stacked alternating nucleosomes are consistent with RICC-seq fragmentation patterns from H3K9me3-marked chromatin, while non-compact structures and solenoid structures are consistent with open chromatin. Our data support a model of chromatin architecture in intact interphase nuclei consistent with variable longitudinal compaction of two-start helical fibres.   \n",
              "3516                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 Although long non-coding RNAs (lncRNAs) are non-protein-coding transcripts by definition, recent studies have shown that a fraction of putative small open reading frames within lncRNAs are translated1,2,3. However, the biological significance of these hidden polypeptides is still unclear. Here we identify and functionally characterize a novel polypeptide encoded by the lncRNA LINC00961. This polypeptide is conserved between human and mouse, is localized to the late endosome/lysosome and interacts with the lysosomal v-ATPase to negatively regulate mTORC1 activation. This regulation of mTORC1 is specific to activation of mTORC1 by amino acid stimulation, rather than by growth factors. Hence, we termed this polypeptide ‘small regulatory polypeptide of amino acid response’ (SPAR). We show that the SPAR-encoding lncRNA is highly expressed in a subset of tissues and use CRISPR/Cas9 engineering to develop a SPAR-polypeptide-specific knockout mouse while maintaining expression of the host lncRNA. We find that the SPAR-encoding lncRNA is downregulated in skeletal muscle upon acute injury, and using this in vivo model we establish that SPAR downregulation enables efficient activation of mTORC1 and promotes muscle regeneration. Our data provide a mechanism by which mTORC1 activation may be finely regulated in a tissue-specific manner in response to injury, and a paradigm by which lncRNAs encoding small polypeptides can modulate general biological pathways and processes to facilitate tissue-specific requirements, consistent with their restricted and highly regulated expression profile.   \n",
              "3517                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              NaN   \n",
              "3518                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           Ash trees (genus Fraxinus, family Oleaceae) are widespread throughout the Northern Hemisphere, but are being devastated in Europe by the fungus Hymenoscyphus fraxineus, causing ash dieback, and in North America by the herbivorous beetle Agrilus planipennis1,2. Here we sequence the genome of a low-heterozygosity Fraxinus excelsior tree from Gloucestershire, UK, annotating 38,852 protein-coding genes of which 25% appear ash specific when compared with the genomes of ten other plant species. Analyses of paralogous genes suggest a whole-genome duplication shared with olive (Olea europaea, Oleaceae). We also re-sequence 37 F. excelsior trees from Europe, finding evidence for apparent long-term decline in effective population size. Using our reference sequence, we re-analyse association transcriptomic data3, yielding improved markers for reduced susceptibility to ash dieback. Surveys of these markers in British populations suggest that reduced susceptibility to ash dieback may be more widespread in Great Britain than in Denmark. We also present evidence that susceptibility of trees to H. fraxineus is associated with their iridoid glycoside levels. This rapid, integrated, multidisciplinary research response to an emerging health threat in a non-model organism opens the way for mitigation of the epidemic.   \n",
              "\n",
              "          citations  \n",
              "0     406 Citations  \n",
              "1     253 Citations  \n",
              "2     205 Citations  \n",
              "3     330 Citations  \n",
              "4     297 Citations  \n",
              "...             ...  \n",
              "3514   40 Citations  \n",
              "3515  110 Citations  \n",
              "3516  422 Citations  \n",
              "3517  216 Citations  \n",
              "3518  130 Citations  \n",
              "\n",
              "[3519 rows x 3 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-97735447-34ed-4020-bc40-8ed8528727d9\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>URL</th>\n",
              "      <th>abstract</th>\n",
              "      <th>citations</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Brain regulatory T cells suppress astrogliosis and potentiate neurological recovery</td>\n",
              "      <td>In addition to maintaining immune tolerance, FOXP3+ regulatory T (Treg) cells perform specialized functions in tissue homeostasis and remodelling1,2. However, the characteristics and functions of brain Treg cells are not well understood because there is a low number of Treg cells in the brain under normal conditions. Here we show that there is massive accumulation of Treg cells in the mouse brain after ischaemic stroke, and this potentiates neurological recovery during the chronic phase of ischaemic brain injury. Although brain Treg cells are similar to Treg cells in other tissues such as visceral adipose tissue and muscle3,4,5, they are apparently distinct and express unique genes related to the nervous system including Htr7, which encodes the serotonin receptor 5-HT7. The amplification of brain Treg cells is dependent on interleukin (IL)-2, IL-33, serotonin and T cell receptor recognition, and infiltration into the brain is driven by the chemokines CCL1 and CCL20. Brain Treg cells suppress neurotoxic astrogliosis by producing amphiregulin, a low-affinity epidermal growth factor receptor (EGFR) ligand. Stroke is a leading cause of neurological disability, and there are currently few effective recovery methods other than rehabilitation during the chronic phase. Our findings suggest that Treg cells and their products may provide therapeutic opportunities for neuronal protection against stroke and neuroinflammatory diseases.</td>\n",
              "      <td>406 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Genomic encoding of transcriptional burst kinetics</td>\n",
              "      <td>Mammalian gene expression is inherently stochastic1,2, and results in discrete bursts of RNA molecules that are synthesized from each allele3,4,5,6,7. Although transcription is known to be regulated by promoters and enhancers, it is unclear how cis-regulatory sequences encode transcriptional burst kinetics. Characterization of transcriptional bursting, including the burst size and frequency, has mainly relied on live-cell4,6,8 or single-molecule RNA fluorescence in situ hybridization3,5,8,9 recordings of selected loci. Here we determine transcriptome-wide burst frequencies and sizes for endogenous mouse and human genes using allele-sensitive single-cell RNA sequencing. We show that core promoter elements affect burst size and uncover synergistic effects between TATA and initiator elements, which were masked at mean expression levels. Notably, we provide transcriptome-wide evidence that enhancers control burst frequencies, and demonstrate that cell-type-specific gene expression is primarily shaped by changes in burst frequencies. Together, our data show that burst frequency is primarily encoded in enhancers and burst size in core promoters, and that allelic single-cell RNA sequencing is a powerful model for investigating transcriptional kinetics.</td>\n",
              "      <td>253 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Cryo-EM structure of the human α1β3γ2 GABAA receptor in a lipid bilayer</td>\n",
              "      <td>Type A γ-aminobutyric acid (GABAA) receptors are pentameric ligand-gated ion channels and the main drivers of fast inhibitory neurotransmission in the vertebrate nervous system1,2. Their dysfunction is implicated in a range of neurological disorders, including depression, epilepsy and schizophrenia3,4. Among the numerous assemblies that are theoretically possible, the most prevalent in the brain are the α1β2/3γ2 GABAA receptors5. The β3 subunit has an important role in maintaining inhibitory tone, and the expression of this subunit alone is sufficient to rescue inhibitory synaptic transmission in β1–β3 triple knockout neurons6. So far, efforts to generate accurate structural models for heteromeric GABAA receptors have been hampered by the use of engineered receptors and the presence of detergents7,8,9. Notably, some recent cryo-electron microscopy reconstructions have reported ‘collapsed’ conformations8,9; however, these disagree with the structure of the prototypical pentameric ligand-gated ion channel the Torpedo nicotinic acetylcholine receptor10,11, the large body of structural work on homologous homopentameric receptor variants12 and the logic of an ion-channel architecture. Here we present a high-resolution cryo-electron microscopy structure of the full-length human α1β3γ2L—a major synaptic GABAA receptor isoform—that is functionally reconstituted in lipid nanodiscs. The receptor is bound to a positive allosteric modulator ‘megabody’ and is in a desensitized conformation. Each GABAA receptor pentamer contains two phosphatidylinositol-4,5-bisphosphate molecules, the head groups of which occupy positively charged pockets in the intracellular juxtamembrane regions of α1 subunits. Beyond this level, the intracellular M3–M4 loops are largely disordered, possibly because interacting post-synaptic proteins are not present. This structure illustrates the molecular principles of heteromeric GABAA receptor organization and provides a reference framework for future mechanistic investigations of GABAergic signalling and pharmacology.</td>\n",
              "      <td>205 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>A wireless closed-loop system for optogenetic peripheral neuromodulation</td>\n",
              "      <td>The fast-growing field of bioelectronic medicine aims to develop engineered systems that can relieve clinical conditions by stimulating the peripheral nervous system1,2,3,4,5. This type of technology relies largely on electrical stimulation to provide neuromodulation of organ function or pain. One example is sacral nerve stimulation to treat overactive bladder, urinary incontinence and interstitial cystitis (also known as bladder pain syndrome)4,6,7. Conventional, continuous stimulation protocols, however, can cause discomfort and pain, particularly when treating symptoms that can be intermittent (for example, sudden urinary urgency)8. Direct physical coupling of electrodes to the nerve can lead to injury and inflammation9,10,11. Furthermore, typical therapeutic stimulators target large nerve bundles that innervate multiple structures, resulting in a lack of organ specificity. Here we introduce a miniaturized bio-optoelectronic implant that avoids these limitations by using (1) an optical stimulation interface that exploits microscale inorganic light-emitting diodes to activate opsins; (2) a soft, high-precision biophysical sensor system that allows continuous measurements of organ function; and (3) a control module and data analytics approach that enables coordinated, closed-loop operation of the system to eliminate pathological behaviours as they occur in real-time. In the example reported here, a soft strain gauge yields real-time information on bladder function in a rat model. Data algorithms identify pathological behaviour, and automated, closed-loop optogenetic neuromodulation of bladder sensory afferents normalizes bladder function. This all-optical scheme for neuromodulation offers chronic stability and the potential to stimulate specific cell types.</td>\n",
              "      <td>330 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>An ultrafast symmetry switch in a Weyl semimetal</td>\n",
              "      <td>Topological quantum materials exhibit fascinating properties1,2,3, with important applications for dissipationless electronics and fault-tolerant quantum computers4,5. Manipulating the topological invariants in these materials would allow the development of topological switching applications analogous to switching of transistors6. Lattice strain provides the most natural means of tuning these topological invariants because it directly modifies the electron–ion interactions and potentially alters the underlying crystalline symmetry on which the topological properties depend7,8,9. However, conventional means of applying strain through heteroepitaxial lattice mismatch10 and dislocations11 are not extendable to controllable time-varying protocols, which are required in transistors. Integration into a functional device requires the ability to go beyond the robust, topologically protected properties of materials and to manipulate the topology at high speeds. Here we use crystallographic measurements by relativistic electron diffraction to demonstrate that terahertz light pulses can be used to induce terahertz-frequency interlayer shear strain with large strain amplitude in the Weyl semimetal WTe2, leading to a topologically distinct metastable phase. Separate nonlinear optical measurements indicate that this transition is associated with a symmetry change to a centrosymmetric, topologically trivial phase. We further show that such shear strain provides an ultrafast, energy-efficient way of inducing robust, well separated Weyl points or of annihilating all Weyl points of opposite chirality. This work demonstrates possibilities for ultrafast manipulation of the topological properties of solids and for the development of a topological switch operating at terahertz frequencies.</td>\n",
              "      <td>297 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3514</th>\n",
              "      <td>Genetic variants regulating expression levels and isoform diversity during embryogenesis</td>\n",
              "      <td>Embryonic development is driven by tightly regulated patterns of gene expression, despite extensive genetic variation among individuals. Studies of expression quantitative trait loci1,2,3,4 (eQTL) indicate that genetic variation frequently alters gene expression in cell-culture models and differentiated tissues5,6. However, the extent and types of genetic variation impacting embryonic gene expression, and their interactions with developmental programs, remain largely unknown. Here we assessed the effect of genetic variation on transcriptional (expression levels) and post-transcriptional (3′ RNA processing) regulation across multiple stages of metazoan development, using 80 inbred Drosophila wild isolates7, identifying thousands of developmental-stage-specific and shared QTL. Given the small blocks of linkage disequilibrium in Drosophila7,8,9, we obtain near base-pair resolution, resolving causal mutations in developmental enhancers, validated transcription-factor-binding sites and RNA motifs. This fine-grain mapping uncovered extensive allelic interactions within enhancers that have opposite effects, thereby buffering their impact on enhancer activity. QTL affecting 3′ RNA processing identify new functional motifs leading to transcript isoform diversity and changes in the lengths of 3′ untranslated regions. These results highlight how developmental stage influences the effects of genetic variation and uncover multiple mechanisms that regulate and buffer expression variation during embryogenesis.</td>\n",
              "      <td>40 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3515</th>\n",
              "      <td>Variable chromatin structure revealed by in situ spatially correlated DNA cleavage mapping</td>\n",
              "      <td>Chromatin structure at the length scale encompassing local nucleosome–nucleosome interactions is thought to play a crucial role in regulating transcription and access to DNA1,2,3. However, this secondary structure of chromatin remains poorly understood compared with the primary structure of single nucleosomes or the tertiary structure of long-range looping interactions4. Here we report the first genome-wide map of chromatin conformation in human cells at the 1–3 nucleosome (50–500 bp) scale, obtained using ionizing radiation-induced spatially correlated cleavage of DNA with sequencing (RICC-seq) to identify DNA–DNA contacts that are spatially proximal. Unbiased analysis of RICC-seq signal reveals regional enrichment of DNA fragments characteristic of alternating rather than adjacent nucleosome interactions in tri-nucleosome units, particularly in H3K9me3-marked heterochromatin. We infer differences in the likelihood of nucleosome–nucleosome contacts among open chromatin, H3K27me3-marked, and H3K9me3-marked repressed chromatin regions. After calibrating RICC-seq signal to three-dimensional distances, we show that compact two-start helical fibre structures with stacked alternating nucleosomes are consistent with RICC-seq fragmentation patterns from H3K9me3-marked chromatin, while non-compact structures and solenoid structures are consistent with open chromatin. Our data support a model of chromatin architecture in intact interphase nuclei consistent with variable longitudinal compaction of two-start helical fibres.</td>\n",
              "      <td>110 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3516</th>\n",
              "      <td>mTORC1 and muscle regeneration are regulated by the LINC00961-encoded SPAR polypeptide</td>\n",
              "      <td>Although long non-coding RNAs (lncRNAs) are non-protein-coding transcripts by definition, recent studies have shown that a fraction of putative small open reading frames within lncRNAs are translated1,2,3. However, the biological significance of these hidden polypeptides is still unclear. Here we identify and functionally characterize a novel polypeptide encoded by the lncRNA LINC00961. This polypeptide is conserved between human and mouse, is localized to the late endosome/lysosome and interacts with the lysosomal v-ATPase to negatively regulate mTORC1 activation. This regulation of mTORC1 is specific to activation of mTORC1 by amino acid stimulation, rather than by growth factors. Hence, we termed this polypeptide ‘small regulatory polypeptide of amino acid response’ (SPAR). We show that the SPAR-encoding lncRNA is highly expressed in a subset of tissues and use CRISPR/Cas9 engineering to develop a SPAR-polypeptide-specific knockout mouse while maintaining expression of the host lncRNA. We find that the SPAR-encoding lncRNA is downregulated in skeletal muscle upon acute injury, and using this in vivo model we establish that SPAR downregulation enables efficient activation of mTORC1 and promotes muscle regeneration. Our data provide a mechanism by which mTORC1 activation may be finely regulated in a tissue-specific manner in response to injury, and a paradigm by which lncRNAs encoding small polypeptides can modulate general biological pathways and processes to facilitate tissue-specific requirements, consistent with their restricted and highly regulated expression profile.</td>\n",
              "      <td>422 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3517</th>\n",
              "      <td>The role of fatty acid β-oxidation in lymphangiogenesis</td>\n",
              "      <td>NaN</td>\n",
              "      <td>216 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3518</th>\n",
              "      <td>Genome sequence and genetic diversity of European ash trees</td>\n",
              "      <td>Ash trees (genus Fraxinus, family Oleaceae) are widespread throughout the Northern Hemisphere, but are being devastated in Europe by the fungus Hymenoscyphus fraxineus, causing ash dieback, and in North America by the herbivorous beetle Agrilus planipennis1,2. Here we sequence the genome of a low-heterozygosity Fraxinus excelsior tree from Gloucestershire, UK, annotating 38,852 protein-coding genes of which 25% appear ash specific when compared with the genomes of ten other plant species. Analyses of paralogous genes suggest a whole-genome duplication shared with olive (Olea europaea, Oleaceae). We also re-sequence 37 F. excelsior trees from Europe, finding evidence for apparent long-term decline in effective population size. Using our reference sequence, we re-analyse association transcriptomic data3, yielding improved markers for reduced susceptibility to ash dieback. Surveys of these markers in British populations suggest that reduced susceptibility to ash dieback may be more widespread in Great Britain than in Denmark. We also present evidence that susceptibility of trees to H. fraxineus is associated with their iridoid glycoside levels. This rapid, integrated, multidisciplinary research response to an emerging health threat in a non-model organism opens the way for mitigation of the epidemic.</td>\n",
              "      <td>130 Citations</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>3519 rows × 3 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-97735447-34ed-4020-bc40-8ed8528727d9')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-97735447-34ed-4020-bc40-8ed8528727d9 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-97735447-34ed-4020-bc40-8ed8528727d9');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-db40a777-b15a-4860-9edf-a6c78a349a1f\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-db40a777-b15a-4860-9edf-a6c78a349a1f')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-db40a777-b15a-4860-9edf-a6c78a349a1f button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "nature_df_16to19",
              "summary": "{\n  \"name\": \"nature_df_16to19\",\n  \"rows\": 3519,\n  \"fields\": [\n    {\n      \"column\": \"URL\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"The active site of low-temperature methane hydroxylation in iron-containing zeolites\",\n          \"Late steps in bacterial translation initiation visualized using time-resolved cryo-EM\",\n          \"Persistent HIV-1 replication maintains the tissue reservoir during therapy\"\n        ],\n        \"num_unique_values\": 3510,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"Cancer metastasis is the primary cause of morbidity and mortality, and accounts for up to 95% of cancer-related deaths1. Cancer cells often reprogram their metabolism to efficiently support cell proliferation and survival2,3. However, whether and how those metabolic alterations contribute to the migration of tumour cells remain largely unknown. UDP-glucose 6-dehydrogenase (UGDH) is a key enzyme in the uronic acid pathway, and converts UDP-glucose to UDP-glucuronic acid4. Here we show that, after activation of EGFR, UGDH is phosphorylated at tyrosine 473 in human\\u00a0lung cancer cells. Phosphorylated UGDH interacts with Hu antigen R (HuR) and converts UDP-glucose to UDP-glucuronic acid, which attenuates the UDP-glucose-mediated inhibition of the association of HuR with SNAI1 mRNA and therefore enhances the stability of SNAI1 mRNA. Increased production of SNAIL initiates the epithelial\\u2013mesenchymal transition, thus promoting the migration of tumour cells and lung cancer metastasis. In addition, phosphorylation of UGDH at tyrosine 473 correlates with metastatic recurrence and poor prognosis of patients with lung cancer. Our findings reveal a tumour-suppressive role of UDP-glucose in lung cancer metastasis and uncover a mechanism by which UGDH promotes tumour metastasis by increasing the stability of SNAI1 mRNA.\",\n          \"The evolution of novel traits can involve many mutations scattered throughout the genome1,2. Detecting and validating such a suite of alleles, particularly if they arose long ago, remains a key challenge in evolutionary genetics1,2,3. Here we dissect an evolutionary trade-off of unprecedented genetic complexity between long-diverged species. When cultured in 1% glucose medium supplemented with galactose, Saccharomyces cerevisiae, but not S. bayanus or other Saccharomyces species, delayed commitment to galactose metabolism until glucose was exhausted. Promoters of seven galactose (GAL) metabolic genes from S. cerevisiae, when introduced together into S. bayanus, largely recapitulated the delay phenotype in 1% glucose\\u2013galactose medium, and most had partial effects when tested in isolation. Variation in GAL coding regions also contributed to the delay when tested individually in 1% glucose\\u2013galactose medium. When combined, S. cerevisiae GAL coding regions gave rise to profound growth defects in the S. bayanus background. In medium containing 2.5% glucose supplemented with galactose, wild-type S. cerevisiae repressed GAL gene expression and had a robust growth advantage relative to S. bayanus; transgenesis of S. cerevisiae GAL promoter alleles or GAL coding regions was sufficient for partial reconstruction of these phenotypes. S. cerevisiae GAL genes thus encode a regulatory program of slow induction and avid repression, and a fitness detriment during the glucose\\u2013galactose transition but a benefit when glucose is in excess. Together, these results make clear that genetic mapping of complex phenotypes is within reach, even in deeply diverged species.\",\n          \"Controlled transport of water molecules through membranes and capillaries is important in areas as diverse as water purification and healthcare technologies1,2,3,4,5,6,7. Previous attempts to control water permeation through membranes (mainly polymeric ones) have concentrated on modulating the structure of the membrane and the physicochemical properties of its surface by varying the pH, temperature or ionic strength3,8. Electrical control over water transport is an attractive alternative; however, theory and simulations9,10,11,12,13,14 have often yielded conflicting results, from freezing of water molecules to melting of ice14,15,16 under an applied electric field. Here we report electrically controlled water permeation through micrometre-thick graphene oxide membranes17,18,19,20,21. Such membranes have previously been shown to exhibit ultrafast permeation of water17,22 and molecular sieving properties18,21, with the potential for industrial-scale production. To achieve electrical control over water permeation, we create conductive filaments in the graphene oxide membranes via controllable electrical breakdown. The electric field that concentrates around these current-carrying filaments ionizes water molecules inside graphene capillaries within the graphene oxide membranes, which impedes water transport. We thus demonstrate precise control of water permeation, from ultrafast permeation to complete blocking. Our work opens up an avenue for developing smart membrane technologies for artificial biological systems, tissue engineering and filtration.\"\n        ],\n        \"num_unique_values\": 2841,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"citations\",\n      \"properties\": {\n        \"dtype\": \"category\",\n        \"samples\": [\n          \"2378 Citations\",\n          \"1066 Citations\",\n          \"86 Citations\"\n        ],\n        \"num_unique_values\": 798,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 113
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19 = nature_df_16to19.rename(columns={\"URL\":\"title\"})"
      ],
      "metadata": {
        "id": "42C4kMqQ2gm9"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "def dataprep(df):\n",
        "  df = df.rename(columns={\"URL\":\"title\"})\n",
        "  df = df.drop_duplicates('title')\n",
        "  df = df.drop_duplicates('abstract')\n",
        "  df.dropna(subset=['abstract'], inplace=True)\n",
        "\n",
        "  return df"
      ],
      "metadata": {
        "id": "c22aFD55DOpl"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#Inspecting duplicated rows\n",
        "nature_df_16to19[nature_df_16to19.duplicated('title', keep=False)].sort_values('title')"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 488
        },
        "id": "2x0nE7YbB62T",
        "outputId": "15a8b797-ec92-4715-df95-957ec0f51001"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                      title abstract  \\\n",
              "2612  Autophagy maintains stemness by preventing senescence      NaN   \n",
              "3153  Autophagy maintains stemness by preventing senescence      NaN   \n",
              "3115                                  Collinge et al. reply      NaN   \n",
              "3281                                  Collinge et al. reply      NaN   \n",
              "2194                                      Dong et al. reply      NaN   \n",
              "2195                                      Dong et al. reply      NaN   \n",
              "2196                                      Dong et al. reply      NaN   \n",
              "2197                                      Dong et al. reply      NaN   \n",
              "2198                                      Dong et al. reply      NaN   \n",
              "3246                                     Lyons et al. reply      NaN   \n",
              "3380                                     Lyons et al. reply      NaN   \n",
              "3460                                 Safikhani et al. reply      NaN   \n",
              "3462                                 Safikhani et al. reply      NaN   \n",
              "3464                                 Safikhani et al. reply      NaN   \n",
              "\n",
              "          citations  \n",
              "2612  849 Citations  \n",
              "3153    5 Citations  \n",
              "3115    2 Citations  \n",
              "3281    1 Citations  \n",
              "2194    3 Citations  \n",
              "2195    2 Citations  \n",
              "2196    3 Citations  \n",
              "2197    1 Citations  \n",
              "2198    1 Citations  \n",
              "3246   13 Altmetric  \n",
              "3380    1 Citations  \n",
              "3460    6 Citations  \n",
              "3462    6 Citations  \n",
              "3464   10 Citations  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-b5feaf38-b921-4d04-b0fd-a3f9cd2b98f8\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>citations</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>2612</th>\n",
              "      <td>Autophagy maintains stemness by preventing senescence</td>\n",
              "      <td>NaN</td>\n",
              "      <td>849 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3153</th>\n",
              "      <td>Autophagy maintains stemness by preventing senescence</td>\n",
              "      <td>NaN</td>\n",
              "      <td>5 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3115</th>\n",
              "      <td>Collinge et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3281</th>\n",
              "      <td>Collinge et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2194</th>\n",
              "      <td>Dong et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2195</th>\n",
              "      <td>Dong et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>2 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2196</th>\n",
              "      <td>Dong et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>3 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2197</th>\n",
              "      <td>Dong et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2198</th>\n",
              "      <td>Dong et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3246</th>\n",
              "      <td>Lyons et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>13 Altmetric</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3380</th>\n",
              "      <td>Lyons et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>1 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3460</th>\n",
              "      <td>Safikhani et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3462</th>\n",
              "      <td>Safikhani et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>6 Citations</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3464</th>\n",
              "      <td>Safikhani et al. reply</td>\n",
              "      <td>NaN</td>\n",
              "      <td>10 Citations</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-b5feaf38-b921-4d04-b0fd-a3f9cd2b98f8')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-b5feaf38-b921-4d04-b0fd-a3f9cd2b98f8 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-b5feaf38-b921-4d04-b0fd-a3f9cd2b98f8');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-1640ca6f-31c7-46a9-af91-6558dc685c7a\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-1640ca6f-31c7-46a9-af91-6558dc685c7a')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-1640ca6f-31c7-46a9-af91-6558dc685c7a button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "repr_error": "0"
            }
          },
          "metadata": {},
          "execution_count": 116
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19 = dataprep(nature_df_16to19)"
      ],
      "metadata": {
        "id": "L9j4rugzBu2q"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19['abstract_len'] = nature_df_16to19['abstract'].apply(len)\n",
        "nature_df_16to19['abstract_len'].hist(bins=100)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 447
        },
        "id": "sS4tMtFzzYD2",
        "outputId": "2ef99612-c26d-475f-e446-5109036e7392"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<Axes: >"
            ]
          },
          "metadata": {},
          "execution_count": 118
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAh8AAAGdCAYAAACyzRGfAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAh20lEQVR4nO3dfXBU5f3+8WsTliURNjwJSTRAUAQFhAqCEaUqIcGiQmUUhXaQOlg1WDF+UWIFAkoFnCIjomjH4jhjfJoK1idsGgRKCQgU1FQnggWhYoIFkwUiy0ru3x/8suMmC8kuu/dmk/drZkf27L3nfPbjeri8z8M6jDFGAAAAliTEugAAANC6ED4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWNUm1gXUV1tbqwMHDqhDhw5yOByxLgcAADSBMUZHjhxRenq6EhLOPLfR7MLHgQMHlJGREesyAABAGPbv36/zzz//jGOaXfjo0KGDpFPFu93uGFdzZj6fT3/729+Uk5Mjp9MZ63JaNHptF/22h17bRb+jx+PxKCMjw//3+Jk0u/BRd6jF7XbHRfhITk6W2+3mSxxl9Nou+m0PvbaLfkdfU06Z4IRTAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABY1SbWBQCIX71mvRfwfO/CsTGqBEA8YeYDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVrWJdQEAIqvXrPcCnu9dODZGlQBAcMx8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK+3wAaHBvEFei0eJhMSoGQIvHzAcAALCK8AEAAKwifAAAAKs45wOIY/XP1QCAeMDMBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsCil8nDx5UrNnz1ZmZqaSkpJ0wQUX6LHHHpMxxj/GGKM5c+YoLS1NSUlJys7O1q5duyJeOAC7es16r8EDAMIRUvhYtGiRnnvuOT3zzDP64osvtGjRIi1evFjLli3zj1m8eLGefvpprVixQlu2bNE555yj3NxcHT9+POLFAwCA+BPSTcY2bdqkcePGaezYsZKkXr166dVXX9XHH38s6dSsx9KlS/Xoo49q3LhxkqSXX35Z3bt31+rVq3XbbbdFuHwAABBvQgofV155pV544QV9+eWXuuiii/TJJ59o48aNWrJkiSRpz549qqioUHZ2tv89KSkpGj58uEpLS4OGD6/XK6/X63/u8XgkST6fTz6fL6wPZUtdfc29zpaAXgfnSjSNjmlKz+qvx5VgGrw3UttCIL7bdtHv6Amlpw7z0xM2GlFbW6tHHnlEixcvVmJiok6ePKkFCxaooKBA0qmZkREjRujAgQNKS0vzv+/WW2+Vw+HQ66+/3mCdhYWFmjdvXoPlRUVFSk5ObvIHAQAAsVNTU6NJkyapurpabrf7jGNDmvl444039Morr6ioqEj9+/fXzp07NWPGDKWnp2vKlClhFVtQUKD8/Hz/c4/Ho4yMDOXk5DRafKz5fD4VFxdr9OjRcjqdsS6nRaPXwQ0o/LDRMWWFuSGvx5Vg9NjQ2oB+R2pbCMR32y76HT11Ry6aIqTwMXPmTM2aNct/+GTgwIH6+uuv9cQTT2jKlClKTU2VJFVWVgbMfFRWVmrw4MFB1+lyueRyuRosdzqdcfPFiKda4x29DuQ96Wh0TFP6dbr1/LTfkdoWguO7bRf9jrxQ+hnS1S41NTVKSAh8S2JiomprayVJmZmZSk1NVUlJif91j8ejLVu2KCsrK5RNAQCAFiqkmY8bb7xRCxYsUI8ePdS/f3/t2LFDS5Ys0W9+8xtJksPh0IwZM/T444+rT58+yszM1OzZs5Wenq7x48dHo34AYeAeHQBiKaTwsWzZMs2ePVv33nuvDh48qPT0dP32t7/VnDlz/GMeeughHTt2THfddZeqqqp01VVXac2aNWrXrl3EiwcAAPEnpPDRoUMHLV26VEuXLj3tGIfDofnz52v+/PlnWxsAAGiB+G0XAABgVUgzHwBwJsHOJdm7cGzI72vKe6K5HgDRxcwHAACwivABAACsInwAAACrOOcDgFXcYwQAMx8AAMAqwgcAALCK8AEAAKwifAAAAKsIHwAAwCrCBwAAsIpLbQG0WOHe7h1AdDHzAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK8AEAAKwifAAAAKsIHwAAwCp+2wVAVAX7fRUArRszHwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK26sDZ1D/1uB7F46NUSUA0HIw8wEAAKwifAAAAKsIHwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqftsFQFzgd3aAloOZDwAAYBXhAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABYxaW2QBypf7lpa0YvgPjFzAcAALCK8AEAAKwifAAAAKs45wMA6uFW7kB0MfMBAACsInwAAACrCB8AAMAqwgcAALCK8AEAAKwifAAAAKsIHwAAwKqQw8c333yjX/3qV+rSpYuSkpI0cOBAbdu2zf+6MUZz5sxRWlqakpKSlJ2drV27dkW0aACIlF6z3mvwABBdIYWP77//XiNGjJDT6dQHH3ygzz//XH/84x/VqVMn/5jFixfr6aef1ooVK7Rlyxadc845ys3N1fHjxyNePAAAiD8h3eF00aJFysjI0MqVK/3LMjMz/X82xmjp0qV69NFHNW7cOEnSyy+/rO7du2v16tW67bbbIlQ2AACIVyGFj7/+9a/Kzc3VLbfcovXr1+u8887Tvffeq2nTpkmS9uzZo4qKCmVnZ/vfk5KSouHDh6u0tDRo+PB6vfJ6vf7nHo9HkuTz+eTz+cL6ULbU1dfc62wJYtVrV6IJWkes1K+nKYLV3Nh6XAmmwXvD2Xa4wqk53HU3Zb3R/PfOfsQu+h09ofTUYYxp8n/R7dq1kyTl5+frlltu0datW3X//fdrxYoVmjJlijZt2qQRI0bowIEDSktL87/v1ltvlcPh0Ouvv95gnYWFhZo3b16D5UVFRUpOTm7yBwEAALFTU1OjSZMmqbq6Wm63+4xjQwofbdu21dChQ7Vp0yb/st/97nfaunWrSktLwwofwWY+MjIy9L///a/R4mPN5/OpuLhYo0ePltPpjHU5LVqsej2g8MOA52WFuda2HUz9epoiWM2NrceVYPTY0FrN3pYgb60j5G2erXBqDnfdTVlvNP+9sx+xi35Hj8fjUdeuXZsUPkI67JKWlqZLLrkkYNnFF1+sv/zlL5Kk1NRUSVJlZWVA+KisrNTgwYODrtPlcsnlcjVY7nQ64+aLEU+1xjvbvfaeDPyLN9b/nuvX0xTBam7qery1jrC2ebbOpuZQ192U9dr4985+xC76HXmh9DOkq11GjBih8vLygGVffvmlevbsKenUyaepqakqKSnxv+7xeLRlyxZlZWWFsikAANBChTTz8cADD+jKK6/UH/7wB9166636+OOP9cILL+iFF16QJDkcDs2YMUOPP/64+vTpo8zMTM2ePVvp6ekaP358NOoHwlb/fg57F46NUSXRxX0rADQ3IYWPyy+/XKtWrVJBQYHmz5+vzMxMLV26VJMnT/aPeeihh3Ts2DHdddddqqqq0lVXXaU1a9b4T1YFAACtW0jhQ5JuuOEG3XDDDad93eFwaP78+Zo/f/5ZFQYAAFqmkMMHALQ2wQ5dtdTDdIAN/LAcAACwivABAACsInwAAACrOOcDQLMTj5cHc14I0HTMfAAAAKsIHwAAwCrCBwAAsIpzPoBmorXc7h0AmPkAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVhE+AACAVdxeHQDCUP92+ACajpkPAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFZxnw+0CtyTAQCaD2Y+AACAVYQPAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWMVvuwDNFL9HA6ClYuYDAABYRfgAAABWcdgFQKsSy8NZwba967GcGFQCxBYzHwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACs4j4fQAiC3adh78KxMagEAOIXMx8AAMAqwgcAALCK8AEAAKzinA/EPc7DAID4wswHAACwivABAACsInwAAACrOOcDLVKw80Ca07Y5JwVAa8bMBwAAsIrwAQAArCJ8AAAAqzjnA/j/YnmeCAC0Jsx8AAAAqwgfAADAqrMKHwsXLpTD4dCMGTP8y44fP668vDx16dJF7du314QJE1RZWXm2dQIAgBYi7PCxdetWPf/887r00ksDlj/wwAN655139Oabb2r9+vU6cOCAbr755rMuFAAAtAxhhY+jR49q8uTJ+tOf/qROnTr5l1dXV+vFF1/UkiVLdN1112nIkCFauXKlNm3apM2bN0esaAAAEL/CutolLy9PY8eOVXZ2th5//HH/8u3bt8vn8yk7O9u/rF+/furRo4dKS0t1xRVXNFiX1+uV1+v1P/d4PJIkn88nn88XTnnW1NXX3OtsCc7Ua1eisV1OgPo1NaWe5vg5fsqVYAL+ifA05bvBfsQu+h09ofTUYYwJae/y2muvacGCBdq6davatWuna665RoMHD9bSpUtVVFSkqVOnBoQJSRo2bJiuvfZaLVq0qMH6CgsLNW/evAbLi4qKlJycHEppAAAgRmpqajRp0iRVV1fL7XafcWxIMx/79+/X/fffr+LiYrVr1+6siqxTUFCg/Px8/3OPx6OMjAzl5OQ0Wnys+Xw+FRcXa/To0XI6nbEup0U7U68HFH4Yo6pOKSvMDXjelHrqv6ep77PFlWD02NBazd6WIG+tI9bltGg7fn8d+xGL2G9HT92Ri6YIKXxs375dBw8e1GWXXeZfdvLkSW3YsEHPPPOMPvzwQ504cUJVVVXq2LGjf0xlZaVSU1ODrtPlcsnlcjVY7nQ64+aLEU+1xrtgvfaejO1fjuHUE+z7EuvPEYy31tEs62pJ6r4L7Efsot+RF0o/Qwofo0aN0meffRawbOrUqerXr58efvhhZWRkyOl0qqSkRBMmTJAklZeXa9++fcrKygplUwAAoIUKKXx06NBBAwYMCFh2zjnnqEuXLv7ld955p/Lz89W5c2e53W7dd999ysrKCnqyKdAShHNbdm7lDhvqf8/2Lhwbo0qAQBH/bZennnpKCQkJmjBhgrxer3Jzc/Xss89GejMAACBOnXX4WLduXcDzdu3aafny5Vq+fPnZrhoAALRA/LYLAACwKuKHXQAA4Qt2PhDnaqClYeYDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFa1iXUBAIDQ9Zr1XsDzvQvHxqgSIHTMfAAAAKsIHwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivt8AEAzV/+eHkC8Y+YDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFZxqS0AxNCAwg+1eNipf3pPOsJeD5fjIp4w8wEAAKwifAAAAKsIHwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK8AEAAKwifAAAAKsIHwAAwCrCBwAAsIrwAQAArCJ8AAAAq9rEugAAQOz0mvVewPO9C8fGqBK0Jsx8AAAAqwgfAADAKsIHAACwinM+AAB+9c8BkRqeB9KUMcCZMPMBAACsInwAAACrCB8AAMAqzvkAgFYi2LkaQCww8wEAAKwifAAAAKsIHwAAwKqQwscTTzyhyy+/XB06dFC3bt00fvx4lZeXB4w5fvy48vLy1KVLF7Vv314TJkxQZWVlRItG69Zr1nsBDwBAfAkpfKxfv155eXnavHmziouL5fP5lJOTo2PHjvnHPPDAA3rnnXf05ptvav369Tpw4IBuvvnmiBcOAADiU0hXu6xZsybg+UsvvaRu3bpp+/btGjlypKqrq/Xiiy+qqKhI1113nSRp5cqVuvjii7V582ZdccUVkascAADEpbO61La6ulqS1LlzZ0nS9u3b5fP5lJ2d7R/Tr18/9ejRQ6WlpUHDh9frldfr9T/3eDySJJ/PJ5/PdzblRV1dfc29zpbgp712JZoYV9PyuRJMwD8RPfHQ6/r7uGD/DcbLfpD9dvSE0lOHMSasb3xtba1uuukmVVVVaePGjZKkoqIiTZ06NSBMSNKwYcN07bXXatGiRQ3WU1hYqHnz5jVYXlRUpOTk5HBKAwAAltXU1GjSpEmqrq6W2+0+49iwZz7y8vJUVlbmDx7hKigoUH5+vv+5x+NRRkaGcnJyGi0+1nw+n4qLizV69Gg5nc5Yl9Oi/bTXP1uwNtbltHiuBKPHhtZq9rYEeWsdsS6nRWspvS4rzI11CU3Cfjt66o5cNEVY4WP69Ol69913tWHDBp1//vn+5ampqTpx4oSqqqrUsWNH//LKykqlpqYGXZfL5ZLL5Wqw3Ol0xs0XI55qjXdOp1Pek/G7g4433loH/bYk3nsdb/tA9tuRF0o/Q7raxRij6dOna9WqVVq7dq0yMzMDXh8yZIicTqdKSkr8y8rLy7Vv3z5lZWWFsikAANBChTTzkZeXp6KiIr399tvq0KGDKioqJEkpKSlKSkpSSkqK7rzzTuXn56tz585yu9267777lJWVxZUuCEvdfTxciUaLh0kDCj+UFL//dwgACDF8PPfcc5Kka665JmD5ypUrdccdd0iSnnrqKSUkJGjChAnyer3Kzc3Vs88+G5FiAQBA/AspfDTlwph27dpp+fLlWr58edhFAQCAlovfdgEAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABYFfav2gLRUHc7dQBAy8XMBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK8AEAAKwifAAAAKsIHwAAwCrCBwAAsIrwAQAArGoT6wIAAC1Pr1nvNVi2d+HYGFSC5oiZDwAAYBXhAwAAWEX4AAAAVnHOB2Im2DFhAEDLx8wHAACwivABAACsInwAAACrOOcDAHDWOIcLoWDmAwAAWEX4AAAAVhE+AACAVYQPAABgFeEDAABYRfgAAABWET4AAIBVhA8AAGAV4QMAAFhF+AAAAFZxe3U0qv5tk/cuHBuR9QAAWidmPgAAgFWEDwAAYBXhAwAAWMU5H4gIzucA0JhInT+G+MfMBwAAsIrwAQAArCJ8AAAAqzjnAwAQE8HOFeM8kNaBmQ8AAGAV4QMAAFhF+AAAAFYRPgAAgFWEDwAAYBXhAwAAWEX4AAAAVnGfDwBAs9WUe4Fwv5D4w8wHAACwivABAACsInwAAACrWt05H/WPDXJcMFCwY6fhjAEA4HSY+QAAAFYRPgAAgFVRO+yyfPlyPfnkk6qoqNCgQYO0bNkyDRs2LFqbs67XrPfkSjRaPEwaUPihyhfcELH1hiPcw0ccQgHQnETq0G+kDrE3ZVvhrNvm5cHN8VLkqMx8vP7668rPz9fcuXP1r3/9S4MGDVJubq4OHjwYjc0BAIA4EpXwsWTJEk2bNk1Tp07VJZdcohUrVig5OVl//vOfo7E5AAAQRyJ+2OXEiRPavn27CgoK/MsSEhKUnZ2t0tLSBuO9Xq+8Xq//eXV1tSTp8OHD8vl8kS5PbX48FvD80KFDYa+nTa1RTU2t2vgSwl5PsPWG42w+Rzz4aa9P1jpiXU6LR7/todd21O0jfT6fampqdOjQITmdzkbf15R9ZDj732DrjdTfI7Ha1pEjRyRJxpjGB5sI++abb4wks2nTpoDlM2fONMOGDWswfu7cuUYSDx48ePDgwaMFPPbv399oVoj5fT4KCgqUn5/vf15bW6vDhw+rS5cucjia9/8FeDweZWRkaP/+/XK73bEup0Wj13bRb3votV30O3qMMTpy5IjS09MbHRvx8NG1a1clJiaqsrIyYHllZaVSU1MbjHe5XHK5XAHLOnbsGOmyosrtdvMltoRe20W/7aHXdtHv6EhJSWnSuIifcNq2bVsNGTJEJSUl/mW1tbUqKSlRVlZWpDcHAADiTFQOu+Tn52vKlCkaOnSohg0bpqVLl+rYsWOaOnVqNDYHAADiSFTCx8SJE/Xdd99pzpw5qqio0ODBg7VmzRp17949GpuLGZfLpblz5zY4bITIo9d20W976LVd9Lt5cBjTlGtiAAAAIoPfdgEAAFYRPgAAgFWEDwAAYBXhAwAAWEX4qKewsFAOhyPg0a9fP//rx48fV15enrp06aL27dtrwoQJDW6otm/fPo0dO1bJycnq1q2bZs6cqR9//NH2R2l2NmzYoBtvvFHp6elyOBxavXp1wOvGGM2ZM0dpaWlKSkpSdna2du3aFTDm8OHDmjx5stxutzp27Kg777xTR48eDRjz6aef6uqrr1a7du2UkZGhxYsXR/ujNUuN9fuOO+5o8F0fM2ZMwBj63TRPPPGELr/8cnXo0EHdunXT+PHjVV5eHjAmUvuOdevW6bLLLpPL5dKFF16ol156Kdofr1lpSq+vueaaBt/tu+++O2AMvY6xiPygSwsyd+5c079/f/Ptt9/6H999953/9bvvvttkZGSYkpISs23bNnPFFVeYK6+80v/6jz/+aAYMGGCys7PNjh07zPvvv2+6du1qCgoKYvFxmpX333/f/P73vzdvvfWWkWRWrVoV8PrChQtNSkqKWb16tfnkk0/MTTfdZDIzM80PP/zgHzNmzBgzaNAgs3nzZvOPf/zDXHjhheb222/3v15dXW26d+9uJk+ebMrKysyrr75qkpKSzPPPP2/rYzYbjfV7ypQpZsyYMQHf9cOHDweMod9Nk5uba1auXGnKysrMzp07zS9+8QvTo0cPc/ToUf+YSOw7/vOf/5jk5GSTn59vPv/8c7Ns2TKTmJho1qxZY/XzxlJTev3zn//cTJs2LeC7XV1d7X+dXsce4aOeuXPnmkGDBgV9raqqyjidTvPmm2/6l33xxRdGkiktLTXGnNrhJyQkmIqKCv+Y5557zrjdbuP1eqNaezyp/5dhbW2tSU1NNU8++aR/WVVVlXG5XObVV181xhjz+eefG0lm69at/jEffPCBcTgc5ptvvjHGGPPss8+aTp06BfT64YcfNn379o3yJ2reThc+xo0bd9r30O/wHTx40Egy69evN8ZEbt/x0EMPmf79+wdsa+LEiSY3NzfaH6nZqt9rY06Fj/vvv/+076HXscdhlyB27dql9PR09e7dW5MnT9a+ffskSdu3b5fP51N2drZ/bL9+/dSjRw+VlpZKkkpLSzVw4MCAG6rl5ubK4/Ho3//+t90PEkf27NmjioqKgN6mpKRo+PDhAb3t2LGjhg4d6h+TnZ2thIQEbdmyxT9m5MiRatu2rX9Mbm6uysvL9f3331v6NPFj3bp16tatm/r27at77rkn4Ge26Xf4qqurJUmdO3eWFLl9R2lpacA66sbUraM1qt/rOq+88oq6du2qAQMGqKCgQDU1Nf7X6HXsxfxXbZub4cOH66WXXlLfvn317bffat68ebr66qtVVlamiooKtW3btsEP33Xv3l0VFRWSpIqKigZ3cq17XjcGDdX1Jljvftrbbt26Bbzepk0bde7cOWBMZmZmg3XUvdapU6eo1B+PxowZo5tvvlmZmZn66quv9Mgjj+j6669XaWmpEhMT6XeYamtrNWPGDI0YMUIDBgyQpIjtO043xuPx6IcfflBSUlI0PlKzFazXkjRp0iT17NlT6enp+vTTT/Xwww+rvLxcb731liR63RwQPuq5/vrr/X++9NJLNXz4cPXs2VNvvPEGXza0KLfddpv/zwMHDtSll16qCy64QOvWrdOoUaNiWFl8y8vLU1lZmTZu3BjrUlq80/X6rrvu8v954MCBSktL06hRo/TVV1/pggsusF0mguCwSyM6duyoiy66SLt371ZqaqpOnDihqqqqgDGVlZVKTU2VJKWmpjY4g73ued0YNFTXm2C9+2lvDx48GPD6jz/+qMOHD9P/COjdu7e6du2q3bt3S6Lf4Zg+fbreffddffTRRzr//PP9yyO17zjdGLfb3er+5+h0vQ5m+PDhkhTw3abXsUX4aMTRo0f11VdfKS0tTUOGDJHT6VRJSYn/9fLycu3bt09ZWVmSpKysLH322WcBO+3i4mK53W5dcskl1uuPF5mZmUpNTQ3orcfj0ZYtWwJ6W1VVpe3bt/vHrF27VrW1tf6dS1ZWljZs2CCfz+cfU1xcrL59+7bKQwCh+O9//6tDhw4pLS1NEv0OhTFG06dP16pVq7R27doGh6Iite/IysoKWEfdmLp1tAaN9TqYnTt3SlLAd5tex1isz3htbh588EGzbt06s2fPHvPPf/7TZGdnm65du5qDBw8aY05dLtejRw+zdu1as23bNpOVlWWysrL876+7hCsnJ8fs3LnTrFmzxpx77rlcamuMOXLkiNmxY4fZsWOHkWSWLFliduzYYb7++mtjzKlLbTt27Gjefvtt8+mnn5px48YFvdT2Zz/7mdmyZYvZuHGj6dOnT8Cln1VVVaZ79+7m17/+tSkrKzOvvfaaSU5ObnWXfhpz5n4fOXLE/N///Z8pLS01e/bsMX//+9/NZZddZvr06WOOHz/uXwf9bpp77rnHpKSkmHXr1gVc3llTU+MfE4l9R93lnzNnzjRffPGFWb58eau7/LOxXu/evdvMnz/fbNu2zezZs8e8/fbbpnfv3mbkyJH+ddDr2CN81DNx4kSTlpZm2rZta8477zwzceJEs3v3bv/rP/zwg7n33ntNp06dTHJysvnlL39pvv3224B17N2711x//fUmKSnJdO3a1Tz44IPG5/PZ/ijNzkcffWQkNXhMmTLFGHPqctvZs2eb7t27G5fLZUaNGmXKy8sD1nHo0CFz++23m/bt2xu3222mTp1qjhw5EjDmk08+MVdddZVxuVzmvPPOMwsXLrT1EZuVM/W7pqbG5OTkmHPPPdc4nU7Ts2dPM23atIBLD42h300VrM+SzMqVK/1jIrXv+Oijj8zgwYNN27ZtTe/evQO20Ro01ut9+/aZkSNHms6dOxuXy2UuvPBCM3PmzID7fBhDr2PNYYwx9uZZAABAa8c5HwAAwCrCBwAAsIrwAQAArCJ8AAAAqwgfAADAKsIHAACwivABAACsInwAAACrCB8AAMAqwgcAALCK8AEAAKwifAAAAKv+H16MuWLfrJZgAAAAAElFTkSuQmCC\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#Inspecting the shortest abstracts\n",
        "nature_df_16to19.sort_values('abstract_len')[:10]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 918
        },
        "id": "nMP2oeYdzpDT",
        "outputId": "5eb6af0e-8aaf-4d68-fb58-602d1d1f6440"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                                   title  \\\n",
              "2948                   Reproducible pharmacogenomic profiling of cancer cell line panels   \n",
              "1007                                                 Adolescence and the next generation   \n",
              "2167       T cells from patients with Parkinson’s disease recognize α-synuclein peptides   \n",
              "1459                                      A Triassic stem turtle with an edentulous beak   \n",
              "383                                          Climate as a risk factor for armed conflict   \n",
              "3253       On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models   \n",
              "1210                                   Emerging trends in global freshwater availability   \n",
              "1384              Cyclin-dependent kinase 12 is a drug target for visceral leishmaniasis   \n",
              "823                  The GenomeAsia 100K Project enables genetic discoveries across Asia   \n",
              "940   The genome of Schmidtea mediterranea and the evolution of core cellular mechanisms   \n",
              "\n",
              "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          abstract  \\\n",
              "2948                                                                                                                                                                                                                                                                                                                                                                                                                                        The use of large-scale genomic and drug response screening of cancer cell lines depends crucially on the reproducibility of results. Here we consider two previously published screens, plus a later critique of these studies. Using independent data, we show that consistency is achievable, and provide a systematic description of the best laboratory and analysis practices for future studies.   \n",
              "1007                                                                                                                                                                                                                                                                                                                                                                                                                          Adolescent growth and social development shape the early development of offspring from preconception through to the post-partum period through distinct processes in males and females. At a time of great change in the forces shaping adolescence, including the timing of parenthood, investments in today’s adolescents, the largest cohort in human history, will yield great dividends for future generations.   \n",
              "2167                                                                                                                                                                                                                                                                                                               Genetic studies have shown the association of Parkinson’s disease with alleles of the major histocompatibility complex1,2,3. Here we show that a defined set of peptides that are derived from α-synuclein, a protein aggregated in Parkinson’s disease4, act as antigenic epitopes displayed by these alleles and drive helper and cytotoxic T cell responses in patients with Parkinson’s disease. These responses may explain the association of Parkinson’s disease with specific major histocompatibility complex alleles.   \n",
              "1459                                                                                                                                                                                              The early evolution of turtles continues to be a contentious issue in vertebrate palaeontology. Recent reports have suggested that they are diapsids1,2,3,4,5,6, but the position of turtles within Diapsida is controversial7,8,9,10,11,12 and the sequence of acquisition of turtle synapomorphies remains unclear1,2,3. Here we describe a Triassic turtle from China that has a mixture of derived characters and plesiomorphic features. To our knowledge, it represents the earliest known stem turtle with an edentulous beak and a rigid puboischiadic plate. The discovery of this new form reveals a complex early history of turtles.   \n",
              "383                                                                                                                                                                            Research findings on the relationship between climate and conflict are diverse and contested. Here we assess the current understanding of the relationship between climate and conflict, based on the structured judgments of experts from diverse disciplines. These experts agree that climate has affected organized armed conflict within countries. However, other drivers, such as low socioeconomic development and low capabilities of the state, are judged to be substantially more influential, and the mechanisms of climate–conflict linkages remain a key uncertainty. Intensifying climate change is estimated to increase future risks of conflict.   \n",
              "3253                                                                                                                                   Clear cell renal cell carcinoma, the most common form of kidney cancer, is usually linked to inactivation of the pVHL tumour suppressor protein and consequent accumulation of the HIF-2α transcription factor (also known as EPAS1)1. Here we show that a small molecule (PT2399) that directly inhibits HIF-2α causes tumour regression in preclinical mouse models of primary and metastatic pVHL-defective clear cell renal cell carcinoma in an on-target fashion. pVHL-defective clear cell renal cell carcinoma cell lines display unexpectedly variable sensitivity to PT2399, however, suggesting the need for predictive biomarkers to be developed to use this approach optimally in the clinic.   \n",
              "1210                                                  Freshwater availability is changing worldwide. Here we quantify 34 trends in terrestrial water storage observed by the Gravity Recovery and Climate Experiment (GRACE) satellites during 2002–2016 and categorize their drivers as natural interannual variability, unsustainable groundwater consumption, climate change or combinations thereof. Several of these trends had been lacking thorough investigation and attribution, including massive changes in northwestern China and the Okavango Delta. Others are consistent with climate model predictions. This observation-based assessment of how the world’s water landscape is responding to human impacts and climate variations provides a blueprint for evaluating and predicting emerging threats to water and food security.   \n",
              "1384                           Visceral leishmaniasis causes considerable mortality and morbidity in many parts of the world. There is an urgent need for the development of new, effective treatments for this disease. Here we describe the development of an anti-leishmanial drug-like chemical series based on a pyrazolopyrimidine scaffold. The leading compound from this series (7, DDD853651/GSK3186899) is efficacious in a mouse model of visceral leishmaniasis, has suitable physicochemical, pharmacokinetic and toxicological properties for further development, and has been declared a preclinical candidate. Detailed mode-of-action studies indicate that compounds from this series act principally by inhibiting the parasite cdc-2-related kinase 12 (CRK12), thus defining a druggable target for visceral leishmaniasis.   \n",
              "823                         The underrepresentation of non-Europeans in human genetic studies so far has limited the diversity of individuals in genomic datasets and led to reduced medical relevance for a large proportion of the world’s population. Population-specific reference genome datasets as well as genome-wide association studies in diverse populations are needed to address this issue. Here we describe the pilot phase of the GenomeAsia 100K Project. This includes a whole-genome sequencing reference dataset from 1,739 individuals of 219 population groups and 64 countries across Asia. We catalogue genetic variation, population structure, disease associations and founder effects. We also explore the use of this dataset in imputation, to facilitate genetic studies in populations across Asia and worldwide.   \n",
              "940   The planarian Schmidtea mediterranea is an important model for stem cell research and regeneration, but adequate genome resources for this species have been lacking. Here we report a highly contiguous genome assembly of S. mediterranea, using long-read sequencing and a de novo assembler (MARVEL) enhanced for low-complexity reads. The S. mediterranea genome is highly polymorphic and repetitive, and harbours a novel class of giant retroelements. Furthermore, the genome assembly lacks a number of highly conserved genes, including critical components of the mitotic spindle assembly checkpoint, but planarians maintain checkpoint function. Our genome assembly provides a key model system resource that will be useful for studying regeneration and the evolutionary plasticity of core cell biological mechanisms.   \n",
              "\n",
              "          citations  abstract_len  \n",
              "2948  184 Citations           390  \n",
              "1007  209 Citations           404  \n",
              "2167  521 Citations           511  \n",
              "1459   43 Citations           624  \n",
              "383   274 Citations           643  \n",
              "3253  293 Citations           683  \n",
              "1210  993 Citations           764  \n",
              "1384  103 Citations           787  \n",
              "823   187 Citations           790  \n",
              "940   144 Citations           812  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-5fac1d12-cd83-4e12-ad94-d875db1260a3\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>citations</th>\n",
              "      <th>abstract_len</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>2948</th>\n",
              "      <td>Reproducible pharmacogenomic profiling of cancer cell line panels</td>\n",
              "      <td>The use of large-scale genomic and drug response screening of cancer cell lines depends crucially on the reproducibility of results. Here we consider two previously published screens, plus a later critique of these studies. Using independent data, we show that consistency is achievable, and provide a systematic description of the best laboratory and analysis practices for future studies.</td>\n",
              "      <td>184 Citations</td>\n",
              "      <td>390</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1007</th>\n",
              "      <td>Adolescence and the next generation</td>\n",
              "      <td>Adolescent growth and social development shape the early development of offspring from preconception through to the post-partum period through distinct processes in males and females. At a time of great change in the forces shaping adolescence, including the timing of parenthood, investments in today’s adolescents, the largest cohort in human history, will yield great dividends for future generations.</td>\n",
              "      <td>209 Citations</td>\n",
              "      <td>404</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2167</th>\n",
              "      <td>T cells from patients with Parkinson’s disease recognize α-synuclein peptides</td>\n",
              "      <td>Genetic studies have shown the association of Parkinson’s disease with alleles of the major histocompatibility complex1,2,3. Here we show that a defined set of peptides that are derived from α-synuclein, a protein aggregated in Parkinson’s disease4, act as antigenic epitopes displayed by these alleles and drive helper and cytotoxic T cell responses in patients with Parkinson’s disease. These responses may explain the association of Parkinson’s disease with specific major histocompatibility complex alleles.</td>\n",
              "      <td>521 Citations</td>\n",
              "      <td>511</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1459</th>\n",
              "      <td>A Triassic stem turtle with an edentulous beak</td>\n",
              "      <td>The early evolution of turtles continues to be a contentious issue in vertebrate palaeontology. Recent reports have suggested that they are diapsids1,2,3,4,5,6, but the position of turtles within Diapsida is controversial7,8,9,10,11,12 and the sequence of acquisition of turtle synapomorphies remains unclear1,2,3. Here we describe a Triassic turtle from China that has a mixture of derived characters and plesiomorphic features. To our knowledge, it represents the earliest known stem turtle with an edentulous beak and a rigid puboischiadic plate. The discovery of this new form reveals a complex early history of turtles.</td>\n",
              "      <td>43 Citations</td>\n",
              "      <td>624</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>383</th>\n",
              "      <td>Climate as a risk factor for armed conflict</td>\n",
              "      <td>Research findings on the relationship between climate and conflict are diverse and contested. Here we assess the current understanding of the relationship between climate and conflict, based on the structured judgments of experts from diverse disciplines. These experts agree that climate has affected organized armed conflict within countries. However, other drivers, such as low socioeconomic development and low capabilities of the state, are judged to be substantially more influential, and the mechanisms of climate–conflict linkages remain a key uncertainty. Intensifying climate change is estimated to increase future risks of conflict.</td>\n",
              "      <td>274 Citations</td>\n",
              "      <td>643</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3253</th>\n",
              "      <td>On-target efficacy of a HIF-2α antagonist in preclinical kidney cancer models</td>\n",
              "      <td>Clear cell renal cell carcinoma, the most common form of kidney cancer, is usually linked to inactivation of the pVHL tumour suppressor protein and consequent accumulation of the HIF-2α transcription factor (also known as EPAS1)1. Here we show that a small molecule (PT2399) that directly inhibits HIF-2α causes tumour regression in preclinical mouse models of primary and metastatic pVHL-defective clear cell renal cell carcinoma in an on-target fashion. pVHL-defective clear cell renal cell carcinoma cell lines display unexpectedly variable sensitivity to PT2399, however, suggesting the need for predictive biomarkers to be developed to use this approach optimally in the clinic.</td>\n",
              "      <td>293 Citations</td>\n",
              "      <td>683</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1210</th>\n",
              "      <td>Emerging trends in global freshwater availability</td>\n",
              "      <td>Freshwater availability is changing worldwide. Here we quantify 34 trends in terrestrial water storage observed by the Gravity Recovery and Climate Experiment (GRACE) satellites during 2002–2016 and categorize their drivers as natural interannual variability, unsustainable groundwater consumption, climate change or combinations thereof. Several of these trends had been lacking thorough investigation and attribution, including massive changes in northwestern China and the Okavango Delta. Others are consistent with climate model predictions. This observation-based assessment of how the world’s water landscape is responding to human impacts and climate variations provides a blueprint for evaluating and predicting emerging threats to water and food security.</td>\n",
              "      <td>993 Citations</td>\n",
              "      <td>764</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1384</th>\n",
              "      <td>Cyclin-dependent kinase 12 is a drug target for visceral leishmaniasis</td>\n",
              "      <td>Visceral leishmaniasis causes considerable mortality and morbidity in many parts of the world. There is an urgent need for the development of new, effective treatments for this disease. Here we describe the development of an anti-leishmanial drug-like chemical series based on a pyrazolopyrimidine scaffold. The leading compound from this series (7, DDD853651/GSK3186899) is efficacious in a mouse model of visceral leishmaniasis, has suitable physicochemical, pharmacokinetic and toxicological properties for further development, and has been declared a preclinical candidate. Detailed mode-of-action studies indicate that compounds from this series act principally by inhibiting the parasite cdc-2-related kinase 12 (CRK12), thus defining a druggable target for visceral leishmaniasis.</td>\n",
              "      <td>103 Citations</td>\n",
              "      <td>787</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>823</th>\n",
              "      <td>The GenomeAsia 100K Project enables genetic discoveries across Asia</td>\n",
              "      <td>The underrepresentation of non-Europeans in human genetic studies so far has limited the diversity of individuals in genomic datasets and led to reduced medical relevance for a large proportion of the world’s population. Population-specific reference genome datasets as well as genome-wide association studies in diverse populations are needed to address this issue. Here we describe the pilot phase of the GenomeAsia 100K Project. This includes a whole-genome sequencing reference dataset from 1,739 individuals of 219 population groups and 64 countries across Asia. We catalogue genetic variation, population structure, disease associations and founder effects. We also explore the use of this dataset in imputation, to facilitate genetic studies in populations across Asia and worldwide.</td>\n",
              "      <td>187 Citations</td>\n",
              "      <td>790</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>940</th>\n",
              "      <td>The genome of Schmidtea mediterranea and the evolution of core cellular mechanisms</td>\n",
              "      <td>The planarian Schmidtea mediterranea is an important model for stem cell research and regeneration, but adequate genome resources for this species have been lacking. Here we report a highly contiguous genome assembly of S. mediterranea, using long-read sequencing and a de novo assembler (MARVEL) enhanced for low-complexity reads. The S. mediterranea genome is highly polymorphic and repetitive, and harbours a novel class of giant retroelements. Furthermore, the genome assembly lacks a number of highly conserved genes, including critical components of the mitotic spindle assembly checkpoint, but planarians maintain checkpoint function. Our genome assembly provides a key model system resource that will be useful for studying regeneration and the evolutionary plasticity of core cell biological mechanisms.</td>\n",
              "      <td>144 Citations</td>\n",
              "      <td>812</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-5fac1d12-cd83-4e12-ad94-d875db1260a3')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-5fac1d12-cd83-4e12-ad94-d875db1260a3 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-5fac1d12-cd83-4e12-ad94-d875db1260a3');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-5697a384-23ac-43b5-9cae-a35209d4c4fe\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-5697a384-23ac-43b5-9cae-a35209d4c4fe')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-5697a384-23ac-43b5-9cae-a35209d4c4fe button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "summary": "{\n  \"name\": \"nature_df_16to19\",\n  \"rows\": 10,\n  \"fields\": [\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"The GenomeAsia 100K Project enables genetic discoveries across Asia\",\n          \"Adolescence and the next generation\",\n          \"On-target efficacy of a HIF-2\\u03b1 antagonist in preclinical kidney cancer models\"\n        ],\n        \"num_unique_values\": 10,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"The underrepresentation of non-Europeans in human genetic studies so far has limited the diversity of individuals in genomic datasets and led to reduced medical relevance for a large proportion of the world\\u2019s population. Population-specific reference genome datasets as well as genome-wide association studies in diverse populations are needed to address this issue. Here we describe the pilot phase of the GenomeAsia 100K Project. This includes a whole-genome sequencing reference dataset from 1,739 individuals of 219 population groups and 64 countries across Asia. We catalogue genetic variation, population structure, disease associations and founder effects. We also explore the use of this dataset in imputation, to facilitate genetic studies in populations across Asia and worldwide.\",\n          \"Adolescent growth and social development shape the early development of offspring from preconception through to the post-partum period through distinct processes in males and females. At a time of great change in the forces shaping adolescence, including the timing of parenthood, investments in today\\u2019s adolescents, the largest cohort in human history, will yield great dividends for future generations.\",\n          \"Clear cell renal cell carcinoma, the most common form of kidney cancer, is usually linked to inactivation of the pVHL tumour suppressor protein and consequent accumulation of the HIF-2\\u03b1 transcription factor (also known as EPAS1)1. Here we show that a small molecule (PT2399) that directly inhibits HIF-2\\u03b1 causes tumour regression in preclinical mouse models of primary and metastatic pVHL-defective clear cell renal cell carcinoma in an on-target fashion. pVHL-defective clear cell renal cell carcinoma cell lines display unexpectedly variable sensitivity to PT2399, however, suggesting the need for predictive biomarkers to be developed to use this approach optimally in the clinic.\"\n        ],\n        \"num_unique_values\": 10,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"citations\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"187 Citations\",\n          \"209 Citations\",\n          \"293 Citations\"\n        ],\n        \"num_unique_values\": 10,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_len\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 158,\n        \"min\": 390,\n        \"max\": 812,\n        \"samples\": [\n          790,\n          404,\n          683\n        ],\n        \"num_unique_values\": 10,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 119
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#Revmoving the word \"Citation\" from the citations\n",
        "nature_df_16to19['citations'] = pd.to_numeric(nature_df_16to19['citations'].str.replace(' Citations', '', regex=False), errors='coerce').fillna(0)"
      ],
      "metadata": {
        "id": "fIt65zRDz_51"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#sorting by the citations\n",
        "nature_df_16to19 = nature_df_16to19.sort_values(by='citations', ascending=False)\n",
        "nature_df_16to19 = nature_df_16to19.reset_index(drop=True)"
      ],
      "metadata": {
        "id": "NdLORlNt0Z-g"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "id": "GkbxKxVv0diV",
        "outputId": "2482d26b-63f8-4e58-b257-4f29c227bc16"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                                        title  \\\n",
              "0                 Dermatologist-level classification of skin cancer with deep neural networks   \n",
              "1                      Unconventional superconductivity in magic-angle graphene superlattices   \n",
              "2                            Quantum supremacy using a programmable superconducting processor   \n",
              "3       Layer-dependent ferromagnetism in a van der Waals crystal down to the monolayer limit   \n",
              "4                              The UK Biobank resource with deep phenotyping and genomic data   \n",
              "...                                                                                       ...   \n",
              "2836          Extreme 13C,15N and 17O isotopic enrichment in the young planetary nebula K4-47   \n",
              "2837                                            Mating preferences of selfish sex chromosomes   \n",
              "2838    Episodic kinematics in continental rifts modulated by changes in mantle melt fraction   \n",
              "2839  Vaccine-driven pharmacodynamic dissection and mitigation of fenethylline psychoactivity   \n",
              "2840                                Extreme disorder in an ultrahigh-affinity protein complex   \n",
              "\n",
              "                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              abstract  \\\n",
              "0                                                                                                                                                                                                                                                                                  Skin cancer, the most common human malignancy1,2,3, is primarily diagnosed visually, beginning with an initial clinical screening and followed potentially by dermoscopic analysis, a biopsy and histopathological examination. Automated classification of skin lesions using images is a challenging task owing to the fine-grained variability in the appearance of skin lesions. Deep convolutional neural networks (CNNs)4,5 show potential for general and highly variable tasks across many fine-grained object categories6,7,8,9,10,11. Here we demonstrate classification of skin lesions using a single CNN, trained end-to-end from images directly, using only pixels and disease labels as inputs. We train a CNN using a dataset of 129,450 clinical images—two orders of magnitude larger than previous datasets12—consisting of 2,032 different diseases. We test its performance against 21 board-certified dermatologists on biopsy-proven clinical images with two critical binary classification use cases: keratinocyte carcinomas versus benign seborrheic keratoses; and malignant melanomas versus benign nevi. The first case represents the identification of the most common cancers, the second represents the identification of the deadliest skin cancer. The CNN achieves performance on par with all tested experts across both tasks, demonstrating an artificial intelligence capable of classifying skin cancer with a level of competence comparable to dermatologists. Outfitted with deep neural networks, mobile devices can potentially extend the reach of dermatologists outside of the clinic. It is projected that 6.3 billion smartphone subscriptions will exist by the year 2021 (ref. 13) and can therefore potentially provide low-cost universal access to vital diagnostic care.   \n",
              "1     The behaviour of strongly correlated materials, and in particular unconventional superconductors, has been studied extensively for decades, but is still not well understood. This lack of theoretical understanding has motivated the development of experimental techniques for studying such behaviour, such as using ultracold atom lattices to simulate quantum materials. Here we report the realization of intrinsic unconventional superconductivity—which cannot be explained by weak electron–phonon interactions—in a two-dimensional superlattice created by stacking two sheets of graphene that are twisted relative to each other by a small angle. For twist angles of about 1.1°—the first ‘magic’ angle—the electronic band structure of this ‘twisted bilayer graphene’ exhibits flat bands near zero Fermi energy, resulting in correlated insulating states at half-filling. Upon electrostatic doping of the material away from these correlated insulating states, we observe tunable zero-resistance states with a critical temperature of up to 1.7 kelvin. The temperature–carrier-density phase diagram of twisted bilayer graphene is similar to that of copper oxides (or cuprates), and includes dome-shaped regions that correspond to superconductivity. Moreover, quantum oscillations in the longitudinal resistance of the material indicate the presence of small Fermi surfaces near the correlated insulating states, in analogy with underdoped cuprates. The relatively high superconducting critical temperature of twisted bilayer graphene, given such a small Fermi surface (which corresponds to a carrier density of about 1011 per square centimetre), puts it among the superconductors with the strongest pairing strength between electrons. Twisted bilayer graphene is a precisely tunable, purely carbon-based, two-dimensional superconductor. It is therefore an ideal material for investigations of strongly correlated phenomena, which could lead to insights into the physics of high-critical-temperature superconductors and quantum spin liquids.   \n",
              "2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            The promise of quantum computers is that certain computational tasks might be executed exponentially faster on a quantum processor than on a classical processor1. A fundamental challenge is to build a high-fidelity processor capable of running quantum algorithms in an exponentially large computational space. Here we report the use of a processor with programmable superconducting qubits2,3,4,5,6,7 to create quantum states on 53 qubits, corresponding to a computational state-space of dimension 253 (about 1016). Measurements from repeated experiments sample the resulting probability distribution, which we verify using classical simulations. Our Sycamore processor takes about 200 seconds to sample one instance of a quantum circuit a million times—our benchmarks currently indicate that the equivalent task for a state-of-the-art classical supercomputer would take approximately 10,000 years. This dramatic increase in speed compared to all known classical algorithms is an experimental realization of quantum supremacy8,9,10,11,12,13,14 for this specific computational task, heralding a much-anticipated computing paradigm.   \n",
              "3                                                                                                                                                                                                                                                                              Since the discovery of graphene1, the family of two-dimensional materials has grown, displaying a broad range of electronic properties. Recent additions include semiconductors with spin–valley coupling2, Ising superconductors3,4,5 that can be tuned into a quantum metal6, possible Mott insulators with tunable charge-density waves7, and topological semimetals with edge transport8,9. However, no two-dimensional crystal with intrinsic magnetism has yet been discovered10,11,12,13,14; such a crystal would be useful in many technologies from sensing to data storage15. Theoretically, magnetic order is prohibited in the two-dimensional isotropic Heisenberg model at finite temperatures by the Mermin–Wagner theorem16. Magnetic anisotropy removes this restriction, however, and enables, for instance, the occurrence of two-dimensional Ising ferromagnetism. Here we use magneto-optical Kerr effect microscopy to demonstrate that monolayer chromium triiodide (CrI3) is an Ising ferromagnet with out-of-plane spin orientation. Its Curie temperature of 45 kelvin is only slightly lower than that of the bulk crystal, 61 kelvin, which is consistent with a weak interlayer coupling. Moreover, our studies suggest a layer-dependent magnetic phase, highlighting thickness-dependent physical properties typical of van der Waals crystals17,18,19. Remarkably, bilayer CrI3 displays suppressed magnetization with a metamagnetic effect20, whereas in trilayer CrI3 the interlayer ferromagnetism observed in the bulk crystal is restored. This work creates opportunities for studying magnetism by harnessing the unusual features of atomically thin materials, such as electrical control for realizing magnetoelectronics12, and van der Waals engineering to produce interface phenomena15.   \n",
              "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. The open resource is unique in its size and scope. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits. Here we describe the centralized analysis of the genetic data, including genotype quality, properties of population structure and relatedness of the genetic data, and efficient phasing and genotype imputation that increases the number of testable variants to around 96 million. Classical allelic variation at 11 human leukocyte antigen genes was imputed, resulting in the recovery of signals with known associations between human leukocyte antigen alleles and many diseases.   \n",
              "...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                ...   \n",
              "2836                                                                                                                                      Carbon, nitrogen and oxygen are the three most abundant elements in the Galaxy after hydrogen and helium. Whereas hydrogen and helium were created in the Big Bang, carbon, nitrogen and oxygen arise from nucleosynthesis in stars. Of particular interest1,2 are the isotopic ratios 12C/13C, 14N/15N and 16O/17O because they are effective tracers of nucleosynthesis and help to benchmark the chemical processes that occurred in primitive interstellar material as it evolved into our Solar System3. However, the origins of the rare isotopes 15N and 17O remain uncertain, although novae and very massive stars that explode as supernovae are postulated4,5,6 to be the main sources of 15N. Here we report millimetre-wavelength observations of the young bipolar planetary nebula K4-47 that indicate another possible source for these isotopes. We identify various carbon-bearing molecules in K4-47 that show that this object is carbon-rich, and find unusually high enrichment in rare carbon (13C), oxygen (17O) and nitrogen (15N) isotopes: 12C/13C = 2.2 ± 0.8, 16O/17O = 21.4 ± 10.3 and 14N/15N = 13.6 ± 6.5 (uncertainties are three standard deviations); for comparison, the corresponding solar ratios7 are 89.4 ± 0.2, 2,632 ± 7 and 435 ± 57. One possible interpretation of these results is that K4-47 arose from a J-type asymptotic giant branch star that underwent a helium-shell flash (an explosive nucleosynthetic event that converts large quantities of helium to carbon and other elements), enriching the resulting planetary nebula in 15N and 17O and creating its bipolar geometry. Other possible explanations are that K4-47 is a binary system or that it resulted from a white dwarf merger, as has been suggested for object CK Vul8. These results suggest that nucleosynthesis of carbon, nitrogen and oxygen is not well understood and that the classification of certain stardust grains must be reconsidered.   \n",
              "2837                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 The evolution of female mating preferences for harmful male traits is a central paradox of sexual selection1,2,3,4,5,6,7,8,9. Two dominant explanations for this paradox8,10 are Fisher’s runaway process, which is based on genetic correlations between preference and trait1,3,4, and Zahavi’s handicap principle, in which the trait is an honest costly signal of male quality2,6,8,11. However, both of these explanations require the exogenous initial spread of female preferences before harmful male traits can evolve1,2,3,4,6,8,11. Here I present a mechanism for the evolution of female mating preferences for harmful male traits that is based on the selfish evolutionary interests of sex chromosomes. I demonstrate that female-biased genetic elements—such as the W and X sex chromosomes—will evolve mating preferences for males who display traits that reduce their fitness and/or that of their male offspring, but increase fitness in female offspring. In particular, W-linked preferences can cause nearly lethal male traits to sweep to fixation. Sex-linked preferences can drive the evolution of traits such as ornamental handicaps and male parental care, and can explain variation in ornamentation and behaviour across taxa with divergent sex-determining mechanisms.   \n",
              "2838                                                                                                                                                                                                                                                                                Oceanic crust is created by the extraction of molten rock from underlying mantle at the seafloor ‘spreading centres’ found between diverging tectonic plates. Modelling studies have suggested that mantle melting can occur through decompression as the mantle flows upwards beneath spreading centres1, but direct observation of this process is difficult beneath the oceans. Continental rifts, however—which are also associated with mantle melt production—are amenable to detailed measurements of their short-term kinematics using geodetic techniques. Here we show that such data can provide evidence for an upwelling mantle flow, as well as information on the dimensions and timescale of mantle melting. For North Island, New Zealand, around ten years of campaign and continuous GPS measurements in the continental rift system known as the Taupo volcanic zone reveal that it is extending at a rate of 6–15 millimetres per year. However, a roughly 70-kilometre-long segment of the rift axis is associated with strong horizontal contraction and rapid subsidence, and is flanked by regions of extension and uplift. These features fit a simple model that involves flexure of an elastic upper crust, which is pulled downwards or pushed upwards along the rift axis by a driving force located at a depth greater than 15 kilometres. We propose that flexure is caused by melt-induced episodic changes in the vertical flow forces that are generated by upwelling mantle beneath the rift axis, triggering a transient lower-crustal flow. A drop in the melt fraction owing to melt extraction raises the mantle flow viscosity and drives subsidence, whereas melt accumulation reduces viscosity and allows uplift—processes that are also likely to occur in oceanic spreading centres.   \n",
              "2839                                                                                                                                                                                                                                                                                                                                                                                                                     Fenethylline, also known by the trade name Captagon, is a synthetic psychoactive stimulant that has recently been linked to a substance-use disorder and ‘pharmacoterrorism’ in the Middle East1,2,3,4. Although fenethylline shares a common phenethylamine core with other amphetamine-type stimulants, it also incorporates a covalently linked xanthine moiety into its parent structure5,6. These independently active pharmacophores are liberated during metabolism, resulting in the release of a structurally diverse chemical mixture into the central nervous system7,8,9. Although the psychoactive properties of fenethylline have been reported to differ from those of other synthetic stimulants, the in vivo chemical complexity it manifests upon ingestion has impeded efforts to unambiguously identify the specific species responsible for these effects10,11. Here we develop a ‘dissection through vaccination’ approach, called DISSECTIV, to mitigate the psychoactive effects of fenethylline and show that its rapid-onset and distinct psychoactive properties are facilitated by functional synergy between theophylline and amphetamine. Our results demonstrate that incremental vaccination against a single chemical species within a multi-component mixture can be used to uncover emergent properties arising from polypharmacological activity. We anticipate that DISSECTIV will be used to expose unidentified active chemical species and resolve pharmacodynamic interactions within other chemically complex systems, such as those found in counterfeit or illegal drug preparations, post-metabolic tissue samples and natural product extracts.   \n",
              "2840                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Molecular communication in biology is mediated by protein interactions. According to the current paradigm, the specificity and affinity required for these interactions are encoded in the precise complementarity of binding interfaces. Even proteins that are disordered under physiological conditions or that contain large unstructured regions commonly interact with well-structured binding sites on other biomolecules. Here we demonstrate the existence of an unexpected interaction mechanism: the two intrinsically disordered human proteins histone H1 and its nuclear chaperone prothymosin-α associate in a complex with picomolar affinity, but fully retain their structural disorder, long-range flexibility and highly dynamic character. On the basis of closely integrated experiments and molecular simulations, we show that the interaction can be explained by the large opposite net charge of the two proteins, without requiring defined binding sites or interactions between specific individual residues. Proteome-wide sequence analysis suggests that this interaction mechanism may be abundant in eukaryotes.   \n",
              "\n",
              "      citations  abstract_len  \n",
              "0        7133.0          1763  \n",
              "1        4972.0          2032  \n",
              "2        3847.0          1129  \n",
              "3        3649.0          1767  \n",
              "4        3429.0          1207  \n",
              "...         ...           ...  \n",
              "2836       13.0          1900  \n",
              "2837       12.0          1265  \n",
              "2838       10.0          1762  \n",
              "2839        9.0          1629  \n",
              "2840        0.0          1107  \n",
              "\n",
              "[2841 rows x 4 columns]"
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-f7acd12e-2feb-4da7-ba1b-a617ae616a15\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>title</th>\n",
              "      <th>abstract</th>\n",
              "      <th>citations</th>\n",
              "      <th>abstract_len</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Dermatologist-level classification of skin cancer with deep neural networks</td>\n",
              "      <td>Skin cancer, the most common human malignancy1,2,3, is primarily diagnosed visually, beginning with an initial clinical screening and followed potentially by dermoscopic analysis, a biopsy and histopathological examination. Automated classification of skin lesions using images is a challenging task owing to the fine-grained variability in the appearance of skin lesions. Deep convolutional neural networks (CNNs)4,5 show potential for general and highly variable tasks across many fine-grained object categories6,7,8,9,10,11. Here we demonstrate classification of skin lesions using a single CNN, trained end-to-end from images directly, using only pixels and disease labels as inputs. We train a CNN using a dataset of 129,450 clinical images—two orders of magnitude larger than previous datasets12—consisting of 2,032 different diseases. We test its performance against 21 board-certified dermatologists on biopsy-proven clinical images with two critical binary classification use cases: keratinocyte carcinomas versus benign seborrheic keratoses; and malignant melanomas versus benign nevi. The first case represents the identification of the most common cancers, the second represents the identification of the deadliest skin cancer. The CNN achieves performance on par with all tested experts across both tasks, demonstrating an artificial intelligence capable of classifying skin cancer with a level of competence comparable to dermatologists. Outfitted with deep neural networks, mobile devices can potentially extend the reach of dermatologists outside of the clinic. It is projected that 6.3 billion smartphone subscriptions will exist by the year 2021 (ref. 13) and can therefore potentially provide low-cost universal access to vital diagnostic care.</td>\n",
              "      <td>7133.0</td>\n",
              "      <td>1763</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>Unconventional superconductivity in magic-angle graphene superlattices</td>\n",
              "      <td>The behaviour of strongly correlated materials, and in particular unconventional superconductors, has been studied extensively for decades, but is still not well understood. This lack of theoretical understanding has motivated the development of experimental techniques for studying such behaviour, such as using ultracold atom lattices to simulate quantum materials. Here we report the realization of intrinsic unconventional superconductivity—which cannot be explained by weak electron–phonon interactions—in a two-dimensional superlattice created by stacking two sheets of graphene that are twisted relative to each other by a small angle. For twist angles of about 1.1°—the first ‘magic’ angle—the electronic band structure of this ‘twisted bilayer graphene’ exhibits flat bands near zero Fermi energy, resulting in correlated insulating states at half-filling. Upon electrostatic doping of the material away from these correlated insulating states, we observe tunable zero-resistance states with a critical temperature of up to 1.7 kelvin. The temperature–carrier-density phase diagram of twisted bilayer graphene is similar to that of copper oxides (or cuprates), and includes dome-shaped regions that correspond to superconductivity. Moreover, quantum oscillations in the longitudinal resistance of the material indicate the presence of small Fermi surfaces near the correlated insulating states, in analogy with underdoped cuprates. The relatively high superconducting critical temperature of twisted bilayer graphene, given such a small Fermi surface (which corresponds to a carrier density of about 1011 per square centimetre), puts it among the superconductors with the strongest pairing strength between electrons. Twisted bilayer graphene is a precisely tunable, purely carbon-based, two-dimensional superconductor. It is therefore an ideal material for investigations of strongly correlated phenomena, which could lead to insights into the physics of high-critical-temperature superconductors and quantum spin liquids.</td>\n",
              "      <td>4972.0</td>\n",
              "      <td>2032</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2</th>\n",
              "      <td>Quantum supremacy using a programmable superconducting processor</td>\n",
              "      <td>The promise of quantum computers is that certain computational tasks might be executed exponentially faster on a quantum processor than on a classical processor1. A fundamental challenge is to build a high-fidelity processor capable of running quantum algorithms in an exponentially large computational space. Here we report the use of a processor with programmable superconducting qubits2,3,4,5,6,7 to create quantum states on 53 qubits, corresponding to a computational state-space of dimension 253 (about 1016). Measurements from repeated experiments sample the resulting probability distribution, which we verify using classical simulations. Our Sycamore processor takes about 200 seconds to sample one instance of a quantum circuit a million times—our benchmarks currently indicate that the equivalent task for a state-of-the-art classical supercomputer would take approximately 10,000 years. This dramatic increase in speed compared to all known classical algorithms is an experimental realization of quantum supremacy8,9,10,11,12,13,14 for this specific computational task, heralding a much-anticipated computing paradigm.</td>\n",
              "      <td>3847.0</td>\n",
              "      <td>1129</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>3</th>\n",
              "      <td>Layer-dependent ferromagnetism in a van der Waals crystal down to the monolayer limit</td>\n",
              "      <td>Since the discovery of graphene1, the family of two-dimensional materials has grown, displaying a broad range of electronic properties. Recent additions include semiconductors with spin–valley coupling2, Ising superconductors3,4,5 that can be tuned into a quantum metal6, possible Mott insulators with tunable charge-density waves7, and topological semimetals with edge transport8,9. However, no two-dimensional crystal with intrinsic magnetism has yet been discovered10,11,12,13,14; such a crystal would be useful in many technologies from sensing to data storage15. Theoretically, magnetic order is prohibited in the two-dimensional isotropic Heisenberg model at finite temperatures by the Mermin–Wagner theorem16. Magnetic anisotropy removes this restriction, however, and enables, for instance, the occurrence of two-dimensional Ising ferromagnetism. Here we use magneto-optical Kerr effect microscopy to demonstrate that monolayer chromium triiodide (CrI3) is an Ising ferromagnet with out-of-plane spin orientation. Its Curie temperature of 45 kelvin is only slightly lower than that of the bulk crystal, 61 kelvin, which is consistent with a weak interlayer coupling. Moreover, our studies suggest a layer-dependent magnetic phase, highlighting thickness-dependent physical properties typical of van der Waals crystals17,18,19. Remarkably, bilayer CrI3 displays suppressed magnetization with a metamagnetic effect20, whereas in trilayer CrI3 the interlayer ferromagnetism observed in the bulk crystal is restored. This work creates opportunities for studying magnetism by harnessing the unusual features of atomically thin materials, such as electrical control for realizing magnetoelectronics12, and van der Waals engineering to produce interface phenomena15.</td>\n",
              "      <td>3649.0</td>\n",
              "      <td>1767</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>4</th>\n",
              "      <td>The UK Biobank resource with deep phenotyping and genomic data</td>\n",
              "      <td>The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. The open resource is unique in its size and scope. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits. Here we describe the centralized analysis of the genetic data, including genotype quality, properties of population structure and relatedness of the genetic data, and efficient phasing and genotype imputation that increases the number of testable variants to around 96 million. Classical allelic variation at 11 human leukocyte antigen genes was imputed, resulting in the recovery of signals with known associations between human leukocyte antigen alleles and many diseases.</td>\n",
              "      <td>3429.0</td>\n",
              "      <td>1207</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>...</th>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "      <td>...</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2836</th>\n",
              "      <td>Extreme 13C,15N and 17O isotopic enrichment in the young planetary nebula K4-47</td>\n",
              "      <td>Carbon, nitrogen and oxygen are the three most abundant elements in the Galaxy after hydrogen and helium. Whereas hydrogen and helium were created in the Big Bang, carbon, nitrogen and oxygen arise from nucleosynthesis in stars. Of particular interest1,2 are the isotopic ratios 12C/13C, 14N/15N and 16O/17O because they are effective tracers of nucleosynthesis and help to benchmark the chemical processes that occurred in primitive interstellar material as it evolved into our Solar System3. However, the origins of the rare isotopes 15N and 17O remain uncertain, although novae and very massive stars that explode as supernovae are postulated4,5,6 to be the main sources of 15N. Here we report millimetre-wavelength observations of the young bipolar planetary nebula K4-47 that indicate another possible source for these isotopes. We identify various carbon-bearing molecules in K4-47 that show that this object is carbon-rich, and find unusually high enrichment in rare carbon (13C), oxygen (17O) and nitrogen (15N) isotopes: 12C/13C = 2.2 ± 0.8, 16O/17O = 21.4 ± 10.3 and 14N/15N = 13.6 ± 6.5 (uncertainties are three standard deviations); for comparison, the corresponding solar ratios7 are 89.4 ± 0.2, 2,632 ± 7 and 435 ± 57. One possible interpretation of these results is that K4-47 arose from a J-type asymptotic giant branch star that underwent a helium-shell flash (an explosive nucleosynthetic event that converts large quantities of helium to carbon and other elements), enriching the resulting planetary nebula in 15N and 17O and creating its bipolar geometry. Other possible explanations are that K4-47 is a binary system or that it resulted from a white dwarf merger, as has been suggested for object CK Vul8. These results suggest that nucleosynthesis of carbon, nitrogen and oxygen is not well understood and that the classification of certain stardust grains must be reconsidered.</td>\n",
              "      <td>13.0</td>\n",
              "      <td>1900</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2837</th>\n",
              "      <td>Mating preferences of selfish sex chromosomes</td>\n",
              "      <td>The evolution of female mating preferences for harmful male traits is a central paradox of sexual selection1,2,3,4,5,6,7,8,9. Two dominant explanations for this paradox8,10 are Fisher’s runaway process, which is based on genetic correlations between preference and trait1,3,4, and Zahavi’s handicap principle, in which the trait is an honest costly signal of male quality2,6,8,11. However, both of these explanations require the exogenous initial spread of female preferences before harmful male traits can evolve1,2,3,4,6,8,11. Here I present a mechanism for the evolution of female mating preferences for harmful male traits that is based on the selfish evolutionary interests of sex chromosomes. I demonstrate that female-biased genetic elements—such as the W and X sex chromosomes—will evolve mating preferences for males who display traits that reduce their fitness and/or that of their male offspring, but increase fitness in female offspring. In particular, W-linked preferences can cause nearly lethal male traits to sweep to fixation. Sex-linked preferences can drive the evolution of traits such as ornamental handicaps and male parental care, and can explain variation in ornamentation and behaviour across taxa with divergent sex-determining mechanisms.</td>\n",
              "      <td>12.0</td>\n",
              "      <td>1265</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2838</th>\n",
              "      <td>Episodic kinematics in continental rifts modulated by changes in mantle melt fraction</td>\n",
              "      <td>Oceanic crust is created by the extraction of molten rock from underlying mantle at the seafloor ‘spreading centres’ found between diverging tectonic plates. Modelling studies have suggested that mantle melting can occur through decompression as the mantle flows upwards beneath spreading centres1, but direct observation of this process is difficult beneath the oceans. Continental rifts, however—which are also associated with mantle melt production—are amenable to detailed measurements of their short-term kinematics using geodetic techniques. Here we show that such data can provide evidence for an upwelling mantle flow, as well as information on the dimensions and timescale of mantle melting. For North Island, New Zealand, around ten years of campaign and continuous GPS measurements in the continental rift system known as the Taupo volcanic zone reveal that it is extending at a rate of 6–15 millimetres per year. However, a roughly 70-kilometre-long segment of the rift axis is associated with strong horizontal contraction and rapid subsidence, and is flanked by regions of extension and uplift. These features fit a simple model that involves flexure of an elastic upper crust, which is pulled downwards or pushed upwards along the rift axis by a driving force located at a depth greater than 15 kilometres. We propose that flexure is caused by melt-induced episodic changes in the vertical flow forces that are generated by upwelling mantle beneath the rift axis, triggering a transient lower-crustal flow. A drop in the melt fraction owing to melt extraction raises the mantle flow viscosity and drives subsidence, whereas melt accumulation reduces viscosity and allows uplift—processes that are also likely to occur in oceanic spreading centres.</td>\n",
              "      <td>10.0</td>\n",
              "      <td>1762</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2839</th>\n",
              "      <td>Vaccine-driven pharmacodynamic dissection and mitigation of fenethylline psychoactivity</td>\n",
              "      <td>Fenethylline, also known by the trade name Captagon, is a synthetic psychoactive stimulant that has recently been linked to a substance-use disorder and ‘pharmacoterrorism’ in the Middle East1,2,3,4. Although fenethylline shares a common phenethylamine core with other amphetamine-type stimulants, it also incorporates a covalently linked xanthine moiety into its parent structure5,6. These independently active pharmacophores are liberated during metabolism, resulting in the release of a structurally diverse chemical mixture into the central nervous system7,8,9. Although the psychoactive properties of fenethylline have been reported to differ from those of other synthetic stimulants, the in vivo chemical complexity it manifests upon ingestion has impeded efforts to unambiguously identify the specific species responsible for these effects10,11. Here we develop a ‘dissection through vaccination’ approach, called DISSECTIV, to mitigate the psychoactive effects of fenethylline and show that its rapid-onset and distinct psychoactive properties are facilitated by functional synergy between theophylline and amphetamine. Our results demonstrate that incremental vaccination against a single chemical species within a multi-component mixture can be used to uncover emergent properties arising from polypharmacological activity. We anticipate that DISSECTIV will be used to expose unidentified active chemical species and resolve pharmacodynamic interactions within other chemically complex systems, such as those found in counterfeit or illegal drug preparations, post-metabolic tissue samples and natural product extracts.</td>\n",
              "      <td>9.0</td>\n",
              "      <td>1629</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>2840</th>\n",
              "      <td>Extreme disorder in an ultrahigh-affinity protein complex</td>\n",
              "      <td>Molecular communication in biology is mediated by protein interactions. According to the current paradigm, the specificity and affinity required for these interactions are encoded in the precise complementarity of binding interfaces. Even proteins that are disordered under physiological conditions or that contain large unstructured regions commonly interact with well-structured binding sites on other biomolecules. Here we demonstrate the existence of an unexpected interaction mechanism: the two intrinsically disordered human proteins histone H1 and its nuclear chaperone prothymosin-α associate in a complex with picomolar affinity, but fully retain their structural disorder, long-range flexibility and highly dynamic character. On the basis of closely integrated experiments and molecular simulations, we show that the interaction can be explained by the large opposite net charge of the two proteins, without requiring defined binding sites or interactions between specific individual residues. Proteome-wide sequence analysis suggests that this interaction mechanism may be abundant in eukaryotes.</td>\n",
              "      <td>0.0</td>\n",
              "      <td>1107</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "<p>2841 rows × 4 columns</p>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-f7acd12e-2feb-4da7-ba1b-a617ae616a15')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-f7acd12e-2feb-4da7-ba1b-a617ae616a15 button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-f7acd12e-2feb-4da7-ba1b-a617ae616a15');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-feb4fded-b921-410d-813f-6a3279c854d4\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-feb4fded-b921-410d-813f-6a3279c854d4')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-feb4fded-b921-410d-813f-6a3279c854d4 button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "variable_name": "nature_df_16to19",
              "summary": "{\n  \"name\": \"nature_df_16to19\",\n  \"rows\": 2841,\n  \"fields\": [\n    {\n      \"column\": \"title\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"Self-organization of the in vitro attached human embryo\",\n          \"The molecular landscape of ETMR at diagnosis and relapse\",\n          \"Oxidative diversification of amino acids and peptides by small-molecule iron catalysis\"\n        ],\n        \"num_unique_values\": 2841,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract\",\n      \"properties\": {\n        \"dtype\": \"string\",\n        \"samples\": [\n          \"Implantation of the blastocyst is a developmental milestone in mammalian embryonic development. At this time, a coordinated program of lineage diversification, cell-fate specification, and morphogenetic movements establishes the generation of extra-embryonic tissues and the embryo proper, and determines the conditions for successful pregnancy and gastrulation. Despite its basic and clinical importance, this process remains mysterious in humans. Here we report the use of a novel in vitro system1,2 to study the post-implantation development of the human embryo. We unveil the self-organizing abilities and autonomy of in vitro attached human embryos. We find human-specific molecular signatures of early cell lineage, timing, and architecture. Embryos display key landmarks of normal development, including epiblast expansion, lineage segregation, bi-laminar disc formation, amniotic and yolk sac cavitation, and trophoblast diversification. Our findings highlight the species-specificity of these developmental events and provide a new understanding of early human embryonic development beyond the blastocyst stage. In addition, our study establishes a new model system relevant to early human pregnancy loss. Finally, our work will also assist in the rational design of differentiation protocols of human embryonic stem cells to specific cell types for disease modelling and cell replacement therapy.\",\n          \"Embryonal tumours with multilayered rosettes (ETMRs) are aggressive paediatric embryonal brain tumours with a universally poor prognosis1. Here we collected 193 primary ETMRs and 23 matched relapse samples to investigate the genomic landscape of this distinct tumour type. We found that patients with tumours in which the proposed driver C19MC2,3,4 was not amplified frequently had germline mutations in DICER1 or other microRNA-related aberrations such as somatic amplification of miR-17-92 (also known as MIR17HG). Whole-genome sequencing revealed that tumours had an overall low recurrence of single-nucleotide variants (SNVs), but showed prevalent genomic instability caused by widespread occurrence of R-loop structures. We show that R-loop-associated chromosomal instability can be induced by the loss of DICER1 function. Comparison of primary tumours and matched relapse samples showed a strong conservation of structural variants, but low conservation of SNVs. Moreover, many newly acquired SNVs are associated with a mutational signature related to cisplatin treatment. Finally, we show that targeting R-loops with topoisomerase and PARP inhibitors might be an effective treatment strategy for this deadly disease.\",\n          \"Secondary metabolites synthesized by non-ribosomal peptide synthetases display diverse and complex topologies and possess a range of biological activities1,2. Much of this diversity derives from a synthetic strategy that entails pre-3 and post-assembly2 oxidation of both the chiral amino acid building blocks and the assembled peptide scaffolds. The vancomycin biosynthetic pathway is an excellent example of the range of oxidative transformations that can be performed by the iron-containing enzymes involved in its biosynthesis4. However, because of the challenges associated with using such oxidative enzymes to carry out chemical transformations in vitro, chemical syntheses guided by these principles have not been fully realized in the laboratory5. Here we report that two small-molecule iron catalysts are capable of facilitating the targeted C\\u2013H oxidative modification of amino acids and peptides with preservation of \\u03b1-centre chirality. Oxidation of proline to 5-hydroxyproline furnishes a versatile intermediate that can be transformed to rigid arylated derivatives or flexible linear carboxylic acids, alcohols, olefins and amines in both monomer and peptide settings. The value of this C\\u2013H oxidation strategy is demonstrated in its capacity for generating diversity: four \\u2018chiral pool\\u2019 amino acids are transformed to twenty-one chiral unnatural amino acids representing seven distinct functional group arrays; late-stage C\\u2013H functionalizations of a single proline-containing tripeptide furnish eight tripeptides, each having different unnatural amino acids. Additionally, a macrocyclic peptide containing a proline turn element is transformed via late-stage C\\u2013H oxidation to one containing a linear unnatural amino acid.\"\n        ],\n        \"num_unique_values\": 2841,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"citations\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 331.06741243643376,\n        \"min\": 0.0,\n        \"max\": 7133.0,\n        \"samples\": [\n          30.0,\n          648.0,\n          1135.0\n        ],\n        \"num_unique_values\": 701,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    },\n    {\n      \"column\": \"abstract_len\",\n      \"properties\": {\n        \"dtype\": \"number\",\n        \"std\": 340,\n        \"min\": 390,\n        \"max\": 2857,\n        \"samples\": [\n          1087,\n          1553,\n          1241\n        ],\n        \"num_unique_values\": 1182,\n        \"semantic_type\": \"\",\n        \"description\": \"\"\n      }\n    }\n  ]\n}"
            }
          },
          "metadata": {},
          "execution_count": 122
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "oDaZeluSFd4L",
        "outputId": "87c71547-cace-4994-8b31-b358029a5294"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(2841, 4)"
            ]
          },
          "metadata": {},
          "execution_count": 123
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "nature_df_16to19.to_csv('nature 16to19.csv', index=False)"
      ],
      "metadata": {
        "id": "NbgUUWvDCX2K"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "source": [
        "##Preparing data sets from different journals"
      ],
      "metadata": {
        "id": "myZe9ZZN7I7p"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_most_1 = pd.read_csv('/content/scopus_most_cited_1.csv')\n",
        "scopus_least_1 = pd.read_csv('/content/scopus_least_cited.csv')"
      ],
      "metadata": {
        "id": "crKa7bkul1k3"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all = pd.concat([scopus_most_1, scopus_least_1])"
      ],
      "metadata": {
        "id": "FqN2nBAn1be7"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.columns"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "E5OIXj151oQz",
        "outputId": "ccc9a293-d830-4047-d8e3-25a5ccf1fa56"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Index(['Title', 'Year', 'Source title', 'Cited by', 'Link', 'Abstract'], dtype='object')"
            ]
          },
          "metadata": {},
          "execution_count": 144
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all = scopus_all.rename(columns={'Source title': 'journal', 'Abstract': 'abstract','Title':'title', \"Cited by\": \"citations\"})"
      ],
      "metadata": {
        "id": "9aCj0OxBrF36"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "#Removing \"Nature\" journals from the non-Nature set\n",
        "def removeNature(df):\n",
        "  mask = df['journal'].str.startswith('Nature')\n",
        "  df = df[~mask]\n",
        "\n",
        "  return df"
      ],
      "metadata": {
        "id": "ITIbhydjKxRb"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all = removeNature(scopus_all)"
      ],
      "metadata": {
        "id": "FyTakuliRM9s"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all = dataprep(scopus_all)"
      ],
      "metadata": {
        "id": "Affn-6CwLJmn"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "e8kdSOZkLFtf",
        "outputId": "33e97c8c-b0af-46aa-fdc6-693dc8f61e10"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(3543, 6)"
            ]
          },
          "metadata": {},
          "execution_count": 149
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "##Inspecting the abstracts (if they have any artifacts) e.g. \"© 2017 American Chemical Society.\""
      ],
      "metadata": {
        "id": "jX7qoN1lD7_s"
      }
    },
    {
      "cell_type": "markdown",
      "source": [
        "###Preparing the whole data set"
      ],
      "metadata": {
        "id": "pBo8bAc98II-"
      }
    },
    {
      "cell_type": "code",
      "source": [
        "pd.set_option('display.max_colwidth', None)"
      ],
      "metadata": {
        "id": "CVvcH6sODaqu"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all[\"abstract\"].head(50)"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "OjbddghdDceb",
        "outputId": "df3d7494-e984-4256-a6e6-96ff8123d141"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     The number of cancer survivors continues to increase because of both advances in early detection and treatment and the aging and growth of the population. For the public health community to better serve these survivors, the American Cancer Society and the National Cancer Institute collaborate to estimate the number of current and future cancer survivors using data from the Surveillance, Epidemiology, and End Results cancer registries. In addition, current treatment patterns for the most prevalent cancer types are presented based on information in the National Cancer Data Base and treatment-related side effects are briefly described. More than 15.5 million Americans with a history of cancer were alive on January 1, 2016, and this number is projected to reach more than 20 million by January 1, 2026. The 3 most prevalent cancers are prostate (3,306,760), colon and rectum (724,690), and melanoma (614,460) among males and breast (3,560,570), uterine corpus (757,190), and colon and rectum (727,350) among females. More than one-half (56%) of survivors were diagnosed within the past 10 years, and almost one-half (47%) are aged 70 years or older. People with a history of cancer have unique medical and psychosocial needs that require proactive assessment and management by primary care providers. Although there are a growing number of tools that can assist patients, caregivers, and clinicians in navigating the various phases of cancer survivorship, further evidence-based resources are needed to optimize care. CA Cancer J Clin 2016;66:271-289. © 2016 American Cancer Society. © 2016 American Cancer Society, Inc.\n",
              "1                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Google Earth Engine is a cloud-based platform for planetary-scale geospatial analysis that brings Google's massive computational capabilities to bear on a variety of high-impact societal issues including deforestation, drought, disaster, disease, food security, water management, climate monitoring and environmental protection. It is unique in the field as an integrated platform designed to empower not only traditional remote sensing scientists, but also a much wider audience that lacks the technical capacity needed to utilize traditional supercomputers or large-scale commodity cloud computing resources. © 2017 The Author(s)\n",
              "2                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            As qualitative research becomes increasingly recognized and valued, it is imperative that it is conducted in a rigorous and methodical manner to yield meaningful and useful results. To be accepted as trustworthy, qualitative researchers must demonstrate that data analysis has been conducted in a precise, consistent, and exhaustive manner through recording, systematizing, and disclosing the methods of analysis with enough detail to enable the reader to determine whether the process is credible. Although there are numerous examples of how to conduct qualitative research, few sophisticated tools are available to researchers for conducting a rigorous and relevant thematic analysis. The purpose of this article is to guide researchers using thematic analysis as a research method. We offer personal insights and practical examples, while exploring issues of rigor and trustworthiness. The process of conducting a thematic analysis is illustrated through the presentation of an auditable decision trail, guiding interpreting and representing textual data. We detail our step-by-step approach to exploring the effectiveness of strategic clinical networks in Alberta, Canada, in our mixed methods case study. This article contributes a purposeful approach to thematic analysis in order to systematize and increase the traceability and verification of the analysis. © 2017, © The Author(s) 2017.\n",
              "3                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    We present the latest version of the Molecular Evolutionary Genetics Analysis (MEGA) software, which contains many sophisticated methods and tools for phylogenomics and phylomedicine. In this major upgrade, MEGA has been optimized for use on 64-bit computing systems for analyzing larger datasets. Researchers can now explore and analyze tens of thousands of sequences in MEGA. The new version also provides an advanced wizard for building timetrees and includes a new functionality to automatically predict gene duplication events in gene family trees. The 64-bit MEGA is made available in two interfaces: graphical and command line. The graphical user interface (GUI) is a native Microsoft Windows application that can also be used on Mac OS X. The command line MEGA is available as native applications for Windows, Linux, and Mac OS X. They are intended for use in high-throughput and scripted analysis. Both versions are available from www.megasoftware.net free of charge. © The Author 2016. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved.\n",
              "4                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Convolutional networks are powerful visual models that yield hierarchies of features. We show that convolutional networks by themselves, trained end-to-end, pixels-to-pixels, improve on the previous best result in semantic segmentation. Our key insight is to build 'fully convolutional' networks that take input of arbitrary size and produce correspondingly-sized output with efficient inference and learning. We define and detail the space of fully convolutional networks, explain their application to spatially dense prediction tasks, and draw connections to prior models. We adapt contemporary classification networks (AlexNet, the VGG net, and GoogLeNet) into fully convolutional networks and transfer their learned representations by fine-tuning to the segmentation task. We then define a skip architecture that combines semantic information from a deep, coarse layer with appearance information from a shallow, fine layer to produce accurate and detailed segmentations. Our fully convolutional networks achieve improved segmentation of PASCAL VOC (30% relative improvement to 67.2% mean IU on 2012), NYUDv2, SIFT Flow, and PASCAL-Context, while inference takes one tenth of a second for a typical image. © 1979-2012 IEEE.\n",
              "5                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Background Despite recent advances in the treatment of advanced non-small-cell lung cancer, there remains a need for effective treatments for progressive disease. We assessed the efficacy of pembrolizumab for patients with previously treated, PD-L1-positive, advanced non-small-cell lung cancer. Methods We did this randomised, open-label, phase 2/3 study at 202 academic medical centres in 24 countries. Patients with previously treated non-small-cell lung cancer with PD-L1 expression on at least 1% of tumour cells were randomly assigned (1:1:1) in blocks of six per stratum with an interactive voice-response system to receive pembrolizumab 2 mg/kg, pembrolizumab 10 mg/kg, or docetaxel 75 mg/m2 every 3 weeks. The primary endpoints were overall survival and progression-free survival both in the total population and in patients with PD-L1 expression on at least 50% of tumour cells. We used a threshold for significance of p<0·00825 (one-sided) for the analysis of overall survival and a threshold of p<0·001 for progression-free survival. This trial is registered at ClinicalTrials.gov, number NCT01905657. Findings Between Aug 28, 2013, and Feb 27, 2015, we enrolled 1034 patients: 345 allocated to pembrolizumab 2 mg/kg, 346 allocated to pembrolizumab 10 mg/kg, and 343 allocated to docetaxel. By Sept 30, 2015, 521 patients had died. In the total population, median overall survival was 10·4 months with pembrolizumab 2 mg/kg, 12·7 months with pembrolizumab 10 mg/kg, and 8·5 months with docetaxel. Overall survival was significantly longer for pembrolizumab 2 mg/kg versus docetaxel (hazard ratio [HR] 0·71, 95% CI 0·58-0·88; p=0·0008) and for pembrolizumab 10 mg/kg versus docetaxel (0·61, 0·49-0·75; p<0·0001). Median progression-free survival was 3·9 months with pembrolizumab 2 mg/kg, 4·0 months with pembrolizumab 10 mg/kg, and 4·0 months with docetaxel, with no significant difference for pembrolizumab 2 mg/kg versus docetaxel (0·88, 0·74-1·05; p=0·07) or for pembrolizumab 10 mg/kg versus docetaxel (HR 0·79, 95% CI 0·66-0·94; p=0·004). Among patients with at least 50% of tumour cells expressing PD-L1, overall survival was significantly longer with pembrolizumab 2 mg/kg than with docetaxel (median 14·9 months vs 8·2 months; HR 0·54, 95% CI 0·38-0·77; p=0·0002) and with pembrolizumab 10 mg/kg than with docetaxel (17·3 months vs 8·2 months; 0·50, 0·36-0·70; p<0·0001). Likewise, for this patient population, progression-free survival was significantly longer with pembrolizumab 2 mg/kg than with docetaxel (median 5·0 months vs 4·1 months; HR 0·59, 95% CI 0·44-0·78; p=0·0001) and with pembrolizumab 10 mg/kg than with docetaxel (5·2 months vs 4·1 months; 0·59, 0·45-0·78; p<0·0001). Grade 3-5 treatment-related adverse events were less common with pembrolizumab than with docetaxel (43 [13%] of 339 patients given 2 mg/kg, 55 [16%] of 343 given 10 mg/kg, and 109 [35%] of 309 given docetaxel). Interpretation Pembrolizumab prolongs overall survival and has a favourable benefit-to-risk profile in patients with previously treated, PD-L1-positive, advanced non-small-cell lung cancer. These data establish pembrolizumab as a new treatment option for this population and validate the use of PD-L1 selection. © 2016 Elsevier Ltd.\n",
              "6                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  IMPORTANCE Deep learning is a family of computational methods that allow an algorithm to program itself by learning from a large set of examples that demonstrate the desired behavior, removing the need to specify rules explicitly. Application of these methods to medical imaging requires further assessment and validation. OBJECTIVE To apply deep learning to create an algorithm for automated detection of diabetic retinopathy and diabetic macular edema in retinal fundus photographs. DESIGN AND SETTING A specific type of neural network optimized for image classification called a deep convolutional neural network was trained using a retrospective development data set of 128 175 retinal images, which were graded 3 to 7 times for diabetic retinopathy, diabetic macular edema, and image gradability by a panel of 54 US licensed ophthalmologists and ophthalmology senior residents between May and December 2015. The resultant algorithm was validated in January and February 2016 using 2 separate data sets, both graded by at least 7 US board-certified ophthalmologists with high intragrader consistency. EXPOSURE Deep learning-trained algorithm. MAIN OUTCOMES AND MEASURES The sensitivity and specificity of the algorithm for detecting referable diabetic retinopathy (RDR), defined as moderate and worse diabetic retinopathy, referable diabetic macular edema, or both, were generated based on the reference standard of the majority decision of the ophthalmologist panel. The algorithm was evaluated at 2 operating points selected from the development set, one selected for high specificity and another for high sensitivity. RESULTS The EyePACS-1 data set consisted of 9963 images from 4997 patients (mean age, 54.4 years; 62.2%women; prevalence ofRDR, 683/8878 fully gradable images [7.8%]); the Messidor-2 data set had 1748 images from 874 patients (mean age, 57.6 years; 42.6%women; prevalence ofRDR, 254/1745 fully gradable images [14.6%]). For detectingRDR, the algorithm had an area under the receiver operating curve of0.991 (95%CI,0.988-0.993) for EyePACS-1 and 0.990(95%CI,0.986-0.995) forMessidor-2. Using the first operating cut point with high specificity, for EyePACS-1, the sensitivitywas90.3%(95%CI, 87.5%-92.7%) and the specificity was 98.1%(95%CI, 97.8%-98.5%). ForMessidor-2, the sensitivitywas 87.0%(95%CI, 81.1%-91.0%)and the specificitywas 98.5%(95%CI, 97.7%-99.1%). Using a second operating point with high sensitivity in the development set, for EyePACS-1 the sensitivitywas 97.5%and specificitywas 93.4%and forMessidor-2 the sensitivitywas 96.1%and specificitywas 93.9%. CONCLUSIONS AND RELEVANCE In this evaluation of retinal fundus photographs from adults with diabetes, an algorithm based on deep machine learning had high sensitivity and specificity for detecting referable diabetic retinopathy. Further research is necessary to determine the feasibility of applying this algorithm in the clinical setting and to determine whether use of the algorithm could lead to improved care and outcomes compared with current ophthalmologic assessment.\n",
              "7                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Non-randomised studies of the effects of interventions are critical to many areas of healthcare evaluation, but their results may be biased. It is therefore important to understand and appraise their strengths and weaknesses. We developed ROBINS-I (\"Risk Of Bias In Non-randomised Studies-of Interventions\"), a new tool for evaluating risk of bias in estimates of the comparative effectiveness (harm or benefit) of interventions from studies that did not use randomisation to allocate units (individuals or clusters of individuals) to comparison groups. The tool will be particularly useful to those undertaking systematic reviews that include non-randomised studies.\n",
              "8                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             The number of published systematic reviews of studies of healthcare interventions has increased rapidly and these are used extensively for clinical and policy decisions. Systematic reviews are subject to a range of biases and increasingly include non-randomised studies of interventions. It is important that users can distinguish high quality reviews. Many instruments have been designed to evaluate different aspects of reviews, but there are few comprehensive critical appraisal instruments. AMSTAR was developed to evaluate systematic reviews of randomised trials. In this paper, we report on the updating of AMSTAR and its adaptation to enable more detailed assessment of systematic reviews that include randomised or non-randomised studies of healthcare interventions, or both. With moves to base more decisions on real world observational evidence we believe that AMSTAR 2 will assist decision makers in the identification of high quality systematic reviews, including those based on non-randomised studies of healthcare interventions.\n",
              "9                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      The last decade has seen a sharp increase in the number of scientific publications describing physiological and pathological functions of extracellular vesicles (EVs), a collective term covering various subtypes of cell-released, membranous structures, called exosomes, microvesicles, microparticles, ectosomes, oncosomes, apoptotic bodies, and many other names. However, specific issues arise when working with these entities, whose size and amount often make them difficult to obtain as relatively pure preparations, and to characterize properly. The International Society for Extracellular Vesicles (ISEV) proposed Minimal Information for Studies of Extracellular Vesicles (“MISEV”) guidelines for the field in 2014. We now update these “MISEV2014” guidelines based on evolution of the collective knowledge in the last four years. An important point to consider is that ascribing a specific function to EVs in general, or to subtypes of EVs, requires reporting of specific information beyond mere description of function in a crude, potentially contaminated, and heterogeneous preparation. For example, claims that exosomes are endowed with exquisite and specific activities remain difficult to support experimentally, given our still limited knowledge of their specific molecular machineries of biogenesis and release, as compared with other biophysically similar EVs. The MISEV2018 guidelines include tables and outlines of suggested protocols and steps to follow to document specific EV-associated functional activities. Finally, a checklist is provided with summaries of key points. © 2018, © 2018 The Author(s). Published by Informa UK Limited, trading as Taylor & Francis Group on behalf of The International Society for Extracellular Vesicles.\n",
              "11                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       [No abstract available]\n",
              "12                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     We introduce a new representation learning approach for domain adaptation, in which data at training and test time come from similar but different distributions. Our approach is directly inspired by the theory on domain adaptation suggesting that, for effective domain transfer to be achieved, predictions must be made based on features that cannot discriminate between the training (source) and test (target) domains. The approach implements this idea in the context of neural network architectures that are trained on labeled data from the source domain and unlabeled data from the target domain (no labeled target-domain data is necessary). As the training progresses, the approach promotes the emergence of features that are (i) discriminative for the main learning task on the source domain and (ii) indiscriminate with respect to the shift between the domains. We show that this adaptation behaviour can be achieved in almost any feed-forward model by augmenting it with few standard layers and a new gradient reversal layer. The resulting augmented architecture can be trained using standard backpropagation and stochastic gradient descent, and can thus be implemented with little effort using any of the deep learning packages. We demonstrate the success of our approach for two distinct classification problems (document sentiment analysis and image classification), where state-of-the-art domain adaptation performance on standard benchmarks is achieved. We also validate the approach for descriptor learning task in the context of person re-identification application. ©2016 Yaroslav Ganin, Evgeniya Ustinova, Hana Ajakan, Pascal Germain, Hugo Larochelle, et al.\n",
              "13                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Objective: To provide an update to “Surviving Sepsis Campaign Guidelines for Management of Sepsis and Septic Shock: 2012”. Design: A consensus committee of 55 international experts representing 25 international organizations was convened. Nominal groups were assembled at key international meetings (for those committee members attending the conference). A formal conflict-of-interest (COI) policy was developed at the onset of the process and enforced throughout. A stand-alone meeting was held for all panel members in December 2015. Teleconferences and electronic-based discussion among subgroups and among the entire committee served as an integral part of the development. Methods: The panel consisted of five sections: hemodynamics, infection, adjunctive therapies, metabolic, and ventilation. Population, intervention, comparison, and outcomes (PICO) questions were reviewed and updated as needed, and evidence profiles were generated. Each subgroup generated a list of questions, searched for best available evidence, and then followed the principles of the Grading of Recommendations Assessment, Development, and Evaluation (GRADE) system to assess the quality of evidence from high to very low, and to formulate recommendations as strong or weak, or best practice statement when applicable. Results: The Surviving Sepsis Guideline panel provided 93 statements on early management and resuscitation of patients with sepsis or septic shock. Overall, 32 were strong recommendations, 39 were weak recommendations, and 18 were best-practice statements. No recommendation was provided for four questions. Conclusions: Substantial agreement exists among a large cohort of international experts regarding many strong recommendations for the best care of patients with sepsis. Although a significant number of aspects of care have relatively weak support, evidence-based recommendations regarding the acute management of sepsis and septic shock are the foundation of improved outcomes for these critically ill patients with high mortality. © 2017, SCCM and ESICM.\n",
              "14                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             We created a new dataset of spatially interpolated monthly climate data for global land areas at a very high spatial resolution (approximately 1 km2). We included monthly temperature (minimum, maximum and average), precipitation, solar radiation, vapour pressure and wind speed, aggregated across a target temporal range of 1970–2000, using data from between 9000 and 60 000 weather stations. Weather station data were interpolated using thin-plate splines with covariates including elevation, distance to the coast and three satellite-derived covariates: maximum and minimum land surface temperature as well as cloud cover, obtained with the MODIS satellite platform. Interpolation was done for 23 regions of varying size depending on station density. Satellite data improved prediction accuracy for temperature variables 5–15% (0.07–0.17 °C), particularly for areas with a low station density, although prediction error remained high in such regions for all climate variables. Contributions of satellite covariates were mostly negligible for the other variables, although their importance varied by region. In contrast to the common approach to use a single model formulation for the entire world, we constructed the final product by selecting the best performing model for each region and variable. Global cross-validation correlations were ≥ 0.99 for temperature and humidity, 0.86 for precipitation and 0.76 for wind speed. The fact that most of our climate surface estimates were only marginally improved by use of satellite covariates highlights the importance having a dense, high-quality network of climate station data. © 2017 Royal Meteorological Society\n",
              "15                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 The Molecular Evolutionary Genetics Analysis (MEGA) software implements many analytical methods and tools for phylogenomics and phylomedicine. Here, we report a transformation of MEGA to enable cross-platform use on Microsoft Windows and Linux operating systems. MEGA X does not require virtualization or emulation software and provides a uniform user experience across platforms. MEGA X has additionally been upgraded to use multiple computing cores for many molecular evolutionary analyses. MEGA X is available in two interfaces (graphical and command line) and can be downloaded from www.megasoftware.net free of charge. © The Author(s) 2018. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved.\n",
              "16                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    Each year, the American Cancer Society estimates the numbers of new cancer cases and deaths that will occur in the United States and compiles the most recent data on cancer incidence, mortality, and survival. Incidence data, available through 2014, were collected by the Surveillance, Epidemiology, and End Results Program; the National Program of Cancer Registries; and the North American Association of Central Cancer Registries. Mortality data, available through 2015, were collected by the National Center for Health Statistics. In 2018, 1,735,350 new cancer cases and 609,640 cancer deaths are projected to occur in the United States. Over the past decade of data, the cancer incidence rate (2005-2014) was stable in women and declined by approximately 2% annually in men, while the cancer death rate (2006-2015) declined by about 1.5% annually in both men and women. The combined cancer death rate dropped continuously from 1991 to 2015 by a total of 26%, translating to approximately 2,378,600 fewer cancer deaths than would have been expected if death rates had remained at their peak. Of the 10 leading causes of death, only cancer declined from 2014 to 2015. In 2015, the cancer death rate was 14% higher in non-Hispanic blacks (NHBs) than non-Hispanic whites (NHWs) overall (death rate ratio [DRR], 1.14; 95% confidence interval [95% CI], 1.13-1.15), but the racial disparity was much larger for individuals aged <65 years (DRR, 1.31; 95% CI, 1.29-1.32) compared with those aged ≥65 years (DRR, 1.07; 95% CI, 1.06-1.09) and varied substantially by state. For example, the cancer death rate was lower in NHBs than NHWs in Massachusetts for all ages and in New York for individuals aged ≥65 years, whereas for those aged <65 years, it was 3 times higher in NHBs in the District of Columbia (DRR, 2.89; 95% CI, 2.16-3.91) and about 50% higher in Wisconsin (DRR, 1.78; 95% CI, 1.56-2.02), Kansas (DRR, 1.51; 95% CI, 1.25-1.81), Louisiana (DRR, 1.49; 95% CI, 1.38-1.60), Illinois (DRR, 1.48; 95% CI, 1.39-1.57), and California (DRR, 1.45; 95% CI, 1.38-1.54). Larger racial inequalities in young and middle-aged adults probably partly reflect less access to high-quality health care. CA Cancer J Clin 2018;68:7-30. © 2018 American Cancer Society. © 2018 American Cancer Society\n",
              "17                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          BACKGROUND: Experimental and clinical data suggest that reducing inflammation without affecting lipid levels may reduce the risk of cardiovascular disease. Yet, the inflammatory hypothesis of atherothrombosis has remained unproved. METHODS: We conducted a randomized, double-blind trial of canakinumab, a therapeutic monoclonal antibody targeting interleukin-1β, involving 10,061 patients with previous myocardial infarction and a high-sensitivity C-reactive protein level of 2 mg or more per liter. The trial compared three doses of canakinumab (50 mg, 150 mg, and 300 mg, administered subcutaneously every 3 months) with placebo. The primary efficacy end point was nonfatal myocardial infarction, nonfatal stroke, or cardiovascular death. RESULTS: At 48 months, the median reduction from baseline in the high-sensitivity C-reactive protein level was 26 percentage points greater in the group that received the 50-mg dose of canakinumab, 37 percentage points greater in the 150-mg group, and 41 percentage points greater in the 300-mg group than in the placebo group. Canakinumab did not reduce lipid levels from baseline. At a median follow-up of 3.7 years, the incidence rate for the primary end point was 4.50 events per 100 person-years in the placebo group, 4.11 events per 100 person-years in the 50-mg group, 3.86 events per 100 person-years in the 150-mg group, and 3.90 events per 100 person-years in the 300-mg group. The hazard ratios as compared with placebo were as follows: in the 50-mg group, 0.93 (95% confidence interval [CI], 0.80 to 1.07; P=0.30); in the 150-mg group, 0.85 (95% CI, 0.74 to 0.98; P=0.021); and in the 300-mg group, 0.86 (95% CI, 0.75 to 0.99; P=0.031). The 150-mg dose, but not the other doses, met the prespecified multiplicity-adjusted threshold for statistical significance for the primary end point and the secondary end point that additionally included hospitalization for unstable angina that led to urgent revascularization (hazard ratio vs. placebo, 0.83; 95% CI, 0.73 to 0.95; P=0.005). Canakinumab was associated with a higher incidence of fatal infection than was placebo. There was no significant difference in all-cause mortality (hazard ratio for all canakinumab doses vs. placebo, 0.94; 95% CI, 0.83 to 1.06; P=0.31). CONCLUSIONS: Antiinflammatory therapy targeting the interleukin-1β innate immunity pathway with canakinumab at a dose of 150 mg every 3 months led to a significantly lower rate of recurrent cardiovascular events than placebo, independent of lipid-level lowering. Copyright © 2017 Massachusetts Medical Society.\n",
              "18                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       This article describes several features in the MAFFT online service for multiple sequence alignment (MSA). As a result of recent advances in sequencing technologies, huge numbers of biological sequences are available and the need for MSAs with large numbers of sequences is increasing. To extract biologically relevant information from such data, sophistication of algorithms is necessary but not sufficient. Intuitive and interactive tools for experimental biologists to semiautomatically handle large data are becoming important. We are working on development of MAFFT toward these two directions. Here, we explain (i) the Web interface for recently developed options for large data and (ii) interactive usage to refine sequence data sets and MSAs. © 2017 The Author. Published by Oxford University Press.\n",
              "20                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       On September 14, 2015 at 09:50:45 UTC the two detectors of the Laser Interferometer Gravitational-Wave Observatory simultaneously observed a transient gravitational-wave signal. The signal sweeps upwards in frequency from 35 to 250 Hz with a peak gravitational-wave strain of 1.0×10-21. It matches the waveform predicted by general relativity for the inspiral and merger of a pair of black holes and the ringdown of the resulting single black hole. The signal was observed with a matched-filter signal-to-noise ratio of 24 and a false alarm rate estimated to be less than 1 event per 203 000 years, equivalent to a significance greater than 5.1σ. The source lies at a luminosity distance of 410-180+160 Mpc corresponding to a redshift z=0.09-0.04+0.03. In the source frame, the initial black hole masses are 36-4+5M⊙ and 29-4+4M⊙, and the final black hole mass is 62-4+4M⊙, with 3.0-0.5+0.5M⊙c2 radiated in gravitational waves. All uncertainties define 90% credible intervals. These observations demonstrate the existence of binary stellar-mass black hole systems. This is the first direct detection of gravitational waves and the first observation of a binary black hole merger. © 2016 authors. Published by the American Physical Society.\n",
              "21                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        The use of bibliometrics is gradually extending to all disciplines. It is particularly suitable for science mapping at a time when the emphasis on empirical contributions is producing voluminous, fragmented, and controversial research streams. Science mapping is complex and unwieldly because it is multi-step and frequently requires numerous and diverse software tools, which are not all necessarily freeware. Although automated workflows that integrate these software tools into an organized data flow are emerging, in this paper we propose a unique open-source tool, designed by the authors, called bibliometrix, for performing comprehensive science mapping analysis. bibliometrix supports a recommended workflow to perform bibliometric analyses. As it is programmed in R, the proposed tool is flexible and can be rapidly upgraded and integrated with other statistical R-packages. It is therefore useful in a constantly changing science such as bibliometrics. © 2017 Elsevier Ltd\n",
              "22                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        KEGG (http://www.kegg.jp/or http://www.genome.jp/kegg/) is an encyclopedia of genes and genomes. Assigning functional meanings to genes and genomes both at the molecular and higher levels is the primary objective of the KEGG database project. Molecularlevel functions are stored in the KO (KEGG Orthology) database, where each KO is defined as a functional ortholog of genes and proteins. Higher-level functions are represented by networks of molecular interactions, reactions and relations in the forms of KEGG pathway maps, BRITE hierarchies and KEGG modules. In the past the KO database was developed for the purpose of defining nodes of molecular networks, but now the content has been expanded and the quality improved irrespective of whether or not the KOs appear in the three molecular network databases. The newly introduced addendum category of the GENES database is a collection of individual proteins whose functions are experimentally characterized and from which an increasing number of KOs are defined. Furthermore, the DISEASE and DRUG databases have been improved by systematic analysis of drug labels for better integration of diseases and drugs with the KEGG molecular networks. KEGG is moving towards becoming a comprehensive knowledge base for both functional interpretation and practical application of genomic information. © 2016 The Author(s).\n",
              "23                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 The formation of a dense and uniform thin layer on the substrates is crucial for the fabrication of high-performance perovskite solar cells (PSCs) containing formamidinium with multiple cations and mixed halide anions. The concentration of defect states, which reduce a cell's performance by decreasing the open-circuit voltage and short-circuit current density, needs to be as low as possible. We show that the introduction of additional iodide ions into the organic cation solution, which are used to form the perovskite layers through an intramolecular exchanging process, decreases the concentration of deep-level defects. The defect-engineered thin perovskite layers enable the fabrication of PSCs with a certified power conversion efficiency of 22.1% in small cells and 19.7% in 1-square-centimeter cells. © 2017, American Association for the Advancement of Science. All rights reserved.\n",
              "24                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               With increasing incidence and mortality, cancer is the leading cause of death in China and is a major public health problem. Because of China's massive population (1.37 billion), previous national incidence and mortality estimates have been limited to small samples of the population using data from the 1990s or based on a specific year. With high-quality data from an additional number of population-based registries now available through the National Central Cancer Registry of China, the authors analyzed data from 72 local, population-based cancer registries (2009-2011), representing 6.5% of the population, to estimate the number of new cases and cancer deaths for 2015. Data from 22 registries were used for trend analyses (2000-2011). The results indicated that an estimated 4292,000 new cancer cases and 2814,000 cancer deaths would occur in China in 2015, with lung cancer being the most common incident cancer and the leading cause of cancer death. Stomach, esophageal, and liver cancers were also commonly diagnosed and were identified as leading causes of cancer death. Residents of rural areas had significantly higher age-standardized (Segi population) incidence and mortality rates for all cancers combined than urban residents (213.6 per 100,000 vs 191.5 per 100,000 for incidence; 149.0 per 100,000 vs 109.5 per 100,000 for mortality, respectively). For all cancers combined, the incidence rates were stable during 2000 through 2011 for males (+0.2% per year; P =.1), whereas they increased significantly (+2.2% per year; P <.05) among females. In contrast, the mortality rates since 2006 have decreased significantly for both males (-1.4% per year; P <.05) and females (-1.1% per year; P <.05). Many of the estimated cancer cases and deaths can be prevented through reducing the prevalence of risk factors, while increasing the effectiveness of clinical care delivery, particularly for those living in rural areas and in disadvantaged populations. © 2016 American Cancer Society.\n",
              "25                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              To be effective as a drug, a potent molecule must reach its target in the body in sufficient concentration, and stay there in a bioactive form long enough for the expected biologic events to occur. Drug development involves assessment of absorption, distribution, metabolism and excretion (ADME) increasingly earlier in the discovery process, at a stage when considered compounds are numerous but access to the physical samples is limited. In that context, computer models constitute valid alternatives to experiments. Here, we present the new SwissADME web tool that gives free access to a pool of fast yet robust predictive models for physicochemical properties, pharmacokinetics, drug-likeness and medicinal chemistry friendliness, among which in-house proficient methods such as the BOILED-Egg, iLOGP and Bioavailability Radar. Easy efficient input and interpretation are ensured thanks to a user-friendly interface through the login-free website http://www.swissadme.ch. Specialists, but also nonexpert in cheminformatics or computational chemistry can predict rapidly key parameters for a collection of molecules to support their drug discovery endeavours. © The Author(s) 2017.\n",
              "26                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      Background Exposure to ambient air pollution increases morbidity and mortality, and is a leading contributor to global disease burden. We explored spatial and temporal trends in mortality and burden of disease attributable to ambient air pollution from 1990 to 2015 at global, regional, and country levels. Methods We estimated global population-weighted mean concentrations of particle mass with aerodynamic diameter less than 2·5 μm (PM2·5) and ozone at an approximate 11 km × 11 km resolution with satellite-based estimates, chemical transport models, and ground-level measurements. Using integrated exposure–response functions for each cause of death, we estimated the relative risk of mortality from ischaemic heart disease, cerebrovascular disease, chronic obstructive pulmonary disease, lung cancer, and lower respiratory infections from epidemiological studies using non-linear exposure–response functions spanning the global range of exposure. Findings Ambient PM2·5 was the fifth-ranking mortality risk factor in 2015. Exposure to PM2·5 caused 4·2 million (95% uncertainty interval [UI] 3·7 million to 4·8 million) deaths and 103·1 million (90·8 million 115·1 million) disability-adjusted life-years (DALYs) in 2015, representing 7·6% of total global deaths and 4·2% of global DALYs, 59% of these in east and south Asia. Deaths attributable to ambient PM2·5 increased from 3·5 million (95% UI 3·0 million to 4·0 million) in 1990 to 4·2 million (3·7 million to 4·8 million) in 2015. Exposure to ozone caused an additional 254 000 (95% UI 97 000–422 000) deaths and a loss of 4·1 million (1·6 million to 6·8 million) DALYs from chronic obstructive pulmonary disease in 2015. Interpretation Ambient air pollution contributed substantially to the global burden of disease in 2015, which increased over the past 25 years, due to population ageing, changes in non-communicable disease rates, and increasing air pollution in low-income and middle-income countries. Modest reductions in burden will occur in the most polluted countries unless PM2·5 values are decreased substantially, but there is potential for substantial health benefits from exposure reduction. Funding Bill & Melinda Gates Foundation and Health Effects Institute. © 2017 The Author(s). Published by Elsevier Ltd. This is an Open Access article under the CC BY 4.0 license\n",
              "27                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      The recent advent of DNA sequencing technologies facilitates the use of genome sequencing data that provide means for more informative and precise classification and identification of members of the Bacteria and Archaea. Because the current species definition is based on the comparison of genome sequences between type and other strains in a given species, building a genome database with correct taxonomic information is of paramount need to enhance our efforts in exploring prokaryotic diversity and discovering novel species as well as for routine identifications. Here we introduce an integrated database, called EzBioCloud, that holds the taxonomic hierarchy of the Bacteria and Archaea, which is represented by quality-controlled 16S rRNA gene and genome sequences. Whole-genome assemblies in the NCBI Assembly Database were screened for low quality and subjected to a composite identification bioinformatics pipeline that employs gene-based searches followed by the calculation of average nucleotide identity. As a result, the database is made of 61 700 species/ phylotypes, including 13 132 with validly published names, and 62 362 whole-genome assemblies that were identified taxonomically at the genus, species and subspecies levels. Genomic properties, such as genome size and DNA G+C content, and the occurrence in human microbiome data were calculated for each genus or higher taxa. This united database of taxonomy, 16S rRNA gene and genome sequences, with accompanying bioinformatics tools, should accelerate genome based classification and identification of members of the Bacteria and Archaea. © 2017 IUMS.\n",
              "28                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             We propose a deep learning method for single image super-resolution (SR). Our method directly learns an end-to-end mapping between the low/high-resolution images. The mapping is represented as a deep convolutional neural network (CNN) that takes the low-resolution image as the input and outputs the high-resolution one. We further show that traditional sparse-coding-based SR methods can also be viewed as a deep convolutional network. But unlike traditional methods that handle each component separately, our method jointly optimizes all layers. Our deep CNN has a lightweight structure, yet demonstrates state-of-the-art restoration quality, and achieves fast speed for practical on-line usage. We explore different network structures and parameter settings to achieve trade-offs between performance and speed. Moreover, we extend our network to cope with three color channels simultaneously, and show better overall reconstruction quality. © 1979-2012 IEEE.\n",
              "30                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Motivation: Recent advances in sequencing technologies promise ultra-long reads of ~100 kb in average, full-length mRNA or cDNA reads in high throughput and genomic contigs over 100 Mb in length. Existing alignment programs are unable or inefficient to process such data at scale, which presses for the development of new alignment algorithms. Results: Minimap2 is a general-purpose alignment program to map DNA or long mRNA sequences against a large reference database. It works with accurate short reads of ≥100 bp in length, ≥1 kb genomic reads at error rate ~15%, full-length noisy Direct RNA or cDNA reads and assembly contigs or closely related full chromosomes of hundreds of megabases in length. Minimap2 does split-read alignment, employs concave gap cost for long insertions and deletions and introduces new heuristics to reduce spurious alignments. It is 3–4 times as fast as mainstream short-read mappers at comparable accuracy, and is ≥30 times faster than long-read genomic or cDNA mappers at higher accuracy, surpassing most aligners specialized in one type of alignment. © The Author(s) 2018. Published by Oxford University Press. All rights reserved.\n",
              "31                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Nonalcoholic fatty liver disease (NAFLD) is a major cause of liver disease worldwide. We estimated the global prevalence, incidence, progression, and outcomes of NAFLD and nonalcoholic steatohepatitis (NASH). PubMed/MEDLINE were searched from 1989 to 2015 for terms involving epidemiology and progression of NAFLD. Exclusions included selected groups (studies that exclusively enrolled morbidly obese or diabetics or pediatric) and no data on alcohol consumption or other liver diseases. Incidence of hepatocellular carcinoma (HCC), cirrhosis, overall mortality, and liver-related mortality were determined. NASH required histological diagnosis. All studies were reviewed by three independent investigators. Analysis was stratified by region, diagnostic technique, biopsy indication, and study population. We used random-effects models to provide point estimates (95% confidence interval [CI]) of prevalence, incidence, mortality and incidence rate ratios, and metaregression with subgroup analysis to account for heterogeneity. Of 729 studies, 86 were included with a sample size of 8,515,431 from 22 countries. Global prevalence of NAFLD is 25.24% (95% CI: 22.10-28.65) with highest prevalence in the Middle East and South America and lowest in Africa. Metabolic comorbidities associated with NAFLD included obesity (51.34%; 95% CI: 41.38-61.20), type 2 diabetes (22.51%; 95% CI: 17.92-27.89), hyperlipidemia (69.16%; 95% CI: 49.91-83.46%), hypertension (39.34%; 95% CI: 33.15-45.88), and metabolic syndrome (42.54%; 95% CI: 30.06-56.05). Fibrosis progression proportion, and mean annual rate of progression in NASH were 40.76% (95% CI: 34.69-47.13) and 0.09 (95% CI: 0.06-0.12). HCC incidence among NAFLD patients was 0.44 per 1,000 person-years (range, 0.29-0.66). Liver-specific mortality and overall mortality among NAFLD and NASH were 0.77 per 1,000 (range, 0.33-1.77) and 11.77 per 1,000 person-years (range, 7.10-19.53) and 15.44 per 1,000 (range, 11.72-20.34) and 25.56 per 1,000 person-years (range, 6.29-103.80). Incidence risk ratios for liver-specific and overall mortality for NAFLD were 1.94 (range, 1.28-2.92) and 1.05 (range, 0.70-1.56). Conclusions: As the global epidemic of obesity fuels metabolic conditions, the clinical and economic burden of NAFLD will become enormous. (Hepatology 2016;64:73–84). © 2015 by the American Association for the Study of Liver Diseases.\n",
              "32                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Cronbach’s alpha is a statistic commonly quoted by authors to demonstrate that tests and scales that have been constructed or adopted for research projects are fit for purpose. Cronbach’s alpha is regularly adopted in studies in science education: it was referred to in 69 different papers published in 4 leading science education journals in a single year (2015)—usually as a measure of reliability. This article explores how this statistic is used in reporting science education research and what it represents. Authors often cite alpha values with little commentary to explain why they feel this statistic is relevant and seldom interpret the result for readers beyond citing an arbitrary threshold for an acceptable value. Those authors who do offer readers qualitative descriptors interpreting alpha values adopt a diverse and seemingly arbitrary terminology. More seriously, illustrative examples from the science education literature demonstrate that alpha may be acceptable even when there are recognised problems with the scales concerned. Alpha is also sometimes inappropriately used to claim an instrument is unidimensional. It is argued that a high value of alpha offers limited evidence of the reliability of a research instrument, and that indeed a very high value may actually be undesirable when developing a test of scientific knowledge or understanding. Guidance is offered to authors reporting, and readers evaluating, studies that present Cronbach’s alpha statistic as evidence of instrument quality. © 2016, The Author(s).\n",
              "34                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           Background VSEARCH is an open source and free of charge multithreaded 64-bit tool for processing and preparing metagenomics, genomics and population genomics nucleotide sequence data. It is designed as an alternative to the widely used USEARCH tool (Edgar, 2010) for which the source code is not publicly available, algorithm details are only rudimentarily described, and only a memory-confined 32-bit version is freely available for academic use. Methods When searching nucleotide sequences, VSEARCH uses a fast heuristic based on words shared by the query and target sequences in order to quickly identify similar sequences, a similar strategy is probably used in USEARCH. VSEARCH then performs optimal global sequence alignment of the query against potential target sequences, using full dynamic programming instead of the seed-and-extend heuristic used by USEARCH. Pairwise alignments are computed in parallel using vectorisation and multiple threads. Results VSEARCH includes most commands for analysing nucleotide sequences available in USEARCH version 7 and several of those available in USEARCH version 8, including searching (exact or based on global alignment), clustering by similarity (using length pre-sorting, abundance pre-sorting or a user-defined order), chimera detection (reference-based or de novo), dereplication (full length or prefix), pairwise alignment, reverse complementation, sorting, and subsampling. VSEARCH also includes commands for FASTQ file processing, i.e., format detection, filtering, read quality statistics, and merging of paired reads. Furthermore, VSEARCH extends functionality with several new commands and improvements, including shuffling, rereplication, masking of low-complexity sequences with the well-known DUST algorithm, a choice among different similarity definitions, and FASTQ file format conversion. VSEARCH is here shown to be more accurate than USEARCH when performing searching, clustering, chimera detection and subsampling, while on a par with USEARCH for paired-ends read merging. VSEARCH is slower than USEARCH when performing clustering and chimera detection, but significantly faster when performing paired-end reads merging and dereplication. VSEARCH is available at https://github.com/torognes/vsearch under either the BSD 2-clause license or the GNU General Public License version 3.0. Discussion VSEARCH has been shown to be a fast, accurate and full-fledged alternative to USEARCH. A free and open-source versatile tool for sequence analysis is now available to the metagenomics community. © 2016 Rognes et al.\n",
              "35                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             We present ORB-SLAM2, a complete simultaneous localization and mapping (SLAM) system for monocular, stereo and RGB-D cameras, including map reuse, loop closing, and relocalization capabilities. The system works in real time on standard central processing units in a wide variety of environments from small hand-held indoors sequences, to drones flying in industrial environments and cars driving around a city. Our back-end, based on bundle adjustment with monocular and stereo observations, allows for accurate trajectory estimation with metric scale. Our system includes a lightweight localization mode that leverages visual odometry tracks for unmapped regions and matches with map points that allow for zero-drift localization. The evaluation on 29 popular public sequences shows that our method achieves state-of-the-art accuracy, being in most cases the most accurate SLAM solution. We publish the source code, not only for the benefit of the SLAM community, but with the aim of being an out-of-the-box SLAM solution for researchers in other fields. © 2004-2012 IEEE.\n",
              "36                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                A system-wide understanding of cellular function requires knowledge of all functional interactions between the expressed proteins. The STRING database aims to collect and integrate this information, by consolidating known and predicted protein-protein association data for a large number of organisms. The associations in STRING include direct (physical) interactions, as well as indirect (functional) interactions, as long as both are specific and biologically meaningful. Apart from collecting and reassessing available experimental data on protein-protein interactions, and importing known pathways and protein complexes from curated databases, interaction predictions are derived from the following sources: (i) systematic co-expression analysis, (ii) detection of shared selective signals across genomes, (iii) automated text-mining of the scientific literature and (iv) computational transfer of interaction knowledge between organisms based on gene orthology. In the latest version 10.5 of STRING, the biggest changes are concerned with data dissemination: the web frontend has been completely redesigned to reduce dependency on outdated browser technologies, and the database can now also be queried from inside the popular Cytoscape software framework. Further improvements include automated background analysis of user inputs for functional enrichments, and streamlined download options. The STRING resource is available online, at http://string-db.org/. © The Author(s) 2016.\n",
              "37                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     The brms package implements Bayesian multilevel models in R using the probabilistic programming language Stan. A wide range of distributions and link functions are supported, allowing users to fit – among others – linear, robust linear, binomial, Poisson, survival, ordinal, zero-inflated, hurdle, and even non-linear models all in a multilevel context. Further modeling options include autocorrelation of the response variable, user defined covariance structures, censored data, as well as meta-analytic standard errors. Prior specifications are flexible and explicitly encourage users to apply prior distributions that actually reflect their beliefs. In addition, model fit can easily be assessed and compared with the Watanabe-Akaike information criterion and leave-one-out cross-validation. © 2017, American Statistical Association. All rights reserved.\n",
              "38                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              By coordinating the design and distribution of global climate model simulations of the past, current, and future climate, the Coupled Model Intercomparison Project (CMIP) has become one of the foundational elements of climate science. However, the need to address an ever-expanding range of scientific questions arising from more and more research communities has made it necessary to revise the organization of CMIP. After a long and wide community consultation, a new and more federated structure has been put in place. It consists of three major elements: (1) a handful of common experiments, the DECK (Diagnostic, Evaluation and Characterization of Klima) and CMIP historical simulations (1850-near present) that will maintain continuity and help document basic characteristics of models across different phases of CMIP; (2) common standards, coordination, infrastructure, and documentation that will facilitate the distribution of model outputs and the characterization of the model ensemble; and (3) an ensemble of CMIP-Endorsed Model Intercomparison Projects (MIPs) that will be specific to a particular phase of CMIP (now CMIP6) and that will build on the DECK and CMIP historical simulations to address a large range of specific questions and fill the scientific gaps of the previous CMIP phases. The DECK and CMIP historical simulations, together with the use of CMIP data standards, will be the entry cards for models participating in CMIP. Participation in CMIP6-Endorsed MIPs by individual modelling groups will be at their own discretion and will depend on their scientific interests and priorities. With the Grand Science Challenges of the World Climate Research Programme (WCRP) as its scientific backdrop, CMIP6 will address three broad questions: How does the Earth system respond to forcing What are the origins and consequences of systematic model biases? How can we assess future climate changes given internal climate variability, predictability, and uncertainties in scenarios? This CMIP6 overview paper presents the background and rationale for the new structure of CMIP, provides a detailed description of the DECK and CMIP6 historical simulations, and includes a brief introduction to the 21 CMIP6-Endorsed MIPs. © 2016 Author(s).\n",
              "40                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Today's best perovskite solar cells use a mixture of formamidinium and methylammonium as the monovalent cations. With the addition of inorganic cesium, the resulting triple cation perovskite compositions are thermally more stable, contain less phase impurities and are less sensitive to processing conditions. This enables more reproducible device performances to reach a stabilized power output of 21.1% and ∼18% after 250 hours under operational conditions. These properties are key for the industrialization of perovskite photovoltaics. © 2016 The Royal Society of Chemistry.\n",
              "41                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Least-squares means are predictions from a linear model, or averages thereof. They are useful in the analysis of experimental data for summarizing the effects of factors, and for testing linear contrasts among predictions. The lsmeans package (Lenth 2016) provides a simple way of obtaining least-squares means and contrasts thereof. It supports many models fitted by R (R Core Team 2015) core packages (as well as a few key contributed ones) that fit linear or mixed models, and provides a simple way of extending it to cover more model classes. © 2016, American Statistical Association. All rights reserved.\n",
              "43                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            Quantum ESPRESSO is an integrated suite of open-source computer codes for quantum simulations of materials using state-of-the-art electronic-structure techniques, based on density-functional theory, density-functional perturbation theory, and many-body perturbation theory, within the plane-wave pseudopotential and projector-augmented-wave approaches. Quantum ESPRESSO owes its popularity to the wide variety of properties and processes it allows to simulate, to its performance on an increasingly broad array of hardware architectures, and to a community of researchers that rely on its capabilities as a core open-source development platform to implement their ideas. In this paper we describe recent extensions and improvements, covering new methodologies and property calculators, improved parallelization, code modularization, and extended interoperability both within the distribution and with external software. © 2017 IOP Publishing Ltd.\n",
              "44                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Bayesian inference of phylogeny using Markov chain Monte Carlo (MCMC) plays a central role in understanding evolutionary history from molecular sequence data. Visualizing and analyzing the MCMC-generated samples from the posterior distribution is a key step in any non-trivial Bayesian inference. We present the software package Tracer (version 1.7) for visualizing and analyzing the MCMC trace files generated through Bayesian phylogenetic inference. Tracer provides kernel density estimation, multivariate visualization, demographic trajectory reconstruction, conditional posterior distribution summary, and more. © The Author(s) 2018. Published by Oxford University Press, on behalf of the Society of Systematic Biologists.\n",
              "45                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               This paper presents cosmological results based on full-mission Planck observations of temperature and polarization anisotropies of the cosmic microwave background (CMB) radiation. Our results are in very good agreement with the 2013 analysis of the Planck nominal-mission temperature data, but with increased precision. The temperature and polarization power spectra are consistent with the standard spatially-flat 6-parameter ΛCDM cosmology with a power-law spectrum of adiabatic scalar perturbations (denoted \"base ΛCDM\" in this paper). From the Planck temperature data combined with Planck lensing, for this cosmology we find a Hubble constant, H0 = (67.8 ± 0.9) km s-1Mpc-1, a matter density parameter Ωm = 0.308 ± 0.012, and a tilted scalar spectral index with ns = 0.968 ± 0.006, consistent with the 2013 analysis. Note that in this abstract we quote 68% confidence limits on measured parameters and 95% upper limits on other parameters. We present the first results of polarization measurements with the Low Frequency Instrument at large angular scales. Combined with the Planck temperature and lensing data, these measurements give a reionization optical depth of τ = 0.066 ± 0.016, corresponding to a reionization redshift of \\hbox{$z-{\\rm re}=8.8{+1.7}-{-1.4}$}. These results are consistent with those from WMAP polarization measurements cleaned for dust emission using 353-GHz polarization maps from the High Frequency Instrument. We find no evidence for any departure from base ΛCDM in the neutrino sector of the theory; for example, combining Planck observations with other astrophysical data we find Neff = 3.15 ± 0.23 for the effective number of relativistic degrees of freedom, consistent with the value Neff = 3.046 of the Standard Model of particle physics. The sum of neutrino masses is constrained to â'mν < 0.23 eV. The spatial curvature of our Universe is found to be very close to zero, with | ΩK | < 0.005. Adding a tensor component as a single-parameter extension to base ΛCDM we find an upper limit on the tensor-to-scalar ratio of r0.002< 0.11, consistent with the Planck 2013 results and consistent with the B-mode polarization constraints from a joint analysis of BICEP2, Keck Array, and Planck (BKP) data. Adding the BKP B-mode data to our analysis leads to a tighter constraint of r0.002 < 0.09 and disfavours inflationarymodels with a V(φ) φ2 potential. The addition of Planck polarization data leads to strong constraints on deviations from a purely adiabatic spectrum of fluctuations. We find no evidence for any contribution from isocurvature perturbations or from cosmic defects. Combining Planck data with other astrophysical data, including Type Ia supernovae, the equation of state of dark energy is constrained to w =-1.006 ± 0.045, consistent with the expected value for a cosmological constant. The standard big bang nucleosynthesis predictions for the helium and deuterium abundances for the best-fit Planck base ΛCDM cosmology are in excellent agreement with observations. We also constraints on annihilating dark matter and on possible deviations from the standard recombination history. In neither case do we find no evidence for new physics. The Planck results for base ΛCDM are in good agreement with baryon acoustic oscillation data and with the JLA sample of Type Ia supernovae. However, as in the 2013 analysis, the amplitude of the fluctuation spectrum is found to be higher than inferred from some analyses of rich cluster counts and weak gravitational lensing. We show that these tensions cannot easily be resolved with simple modifications of the base ΛCDM cosmology. Apart from these tensions, the base ΛCDM cosmology provides an excellent description of the Planck CMB observations and many other astrophysical data sets. © 2016 ESO.\n",
              "46                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               KEGG (http://www.kegg.jp/ or http://www.genome.jp/kegg/) is an integrated database resource for biological interpretation of genome sequences and other high-throughput data. Molecular functions of genes and proteins are associated with ortholog groups and stored in the KEGG Orthology (KO) database. The KEGG pathway maps, BRITE hierarchies and KEGG modules are developed as networks of KO nodes, representing high-level functions of the cell and the organism. Currently, more than 4000 complete genomes are annotated with KOs in the KEGG GENES database, which can be used as a reference data set for KO assignment and subsequent reconstruction of KEGG pathways and other molecular networks. As an annotation resource, the following improvements have been made. First, each KO record is re-examined and associated with protein sequence data used in experiments of functional characterization. Second, the GENES database now includes viruses, plasmids, and the addendum category for functionally characterized proteins that are not represented in complete genomes. Third, new automatic annotation servers, BlastKOALA and GhostKOALA, are made available utilizing the nonredundant pangenome data set generated from the GENES database. As a resource for translational bioinformatics, various data sets are created for antimicrobial resistance and drug interaction networks. © The Author(s) 2015.\n",
              "47                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           The Modern-Era Retrospective Analysis for Research and Applications, version 2 (MERRA-2), is the latest atmospheric reanalysis of the modern satellite era produced by NASA's Global Modeling and Assimilation Office (GMAO). MERRA-2 assimilates observation types not available to its predecessor, MERRA, and includes updates to the Goddard Earth Observing System (GEOS) model and analysis scheme so as to provide a viable ongoing climate analysis beyond MERRA's terminus. While addressing known limitations of MERRA, MERRA-2 is also intended to be a development milestone for a future integrated Earth system analysis (IESA) currently under development at GMAO. This paper provides an overview of the MERRA-2 system and various performance metrics. Among the advances in MERRA-2 relevant to IESA are the assimilation of aerosol observations, several improvements to the representation of the stratosphere including ozone, and improved representations of cryospheric processes. Other improvements in the quality of MERRA-2 compared with MERRA include the reduction of some spurious trends and jumps related to changes in the observing system and reduced biases and imbalances in aspects of the water cycle. Remaining deficiencies are also identified. Production of MERRA-2 began in June 2014 in four processing streams and converged to a single near-real-time stream in mid-2015. MERRA-2 products are accessible online through the NASA Goddard Earth Sciences Data Information Services Center (GES DISC). © 2017 American Meteorological Society.\n",
              "48    Background Non-fatal outcomes of disease and injury increasingly detract from the ability of the world's population to live in full health, a trend largely attributable to an epidemiological transition in many countries from causes affecting children, to non-communicable diseases (NCDs) more common in adults. For the Global Burden of Diseases, Injuries, and Risk Factors Study 2015 (GBD 2015), we estimated the incidence, prevalence, and years lived with disability for diseases and injuries at the global, regional, and national scale over the period of 1990 to 2015. Methods We estimated incidence and prevalence by age, sex, cause, year, and geography with a wide range of updated and standardised analytical procedures. Improvements from GBD 2013 included the addition of new data sources, updates to literature reviews for 85 causes, and the identification and inclusion of additional studies published up to November, 2015, to expand the database used for estimation of non-fatal outcomes to 60 900 unique data sources. Prevalence and incidence by cause and sequelae were determined with DisMod-MR 2.1, an improved version of the DisMod-MR Bayesian meta-regression tool first developed for GBD 2010 and GBD 2013. For some causes, we used alternative modelling strategies where the complexity of the disease was not suited to DisMod-MR 2.1 or where incidence and prevalence needed to be determined from other data. For GBD 2015 we created a summary indicator that combines measures of income per capita, educational attainment, and fertility (the Socio-demographic Index [SDI]) and used it to compare observed patterns of health loss to the expected pattern for countries or locations with similar SDI scores. Findings We generated 9·3 billion estimates from the various combinations of prevalence, incidence, and YLDs for causes, sequelae, and impairments by age, sex, geography, and year. In 2015, two causes had acute incidences in excess of 1 billion: upper respiratory infections (17·2 billion, 95% uncertainty interval [UI] 15·4–19·2 billion) and diarrhoeal diseases (2·39 billion, 2·30–2·50 billion). Eight causes of chronic disease and injury each affected more than 10% of the world's population in 2015: permanent caries, tension-type headache, iron-deficiency anaemia, age-related and other hearing loss, migraine, genital herpes, refraction and accommodation disorders, and ascariasis. The impairment that affected the greatest number of people in 2015 was anaemia, with 2·36 billion (2·35–2·37 billion) individuals affected. The second and third leading impairments by number of individuals affected were hearing loss and vision loss, respectively. Between 2005 and 2015, there was little change in the leading causes of years lived with disability (YLDs) on a global basis. NCDs accounted for 18 of the leading 20 causes of age-standardised YLDs on a global scale. Where rates were decreasing, the rate of decrease for YLDs was slower than that of years of life lost (YLLs) for nearly every cause included in our analysis. For low SDI geographies, Group 1 causes typically accounted for 20–30% of total disability, largely attributable to nutritional deficiencies, malaria, neglected tropical diseases, HIV/AIDS, and tuberculosis. Lower back and neck pain was the leading global cause of disability in 2015 in most countries. The leading cause was sense organ disorders in 22 countries in Asia and Africa and one in central Latin America; diabetes in four countries in Oceania; HIV/AIDS in three southern sub-Saharan African countries; collective violence and legal intervention in two north African and Middle Eastern countries; iron-deficiency anaemia in Somalia and Venezuela; depression in Uganda; onchoceriasis in Liberia; and other neglected tropical diseases in the Democratic Republic of the Congo. Interpretation Ageing of the world's population is increasing the number of people living with sequelae of diseases and injuries. Shifts in the epidemiological profile driven by socioeconomic change also contribute to the continued increase in years lived with disability (YLDs) as well as the rate of increase in YLDs. Despite limitations imposed by gaps in data availability and the variable quality of the data available, the standardised and comprehensive approach of the GBD study provides opportunities to examine broad trends, compare those trends between countries or subnational geographies, benchmark against locations at similar stages of development, and gauge the strength or weakness of the estimates available. Funding Bill & Melinda Gates Foundation. © 2016 The Author(s). Published by Elsevier Ltd. This is an Open Access article under the CC BY license\n",
              "49                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           Background: Scoping reviews are a relatively new approach to evidence synthesis and currently there exists little guidance regarding the decision to choose between a systematic review or scoping review approach when synthesising evidence. The purpose of this article is to clearly describe the differences in indications between scoping reviews and systematic reviews and to provide guidance for when a scoping review is (and is not) appropriate. Results: Researchers may conduct scoping reviews instead of systematic reviews where the purpose of the review is to identify knowledge gaps, scope a body of literature, clarify concepts or to investigate research conduct. While useful in their own right, scoping reviews may also be helpful precursors to systematic reviews and can be used to confirm the relevance of inclusion criteria and potential questions. Conclusions: Scoping reviews are a useful tool in the ever increasing arsenal of evidence synthesis approaches. Although conducted for different purposes compared to systematic reviews, scoping reviews still require rigorous and transparent methods in their conduct to ensure that the results are trustworthy. Our hope is that with clear guidance available regarding whether to conduct a scoping review or a systematic review, there will be less scoping reviews being performed for inappropriate indications better served by a systematic review, and vice-versa. © 2018 The Author(s).\n",
              "50                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      We present a novel and practical deep fully convolutional neural network architecture for semantic pixel-wise segmentation termed SegNet. This core trainable segmentation engine consists of an encoder network, a corresponding decoder network followed by a pixel-wise classification layer. The architecture of the encoder network is topologically identical to the 13 convolutional layers in the VGG16 network [1]. The role of the decoder network is to map the low resolution encoder feature maps to full input resolution feature maps for pixel-wise classification. The novelty of SegNet lies is in the manner in which the decoder upsamples its lower resolution input feature map(s). Specifically, the decoder uses pooling indices computed in the max-pooling step of the corresponding encoder to perform non-linear upsampling. This eliminates the need for learning to upsample. The upsampled maps are sparse and are then convolved with trainable filters to produce dense feature maps. We compare our proposed architecture with the widely adopted FCN [2] and also with the well known DeepLab-LargeFOV [3] , DeconvNet [4] architectures. This comparison reveals the memory versus accuracy trade-off involved in achieving good segmentation performance. SegNet was primarily motivated by scene understanding applications. Hence, it is designed to be efficient both in terms of memory and computational time during inference. It is also significantly smaller in the number of trainable parameters than other competing architectures and can be trained end-to-end using stochastic gradient descent. We also performed a controlled benchmark of SegNet and other architectures on both road scenes and SUN RGB-D indoor scene segmentation tasks. These quantitative assessments show that SegNet provides good performance with competitive inference time and most efficient inference memory-wise as compared to other architectures. We also provide a Caffe implementation of SegNet and a web demo at http://mi.eng.cam.ac.uk/projects/segnet/. © 1979-2012 IEEE.\n",
              "51                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions. There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018. In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality. Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality). Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality. The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors. It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries. The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts. CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society. © 2018 American Cancer Society\n",
              "52                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              State-of-the-art object detection networks depend on region proposal algorithms to hypothesize object locations. Advances like SPPnet [1] and Fast R-CNN [2] have reduced the running time of these detection networks, exposing region proposal computation as a bottleneck. In this work, we introduce a Region Proposal Network (RPN) that shares full-image convolutional features with the detection network, thus enabling nearly cost-free region proposals. An RPN is a fully convolutional network that simultaneously predicts object bounds and objectness scores at each position. The RPN is trained end-to-end to generate high-quality region proposals, which are used by Fast R-CNN for detection. We further merge RPN and Fast R-CNN into a single network by sharing their convolutional features - using the recently popular terminology of neural networks with 'attention' mechanisms, the RPN component tells the unified network where to look. For the very deep VGG-16 model [3], our detection system has a frame rate of 5 fps (including all steps) on a GPU, while achieving state-of-the-art object detection accuracy on PASCAL VOC 2007, 2012, and MS COCO datasets with only 300 proposals per image. In ILSVRC and COCO 2015 competitions, Faster R-CNN and RPN are the foundations of the 1st-place winning entries in several tracks. Code has been made publicly available. © 2016 IEEE.\n",
              "53                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     Proteins and their functional interactions form the backbone of the cellular machinery. Their connectivity network needs to be considered for the full understanding of biological phenomena, but the available information on protein-protein associations is incomplete and exhibits varying levels of annotation granularity and reliability. The STRING database aims to collect, score and integrate all publicly available sources of protein-protein interaction information, and to complement these with computational predictions. Its goal is to achieve a comprehensive and objective global network, including direct (physical) as well as indirect (functional) interactions. The latest version of STRING (11.0) more than doubles the number of organisms it covers, to 5090. The most important new feature is an option to upload entire, genome-wide datasets as input, allowing users to visualize subsets as interaction networks and to perform gene-set enrichment analysis on the entire input. For the enrichment analysis, STRING implements well-known classification systems such as Gene Ontology and KEGG, but also offers additional, new classification systems based on high-throughput text-mining as well as on a hierarchical clustering of the association network itself. The STRING resource is available online at https://string-db.org/. © 2018 The Author(s).\n",
              "54                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Aims: To provide global estimates of diabetes prevalence for 2019 and projections for 2030 and 2045. Methods: A total of 255 high-quality data sources, published between 1990 and 2018 and representing 138 countries were identified. For countries without high quality in-country data, estimates were extrapolated from similar countries matched by economy, ethnicity, geography and language. Logistic regression was used to generate smoothed age-specific diabetes prevalence estimates (including previously undiagnosed diabetes) in adults aged 20–79 years. Results: The global diabetes prevalence in 2019 is estimated to be 9.3% (463 million people), rising to 10.2% (578 million) by 2030 and 10.9% (700 million) by 2045. The prevalence is higher in urban (10.8%) than rural (7.2%) areas, and in high-income (10.4%) than low-income countries (4.0%). One in two (50.1%) people living with diabetes do not know that they have diabetes. The global prevalence of impaired glucose tolerance is estimated to be 7.5% (374 million) in 2019 and projected to reach 8.0% (454 million) by 2030 and 8.6% (548 million) by 2045. Conclusions: Just under half a billion people are living with diabetes worldwide and the number is projected to increase by 25% in 2030 and 51% in 2045. © 2019\n",
              "55                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Single-cell transcriptomics has transformed our ability to characterize cell states, but deep biological understanding requires more than a taxonomic listing of clusters. As new methods arise to measure distinct cellular modalities, a key analytical challenge is to integrate these datasets to better understand cellular identity and function. Here, we develop a strategy to “anchor” diverse datasets together, enabling us to integrate single-cell measurements not only across scRNA-seq technologies, but also across different modalities. After demonstrating improvement over existing methods for integrating scRNA-seq data, we anchor scRNA-seq experiments with scATAC-seq to explore chromatin differences in closely related interneuron subsets and project protein expression measurements onto a bone marrow atlas to characterize lymphocyte populations. Lastly, we harmonize in situ gene expression and scRNA-seq datasets, allowing transcriptome-wide imputation of spatial gene expression patterns. Our work presents a strategy for the assembly of harmonized references and transfer of information across datasets. © 2019 Elsevier Inc.; A computational approach to integrate diverse modalities associated with single-cell sequencing datasets can be used to better understand cellular identity and function. © 2019 Elsevier Inc.\n",
              "Name: abstract, dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 151
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#Removing the last sentences of the abstract to clean the text\n",
        "def remove_last_two_sentences(abstract):\n",
        "    sentences = nltk.sent_tokenize(abstract)\n",
        "    if len(sentences) > 1:\n",
        "        abstract = ' '.join(sentences[:-2])\n",
        "    return abstract"
      ],
      "metadata": {
        "id": "S6IYfqcdC8gX"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.iloc[55]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "YeG0AVSAN78u",
        "outputId": "1323f145-6aa3-4021-b87f-d2baa9014908"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "title                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         Physics-informed neural networks: A deep learning framework for solving forward and inverse problems involving nonlinear partial differential equations\n",
              "Year                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             2019\n",
              "journal                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              Journal of Computational Physics\n",
              "citations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        4658\n",
              "Link                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057501376&doi=10.1016%2fj.jcp.2018.10.045&partnerID=40&md5=ef98293ff9dd4d802adac77f84213d80\n",
              "abstract     We introduce physics-informed neural networks – neural networks that are trained to solve supervised learning tasks while respecting any given laws of physics described by general nonlinear partial differential equations. In this work, we present our developments in the context of solving two main classes of problems: data-driven solution and data-driven discovery of partial differential equations. Depending on the nature and arrangement of the available data, we devise two distinct types of algorithms, namely continuous time and discrete time models. The first type of models forms a new family of data-efficient spatio-temporal function approximators, while the latter type allows the use of arbitrarily accurate implicit Runge–Kutta time stepping schemes with unlimited number of stages. The effectiveness of the proposed framework is demonstrated through a collection of classical problems in fluids, quantum mechanics, reaction–diffusion systems, and the propagation of nonlinear shallow-water waves. © 2018 Elsevier Inc.\n",
              "Name: 62, dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 153
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all['abstract'] = scopus_all['abstract'].apply(remove_last_two_sentences)"
      ],
      "metadata": {
        "id": "vxfmM5riEx5h"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.iloc[55]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HAH2uumQOMxZ",
        "outputId": "2cc2c088-1264-4b62-90bb-8e627f5a9e57"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "title                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Physics-informed neural networks: A deep learning framework for solving forward and inverse problems involving nonlinear partial differential equations\n",
              "Year                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 2019\n",
              "journal                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  Journal of Computational Physics\n",
              "citations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            4658\n",
              "Link                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    https://www.scopus.com/inward/record.uri?eid=2-s2.0-85057501376&doi=10.1016%2fj.jcp.2018.10.045&partnerID=40&md5=ef98293ff9dd4d802adac77f84213d80\n",
              "abstract     We introduce physics-informed neural networks – neural networks that are trained to solve supervised learning tasks while respecting any given laws of physics described by general nonlinear partial differential equations. In this work, we present our developments in the context of solving two main classes of problems: data-driven solution and data-driven discovery of partial differential equations. Depending on the nature and arrangement of the available data, we devise two distinct types of algorithms, namely continuous time and discrete time models. The first type of models forms a new family of data-efficient spatio-temporal function approximators, while the latter type allows the use of arbitrarily accurate implicit Runge–Kutta time stepping schemes with unlimited number of stages.\n",
              "Name: 62, dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 156
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all['abstract_len'] = scopus_all['abstract'].apply(len)\n",
        "scopus_all['abstract_len'].hist(bins=100)"
      ],
      "metadata": {
        "id": "xz0rZgwDs9hP",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 447
        },
        "outputId": "49cc416c-b70f-4376-947a-e3dec9bffab9"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<Axes: >"
            ]
          },
          "metadata": {},
          "execution_count": 157
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 640x480 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAjcAAAGdCAYAAADuR1K7AAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjcuMSwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy/bCgiHAAAACXBIWXMAAA9hAAAPYQGoP6dpAAAnTUlEQVR4nO3df3RU5Z3H8U9+TCaJMIkBkyE1gfgTEIQUBFJZl0pIgBysytktNrXocmBlQ1dMi0JVCNo2LPXYrpwo654WuqdEKucoLj8KRFAoNYCkIgQ4VBCLFRK2ZJMA0WEgz/5hmXVIAiTMzE2evF/nzJF775PnPvc7+fHxuffOjTLGGAEAAFgi2ukBAAAAhBLhBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABglVinB9ARzc3NOn78uHr27KmoqCinhwMAAK6CMUanT59Wenq6oqPDN7/SJcPN8ePHlZGR4fQwAABAB3z66ae68cYbw9Z/lww3PXv2lPRlcTweT0j79vv92rRpk/Ly8uRyuULaN1pHzSOPmjuDukceNXdGW3VvbGxURkZG4O94uHTJcHPxVJTH4wlLuElMTJTH4+EHIUKoeeRRc2dQ98ij5s64Ut3DfUkJFxQDAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWCXW6QEgNPrNXRe0/MmiAodGAgCAs5i5AQAAVmHmpgtgVgYAgKvHzA0AALAK4QYAAFiFcAMAAKzSrnBTWlqqu+66Sz179lRqaqruv/9+HTp0KKjNmDFjFBUVFfR67LHHgtocO3ZMBQUFSkxMVGpqqubMmaPz589f+9EAAIBur10XFG/dulVFRUW66667dP78ef3oRz9SXl6eDhw4oOuuuy7Qbvr06XruuecCy4mJiYF/X7hwQQUFBfJ6vXrvvfd04sQJfe9735PL5dJPf/rTEBwSAADoztoVbjZs2BC0vHz5cqWmpqqqqkr33HNPYH1iYqK8Xm+rfWzatEkHDhzQ22+/rbS0NA0dOlTPP/+8nnrqKZWUlCguLq4DhwEAAPCla7oVvKGhQZKUkpIStH7FihX6zW9+I6/Xq0mTJunZZ58NzN5UVlZq8ODBSktLC7TPz8/XzJkztX//fmVnZ7fYj8/nk8/nCyw3NjZKkvx+v/x+/7UcQgsX+wt1v9fCHWOCllsb29W06aw6Y81tR82dQd0jj5o7o626R+p9iDLGmCs3a6m5uVn33Xef6uvrtX379sD6V199VX379lV6err27t2rp556SiNGjNAbb7whSZoxY4b+/Oc/a+PGjYGvaWpq0nXXXaf169drwoQJLfZVUlKihQsXtlhfXl4edMoLAAB0Xk1NTfrOd76jhoYGeTyesO2nwzM3RUVFqq6uDgo20pfh5aLBgwerT58+Gjt2rI4cOaKbb765Q/uaN2+eiouLA8uNjY3KyMhQXl5eyIvj9/tVUVGhcePGyeVyhbTvjhpUsjFoubokv0NtOqvOWHPbUXNnUPfIo+bOaKvuF8+8hFuHws2sWbO0du1abdu2TTfeeONl244cOVKSdPjwYd18883yer3atWtXUJva2lpJavM6HbfbLbfb3WK9y+UK2zdrOPtuL9+FqKDl1sZ1NW06u85U8+6CmjuDukceNXfGpXWP1HvQrlvBjTGaNWuW3nzzTW3ZskVZWVlX/Jo9e/ZIkvr06SNJysnJ0b59+3Ty5MlAm4qKCnk8Hg0cOLA9wwEAAGihXTM3RUVFKi8v11tvvaWePXuqpqZGkpSUlKSEhAQdOXJE5eXlmjhxonr16qW9e/fqiSee0D333KM777xTkpSXl6eBAwfq4Ycf1uLFi1VTU6NnnnlGRUVFrc7OAAAAtEe7Zm5eeeUVNTQ0aMyYMerTp0/g9dvf/laSFBcXp7ffflt5eXnq37+/fvCDH2jy5Mlas2ZNoI+YmBitXbtWMTExysnJ0Xe/+11973vfC/pcHAAAgI5q18zNlW6sysjI0NatW6/YT9++fbV+/fr27BoAAOCq8GwpAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGCVDj0VHJ1fv7nrWqz7ZFGBAyMBACCymLkBAABWIdwAAACrcFqqG7n0VBWnqQAANmLmBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGCVWKcHgGD95q4LSRsAALorZm4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALBKu8JNaWmp7rrrLvXs2VOpqam6//77dejQoaA2X3zxhYqKitSrVy/16NFDkydPVm1tbVCbY8eOqaCgQImJiUpNTdWcOXN0/vz5az8aAADQ7bUr3GzdulVFRUXasWOHKioq5Pf7lZeXp7NnzwbaPPHEE1qzZo1WrVqlrVu36vjx43rwwQcD2y9cuKCCggKdO3dO7733nn79619r+fLlmj9/fuiOCgAAdFux7Wm8YcOGoOXly5crNTVVVVVVuueee9TQ0KBf/vKXKi8v17333itJWrZsmQYMGKAdO3Zo1KhR2rRpkw4cOKC3335baWlpGjp0qJ5//nk99dRTKikpUVxcXOiODpfVb+66Fus+WVTgwEgAAAiddoWbSzU0NEiSUlJSJElVVVXy+/3Kzc0NtOnfv78yMzNVWVmpUaNGqbKyUoMHD1ZaWlqgTX5+vmbOnKn9+/crOzu7xX58Pp98Pl9gubGxUZLk9/vl9/uv5RBauNhfqPu9Wu4Y48h+L3LiuJ2ueXdEzZ1B3SOPmjujrbpH6n3ocLhpbm7W7Nmzdffdd2vQoEGSpJqaGsXFxSk5OTmobVpammpqagJtvhpsLm6/uK01paWlWrhwYYv1mzZtUmJiYkcP4bIqKirC0u+VLB7hyG4D1q9f79i+nap5d0bNnUHdI4+aO+PSujc1NUVkvx0ON0VFRaqurtb27dtDOZ5WzZs3T8XFxYHlxsZGZWRkKC8vTx6PJ6T78vv9qqio0Lhx4+RyuULa99UYVLIx4vv8quqS/Ijv0+mad0fU3BnUPfKouTPaqvvFMy/h1qFwM2vWLK1du1bbtm3TjTfeGFjv9Xp17tw51dfXB83e1NbWyuv1Btrs2rUrqL+Ld1NdbHMpt9stt9vdYr3L5QrbN2s4+74c34WoiO/zq5z84Xeq5t0ZNXcGdY88au6MS+seqfegXXdLGWM0a9Ysvfnmm9qyZYuysrKCtg8bNkwul0ubN28OrDt06JCOHTumnJwcSVJOTo727dunkydPBtpUVFTI4/Fo4MCB13IsAAAA7Zu5KSoqUnl5ud566y317NkzcI1MUlKSEhISlJSUpGnTpqm4uFgpKSnyeDz6/ve/r5ycHI0aNUqSlJeXp4EDB+rhhx/W4sWLVVNTo2eeeUZFRUWtzs4AAAC0R7vCzSuvvCJJGjNmTND6ZcuW6ZFHHpEk/fznP1d0dLQmT54sn8+n/Px8vfzyy4G2MTExWrt2rWbOnKmcnBxdd911mjp1qp577rlrOxIAAAC1M9wYc+XblOPj41VWVqaysrI22/Tt29fRu3LQtks/+4bPvQEAdDU8WwoAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsEqHnwqO0Lj0Q/MAAMC1YeYGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAViHcAAAAq8Q6PQB0bv3mrmux7pNFBQ6MBACAq8PMDQAAsArhBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFVinR5Ad9Jv7jqnhwAAgPWYuQEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYJV2h5tt27Zp0qRJSk9PV1RUlFavXh20/ZFHHlFUVFTQa/z48UFt6urqVFhYKI/Ho+TkZE2bNk1nzpy5pgMBAACQOhBuzp49qyFDhqisrKzNNuPHj9eJEycCr9deey1oe2Fhofbv36+KigqtXbtW27Zt04wZM9o/egAAgEu0+3NuJkyYoAkTJly2jdvtltfrbXXbwYMHtWHDBr3//vsaPny4JGnJkiWaOHGiXnjhBaWnp7d3SAAAAAFh+RC/d999V6mpqbr++ut177336sc//rF69eolSaqsrFRycnIg2EhSbm6uoqOjtXPnTj3wwAMt+vP5fPL5fIHlxsZGSZLf75ff7w/p2C/2F+p+JckdY0LepxO6Us3ROmruDOoeedTcGW3VPVLvQ8jDzfjx4/Xggw8qKytLR44c0Y9+9CNNmDBBlZWViomJUU1NjVJTU4MHERurlJQU1dTUtNpnaWmpFi5c2GL9pk2blJiYGOpDkCRVVFSEvM/FI0LepSPWr18fln7DUXNcHjV3BnWPPGrujEvr3tTUFJH9hjzcTJkyJfDvwYMH684779TNN9+sd999V2PHju1Qn/PmzVNxcXFgubGxURkZGcrLy5PH47nmMX+V3+9XRUWFxo0bJ5fLFdK+B5VsDGl/TqkuyQ9pf+GsOVpHzZ1B3SOPmjujrbpfPPMSbmF/ttRNN92k3r176/Dhwxo7dqy8Xq9OnjwZ1Ob8+fOqq6tr8zodt9stt9vdYr3L5QrbN2s4+vZdiAppf07pSjXH5VFzZ1D3yKPmzri07pF6D8L+OTd/+ctfdOrUKfXp00eSlJOTo/r6elVVVQXabNmyRc3NzRo5cmS4hwMAACzX7pmbM2fO6PDhw4Hlo0ePas+ePUpJSVFKSooWLlyoyZMny+v16siRI3ryySd1yy23KD//y1MZAwYM0Pjx4zV9+nQtXbpUfr9fs2bN0pQpU7hTCgAAXLN2z9zs3r1b2dnZys7OliQVFxcrOztb8+fPV0xMjPbu3av77rtPt912m6ZNm6Zhw4bp97//fdBppRUrVqh///4aO3asJk6cqNGjR+vVV18N3VEBAIBuq90zN2PGjJExbd/SvHHjlS+aTUlJUXl5eXt3DQAAcEU8WwoAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKwS6/QA0PX0m7suaPmTRQUOjQQAgJaYuQEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsAq3guOaXXpruMTt4QAA5zBzAwAArEK4AQAAViHcAAAAqxBuAACAVbigOIxau9AWAACEFzM3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYJV2h5tt27Zp0qRJSk9PV1RUlFavXh203Rij+fPnq0+fPkpISFBubq4++uijoDZ1dXUqLCyUx+NRcnKypk2bpjNnzlzTgQAAAEgdCDdnz57VkCFDVFZW1ur2xYsX66WXXtLSpUu1c+dOXXfddcrPz9cXX3wRaFNYWKj9+/eroqJCa9eu1bZt2zRjxoyOHwUAAMDfxLb3CyZMmKAJEya0us0Yo1/84hd65pln9K1vfUuS9F//9V9KS0vT6tWrNWXKFB08eFAbNmzQ+++/r+HDh0uSlixZookTJ+qFF15Qenr6NRwOAADo7todbi7n6NGjqqmpUW5ubmBdUlKSRo4cqcrKSk2ZMkWVlZVKTk4OBBtJys3NVXR0tHbu3KkHHnigRb8+n08+ny+w3NjYKEny+/3y+/2hPIRAf6Ho1x1jrrmPrqo99QtlzXF1qLkzqHvkUXNntFX3SL0PIQ03NTU1kqS0tLSg9WlpaYFtNTU1Sk1NDR5EbKxSUlICbS5VWlqqhQsXtli/adMmJSYmhmLoLVRUVFxzH4tHhGAgXdT69evb/TWhqDnah5o7g7pHHjV3xqV1b2pqish+QxpuwmXevHkqLi4OLDc2NiojI0N5eXnyeDwh3Zff71dFRYXGjRsnl8t1TX0NKtkYolF1PdUl+VfdNpQ1x9Wh5s6g7pFHzZ3RVt0vnnkJt5CGG6/XK0mqra1Vnz59Autra2s1dOjQQJuTJ08Gfd358+dVV1cX+PpLud1uud3uFutdLlfYvllD0bfvQlSIRtP1dKR24Xw/0Tpq7gzqHnnU3BmX1j1S70FIP+cmKytLXq9XmzdvDqxrbGzUzp07lZOTI0nKyclRfX29qqqqAm22bNmi5uZmjRw5MpTDAQAA3VC7Z27OnDmjw4cPB5aPHj2qPXv2KCUlRZmZmZo9e7Z+/OMf69Zbb1VWVpaeffZZpaen6/7775ckDRgwQOPHj9f06dO1dOlS+f1+zZo1S1OmTOFOKYv0m7suaPmTRQUOjQQA0N20O9zs3r1b3/zmNwPLF6+FmTp1qpYvX64nn3xSZ8+e1YwZM1RfX6/Ro0drw4YNio+PD3zNihUrNGvWLI0dO1bR0dGaPHmyXnrppRAcDgAA6O7aHW7GjBkjY9q+xTkqKkrPPfecnnvuuTbbpKSkqLy8vL27BgAAuCKeLQUAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrxDo9AFv0m7vO6SEAAAAxcwMAACxDuAEAAFYh3AAAAKsQbgAAgFUINwAAwCrcLYWIaO1usk8WFTgwEgCA7Zi5AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFcINAACwCuEGAABYhXADAACsQrgBAABWIdwAAACrEG4AAIBVCDcAAMAqhBsAAGAVwg0AALAK4QYAAFgl1ukBoPvqN3edJMkdY7R4hMODAQBYg5kbAABgFcINAACwCuEGAABYhXADAACsQrgBAABW4W4pdBqDSjbKdyEqaN0niwocGg0AoKti5gYAAFiFcAMAAKxCuAEAAFYh3AAAAKtwQXEHXXx0AAAA6FyYuQEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYJWQh5uSkhJFRUUFvfr37x/Y/sUXX6ioqEi9evVSjx49NHnyZNXW1oZ6GAAAoJsKy8zNHXfcoRMnTgRe27dvD2x74okntGbNGq1atUpbt27V8ePH9eCDD4ZjGAAAoBsKy+fcxMbGyuv1tljf0NCgX/7ylyovL9e9994rSVq2bJkGDBigHTt2aNSoUeEYDgAA6EbCEm4++ugjpaenKz4+Xjk5OSotLVVmZqaqqqrk9/uVm5sbaNu/f39lZmaqsrKyzXDj8/nk8/kCy42NjZIkv98vv98f0rFf7O9K/bpjTEj32525o03Qf78q1O8vvnS13+cILeoeedTcGW3VPVLvQ5QxJqR/pX/3u9/pzJkzuv3223XixAktXLhQn332maqrq7VmzRo9+uijQUFFkkaMGKFvfvOb+rd/+7dW+ywpKdHChQtbrC8vL1diYmIohw8AAMKkqalJ3/nOd9TQ0CCPxxO2/YQ83Fyqvr5effv21YsvvqiEhIQOhZvWZm4yMjL017/+NeTF8fv9qqio0Lhx4+RyudpsN6hkY0j32525o42eH96sZ3dHy9ccFbStuiTfoVHZ7Wq/zxFa1D3yqLkz2qp7Y2OjevfuHfZwE/ZnSyUnJ+u2227T4cOHNW7cOJ07d0719fVKTk4OtKmtrW31Gp2L3G633G53i/Uulyts36xX6tt3IarNbegYX3NUi7ryyyi8wvkzhLZR98ij5s64tO6Reg/C/jk3Z86c0ZEjR9SnTx8NGzZMLpdLmzdvDmw/dOiQjh07ppycnHAPBQAAdAMhn7n54Q9/qEmTJqlv3746fvy4FixYoJiYGD300ENKSkrStGnTVFxcrJSUFHk8Hn3/+99XTk4Od0rhqrT2NPZPFhU4MBIAQGcV8nDzl7/8RQ899JBOnTqlG264QaNHj9aOHTt0ww03SJJ+/vOfKzo6WpMnT5bP51N+fr5efvnlUA8DAAB0UyEPNytXrrzs9vj4eJWVlamsrCzUuwYAAODZUgAAwC6EGwAAYBXCDQAAsArhBgAAWCXsH+IHXIvWbv0GAOBymLkBAABWIdwAAACrEG4AAIBVCDcAAMAqXFAM6/D8KQDo3pi5AQAAViHcAAAAqxBuAACAVQg3AADAKoQbAABgFe6Wugo8AgAAgK6DmRsAAGAVwg0AALAK4QYAAFiFa27Q5XFNFADgq5i5AQAAViHcAAAAqxBuAACAVbjmBt0STw4HAHsxcwMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAVuHxC20YVLJRvgtRTg8DDuIRDQDQNRFu0C20FlQAAHbitBQAALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCp8zg3ggEs/d4cPBwSA0GHmBgAAWIVwAwAArMJpKeBveEQDANiBmRsAAGAVZm6AEONiYQBwFjM3AADAKszcANeA63QAoPNh5gYAAFiFmRugHToyU8PsDgBEFuEG6KS4MBkAOoZwA3QCzO4AQOhwzQ0AALAKMzdAF9Ha7E6oTlVdzcwRp8UAdBWOhpuysjL97Gc/U01NjYYMGaIlS5ZoxIgRTg4J6FI6ejrro+fzQrIvAg+Azsix01K//e1vVVxcrAULFuiPf/yjhgwZovz8fJ08edKpIQEAAAs4NnPz4osvavr06Xr00UclSUuXLtW6dev0q1/9SnPnznVqWEC3MKhkoxaP+PK/vgtRHe7nau7o4q4vAJHmSLg5d+6cqqqqNG/evMC66Oho5ebmqrKyskV7n88nn88XWG5oaJAk1dXVye/3h3Rsfr9fTU1NivVH60Jzx3/p4+rFNhs1NTVT8wgKV81v+eHrLfd1yfKpU6datBlZujloeee8sR1qczUu7ac1He37SvtyRxs9k92soU+/Id/f6n41+wrXmFvr99J+rqZNZ3bxd/qpU6fkcrmcHk6nFer3ua26nz59WpJkjOlw31fFOOCzzz4zksx7770XtH7OnDlmxIgRLdovWLDASOLFixcvXrx4WfD69NNPw5ozusTdUvPmzVNxcXFgubm5WXV1derVq5eiokL7f/qNjY3KyMjQp59+Ko/HE9K+0TpqHnnU3BnUPfKouTPaqrsxRqdPn1Z6enpY9+9IuOndu7diYmJUW1sbtL62tlZer7dFe7fbLbfbHbQuOTk5nEOUx+PhByHCqHnkUXNnUPfIo+bOaK3uSUlJYd+vI3dLxcXFadiwYdq8+f/P8TU3N2vz5s3KyclxYkgAAMASjp2WKi4u1tSpUzV8+HCNGDFCv/jFL3T27NnA3VMAAAAd4Vi4+fa3v63/+Z//0fz581VTU6OhQ4dqw4YNSktLc2pIkr48BbZgwYIWp8EQPtQ88qi5M6h75FFzZzhd9yhjwn0/FgAAQOTw4EwAAGAVwg0AALAK4QYAAFiFcAMAAKxCuPmKsrIy9evXT/Hx8Ro5cqR27drl9JC6jNLSUt11113q2bOnUlNTdf/99+vQoUNBbb744gsVFRWpV69e6tGjhyZPntzigxyPHTumgoICJSYmKjU1VXPmzNH58+eD2rz77rv6+te/LrfbrVtuuUXLly8P9+F1CYsWLVJUVJRmz54dWEfNQ++zzz7Td7/7XfXq1UsJCQkaPHiwdu/eHdhujNH8+fPVp08fJSQkKDc3Vx999FFQH3V1dSosLJTH41FycrKmTZumM2fOBLXZu3ev/u7v/k7x8fHKyMjQ4sWLI3J8ndGFCxf07LPPKisrSwkJCbr55pv1/PPPBz2fiLpfm23btmnSpElKT09XVFSUVq9eHbQ9kvVdtWqV+vfvr/j4eA0ePFjr169v/wGF9eEOXcjKlStNXFyc+dWvfmX2799vpk+fbpKTk01tba3TQ+sS8vPzzbJly0x1dbXZs2ePmThxosnMzDRnzpwJtHnsscdMRkaG2bx5s9m9e7cZNWqU+cY3vhHYfv78eTNo0CCTm5trPvjgA7N+/XrTu3dvM2/evECbjz/+2CQmJpri4mJz4MABs2TJEhMTE2M2bNgQ0ePtbHbt2mX69etn7rzzTvP4448H1lPz0KqrqzN9+/Y1jzzyiNm5c6f5+OOPzcaNG83hw4cDbRYtWmSSkpLM6tWrzYcffmjuu+8+k5WVZT7//PNAm/Hjx5shQ4aYHTt2mN///vfmlltuMQ899FBge0NDg0lLSzOFhYWmurravPbaayYhIcH8x3/8R0SPt7P4yU9+Ynr16mXWrl1rjh49alatWmV69Ohh/v3f/z3Qhrpfm/Xr15unn37avPHGG0aSefPNN4O2R6q+f/jDH0xMTIxZvHixOXDggHnmmWeMy+Uy+/bta9fxEG7+ZsSIEaaoqCiwfOHCBZOenm5KS0sdHFXXdfLkSSPJbN261RhjTH19vXG5XGbVqlWBNgcPHjSSTGVlpTHmyx+u6OhoU1NTE2jzyiuvGI/HY3w+nzHGmCeffNLccccdQfv69re/bfLz88N9SJ3W6dOnza233moqKirM3//93wfCDTUPvaeeesqMHj26ze3Nzc3G6/Wan/3sZ4F19fX1xu12m9dee80YY8yBAweMJPP+++8H2vzud78zUVFR5rPPPjPGGPPyyy+b66+/PvAeXNz37bffHupD6hIKCgrMP/3TPwWte/DBB01hYaExhrqH2qXhJpL1/cd//EdTUFAQNJ6RI0eaf/7nf27XMXBaStK5c+dUVVWl3NzcwLro6Gjl5uaqsrLSwZF1XQ0NDZKklJQUSVJVVZX8fn9Qjfv376/MzMxAjSsrKzV48OCgD3LMz89XY2Oj9u/fH2jz1T4utunO71NRUZEKCgpa1IWah95///d/a/jw4fqHf/gHpaamKjs7W//5n/8Z2H706FHV1NQE1SspKUkjR44MqnlycrKGDx8eaJObm6vo6Gjt3Lkz0Oaee+5RXFxcoE1+fr4OHTqk//3f/w33YXY63/jGN7R582b96U9/kiR9+OGH2r59uyZMmCCJuodbJOsbqt83hBtJf/3rX3XhwoUWn46clpammpoah0bVdTU3N2v27Nm6++67NWjQIElSTU2N4uLiWjzw9Ks1rqmpafU9uLjtcm0aGxv1+eefh+NwOrWVK1fqj3/8o0pLS1tso+ah9/HHH+uVV17Rrbfeqo0bN2rmzJn613/9V/3617+W9P81u9zvkpqaGqWmpgZtj42NVUpKSrvel+5k7ty5mjJlivr37y+Xy6Xs7GzNnj1bhYWFkqh7uEWyvm21aW/9HXv8AuxVVFSk6upqbd++3emhWO3TTz/V448/roqKCsXHxzs9nG6hublZw4cP109/+lNJUnZ2tqqrq7V06VJNnTrV4dHZ6/XXX9eKFStUXl6uO+64Q3v27NHs2bOVnp5O3dEqZm4k9e7dWzExMS3uIqmtrZXX63VoVF3TrFmztHbtWr3zzju68cYbA+u9Xq/OnTun+vr6oPZfrbHX6231Pbi47XJtPB6PEhISQn04nVpVVZVOnjypr3/964qNjVVsbKy2bt2ql156SbGxsUpLS6PmIdanTx8NHDgwaN2AAQN07NgxSf9fs8v9LvF6vTp58mTQ9vPnz6uurq5d70t3MmfOnMDszeDBg/Xwww/riSeeCMxYUvfwimR922rT3voTbiTFxcVp2LBh2rx5c2Bdc3OzNm/erJycHAdH1nUYYzRr1iy9+eab2rJli7KysoK2Dxs2TC6XK6jGhw4d0rFjxwI1zsnJ0b59+4J+QCoqKuTxeAJ/UHJycoL6uNimO75PY8eO1b59+7Rnz57Aa/jw4SosLAz8m5qH1t13393iIw7+9Kc/qW/fvpKkrKwseb3eoHo1NjZq586dQTWvr69XVVVVoM2WLVvU3NyskSNHBtps27ZNfr8/0KaiokK33367rr/++rAdX2fV1NSk6OjgP1cxMTFqbm6WRN3DLZL1Ddnvm3ZdfmyxlStXGrfbbZYvX24OHDhgZsyYYZKTk4PuIkHbZs6caZKSksy7775rTpw4EXg1NTUF2jz22GMmMzPTbNmyxezevdvk5OSYnJycwPaLtyXn5eWZPXv2mA0bNpgbbrih1duS58yZYw4ePGjKysq67W3Jrfnq3VLGUPNQ27Vrl4mNjTU/+clPzEcffWRWrFhhEhMTzW9+85tAm0WLFpnk5GTz1ltvmb1795pvfetbrd4ym52dbXbu3Gm2b99ubr311qBbZuvr601aWpp5+OGHTXV1tVm5cqVJTEzsFrckt2bq1Knma1/7WuBW8DfeeMP07t3bPPnkk4E21P3anD592nzwwQfmgw8+MJLMiy++aD744APz5z//2RgTufr+4Q9/MLGxseaFF14wBw8eNAsWLOBW8Gu1ZMkSk5mZaeLi4syIESPMjh07nB5SlyGp1deyZcsCbT7//HPzL//yL+b66683iYmJ5oEHHjAnTpwI6ueTTz4xEyZMMAkJCaZ3797mBz/4gfH7/UFt3nnnHTN06FATFxdnbrrppqB9dHeXhhtqHnpr1qwxgwYNMm632/Tv39+8+uqrQdubm5vNs88+a9LS0ozb7TZjx441hw4dCmpz6tQp89BDD5kePXoYj8djHn30UXP69OmgNh9++KEZPXq0cbvd5mtf+5pZtGhR2I+ts2psbDSPP/64yczMNPHx8eamm24yTz/9dNAtxdT92rzzzjut/g6fOnWqMSay9X399dfNbbfdZuLi4swdd9xh1q1b1+7jiTLmKx/xCAAA0MVxzQ0AALAK4QYAAFiFcAMAAKxCuAEAAFYh3AAAAKsQbgAAgFUINwAAwCqEGwAAYBXCDQAAsArhBgAAWIVwAwAArEK4AQAAVvk/7IZ2CMjzc4EAAAAASUVORK5CYII=\n"
          },
          "metadata": {}
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#Inspecting the shortest abstracts\n",
        "scopus_all.sort_values('abstract_len')[:10]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 745
        },
        "id": "zoEUIdX_2YvY",
        "outputId": "7cd2993f-1389-4dee-8a0a-ff2358c1bbf1"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "                                                                                                                                  title  \\\n",
              "1296                                 Analysis of the Near-Surface Layers of Lithium Coatings Using Laser Induced Breakdown Spectroscopy   \n",
              "1559  Criteria for the Existence of Ferry Vortex Singularities in Supersonic Conic Flows in the Absence of Branch Points of Shock Waves   \n",
              "1692                                                             Optical Materials and Fibers in Experiments on Pulsed Nuclear Reactors   \n",
              "1631                                                                                A-statistical convergence of order α via ϕ-function   \n",
              "186                                           Well-posedness of Third Order Differential Equations in Hölder Continuous Function Spaces   \n",
              "1118                                                                                  A Beaded Scarab in the Victoria and Albert Museum   \n",
              "409                    How many copies are enough revisited: Open access legal scholarship in the time of collection budget constraints   \n",
              "1209                                                On the existence results for (p, q)-Kirchhoff type systems with multiple parameters   \n",
              "1777                                                                       Synthesis of Symmetrical N-(Het)aryl-C-phosphonoacetamidines   \n",
              "1877                                                                                                                   Linear Conjugacy   \n",
              "\n",
              "      Year                                       journal  citations  \\\n",
              "1296  2019                      Physics of Atomic Nuclei          0   \n",
              "1559  2019                               Doklady Physics          0   \n",
              "1692  2019                      Physics of Atomic Nuclei          0   \n",
              "1631  2019  Applicable Analysis and Discrete Mathematics          0   \n",
              "186   2019                Canadian Mathematical Bulletin          0   \n",
              "1118  2019               Journal of Egyptian Archaeology          0   \n",
              "409   2019                           Law Library Journal          0   \n",
              "1209  2019  Jordan Journal of Mathematics and Statistics          0   \n",
              "1777  2019          Russian Journal of General Chemistry          0   \n",
              "1877  2019                Canadian Mathematical Bulletin          0   \n",
              "\n",
              "                                                                                                                                                   Link  \\\n",
              "1296  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082404364&doi=10.1134%2fS106377881910020X&partnerID=40&md5=01e4e619a706d60dc30ffae02f7e0965   \n",
              "1559  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081020838&doi=10.1134%2fS1028335819120127&partnerID=40&md5=cc505c709d76f0ab66a619c4d1270123   \n",
              "1692  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080125357&doi=10.1134%2fS1063778819080064&partnerID=40&md5=03f49f091e352c22b31c0a102b968e27   \n",
              "1631     https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080872936&doi=10.2298%2fAADM181218043S&partnerID=40&md5=305e3ca8f76ce381c32a8f8f68a1e067   \n",
              "186   https://www.scopus.com/inward/record.uri?eid=2-s2.0-85106816570&doi=10.4153%2fS0008439518000048&partnerID=40&md5=490e625f20dc36debead49c435561a83   \n",
              "1118   https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083804921&doi=10.1177%2f0307513319899955&partnerID=40&md5=57baa83b68a0847e8e21b882aa98db09   \n",
              "409                                   https://www.scopus.com/inward/record.uri?eid=2-s2.0-85091142940&partnerID=40&md5=0cdd0526c5c5be4e4997f2bb4afc7712   \n",
              "1209                                  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082311348&partnerID=40&md5=5155d3fbd3887515004b0737330bbecc   \n",
              "1777  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079604269&doi=10.1134%2fS1070363219120119&partnerID=40&md5=b532ddf91be8bc6e14302294c8b96f47   \n",
              "1877  https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079487636&doi=10.4153%2fS0008439519000031&partnerID=40&md5=5a522f510b06e60791329bb849a0d28f   \n",
              "\n",
              "     abstract  abstract_len  \n",
              "1296                      0  \n",
              "1559                      0  \n",
              "1692                      0  \n",
              "1631                      0  \n",
              "186                       0  \n",
              "1118                      0  \n",
              "409                       0  \n",
              "1209                      0  \n",
              "1777                      0  \n",
              "1877                      0  "
            ],
            "text/html": [
              "\n",
              "  <div id=\"df-32fa18cc-0546-4776-855a-1aa2f76812bd\" class=\"colab-df-container\">\n",
              "    <div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>title</th>\n",
              "      <th>Year</th>\n",
              "      <th>journal</th>\n",
              "      <th>citations</th>\n",
              "      <th>Link</th>\n",
              "      <th>abstract</th>\n",
              "      <th>abstract_len</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>1296</th>\n",
              "      <td>Analysis of the Near-Surface Layers of Lithium Coatings Using Laser Induced Breakdown Spectroscopy</td>\n",
              "      <td>2019</td>\n",
              "      <td>Physics of Atomic Nuclei</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082404364&amp;doi=10.1134%2fS106377881910020X&amp;partnerID=40&amp;md5=01e4e619a706d60dc30ffae02f7e0965</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1559</th>\n",
              "      <td>Criteria for the Existence of Ferry Vortex Singularities in Supersonic Conic Flows in the Absence of Branch Points of Shock Waves</td>\n",
              "      <td>2019</td>\n",
              "      <td>Doklady Physics</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85081020838&amp;doi=10.1134%2fS1028335819120127&amp;partnerID=40&amp;md5=cc505c709d76f0ab66a619c4d1270123</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1692</th>\n",
              "      <td>Optical Materials and Fibers in Experiments on Pulsed Nuclear Reactors</td>\n",
              "      <td>2019</td>\n",
              "      <td>Physics of Atomic Nuclei</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080125357&amp;doi=10.1134%2fS1063778819080064&amp;partnerID=40&amp;md5=03f49f091e352c22b31c0a102b968e27</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1631</th>\n",
              "      <td>A-statistical convergence of order α via ϕ-function</td>\n",
              "      <td>2019</td>\n",
              "      <td>Applicable Analysis and Discrete Mathematics</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85080872936&amp;doi=10.2298%2fAADM181218043S&amp;partnerID=40&amp;md5=305e3ca8f76ce381c32a8f8f68a1e067</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>186</th>\n",
              "      <td>Well-posedness of Third Order Differential Equations in Hölder Continuous Function Spaces</td>\n",
              "      <td>2019</td>\n",
              "      <td>Canadian Mathematical Bulletin</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85106816570&amp;doi=10.4153%2fS0008439518000048&amp;partnerID=40&amp;md5=490e625f20dc36debead49c435561a83</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1118</th>\n",
              "      <td>A Beaded Scarab in the Victoria and Albert Museum</td>\n",
              "      <td>2019</td>\n",
              "      <td>Journal of Egyptian Archaeology</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85083804921&amp;doi=10.1177%2f0307513319899955&amp;partnerID=40&amp;md5=57baa83b68a0847e8e21b882aa98db09</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>409</th>\n",
              "      <td>How many copies are enough revisited: Open access legal scholarship in the time of collection budget constraints</td>\n",
              "      <td>2019</td>\n",
              "      <td>Law Library Journal</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85091142940&amp;partnerID=40&amp;md5=0cdd0526c5c5be4e4997f2bb4afc7712</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1209</th>\n",
              "      <td>On the existence results for (p, q)-Kirchhoff type systems with multiple parameters</td>\n",
              "      <td>2019</td>\n",
              "      <td>Jordan Journal of Mathematics and Statistics</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85082311348&amp;partnerID=40&amp;md5=5155d3fbd3887515004b0737330bbecc</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1777</th>\n",
              "      <td>Synthesis of Symmetrical N-(Het)aryl-C-phosphonoacetamidines</td>\n",
              "      <td>2019</td>\n",
              "      <td>Russian Journal of General Chemistry</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079604269&amp;doi=10.1134%2fS1070363219120119&amp;partnerID=40&amp;md5=b532ddf91be8bc6e14302294c8b96f47</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1877</th>\n",
              "      <td>Linear Conjugacy</td>\n",
              "      <td>2019</td>\n",
              "      <td>Canadian Mathematical Bulletin</td>\n",
              "      <td>0</td>\n",
              "      <td>https://www.scopus.com/inward/record.uri?eid=2-s2.0-85079487636&amp;doi=10.4153%2fS0008439519000031&amp;partnerID=40&amp;md5=5a522f510b06e60791329bb849a0d28f</td>\n",
              "      <td></td>\n",
              "      <td>0</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>\n",
              "    <div class=\"colab-df-buttons\">\n",
              "\n",
              "  <div class=\"colab-df-container\">\n",
              "    <button class=\"colab-df-convert\" onclick=\"convertToInteractive('df-32fa18cc-0546-4776-855a-1aa2f76812bd')\"\n",
              "            title=\"Convert this dataframe to an interactive table.\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "  <svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\" viewBox=\"0 -960 960 960\">\n",
              "    <path d=\"M120-120v-720h720v720H120Zm60-500h600v-160H180v160Zm220 220h160v-160H400v160Zm0 220h160v-160H400v160ZM180-400h160v-160H180v160Zm440 0h160v-160H620v160ZM180-180h160v-160H180v160Zm440 0h160v-160H620v160Z\"/>\n",
              "  </svg>\n",
              "    </button>\n",
              "\n",
              "  <style>\n",
              "    .colab-df-container {\n",
              "      display:flex;\n",
              "      gap: 12px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert {\n",
              "      background-color: #E8F0FE;\n",
              "      border: none;\n",
              "      border-radius: 50%;\n",
              "      cursor: pointer;\n",
              "      display: none;\n",
              "      fill: #1967D2;\n",
              "      height: 32px;\n",
              "      padding: 0 0 0 0;\n",
              "      width: 32px;\n",
              "    }\n",
              "\n",
              "    .colab-df-convert:hover {\n",
              "      background-color: #E2EBFA;\n",
              "      box-shadow: 0px 1px 2px rgba(60, 64, 67, 0.3), 0px 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "      fill: #174EA6;\n",
              "    }\n",
              "\n",
              "    .colab-df-buttons div {\n",
              "      margin-bottom: 4px;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert {\n",
              "      background-color: #3B4455;\n",
              "      fill: #D2E3FC;\n",
              "    }\n",
              "\n",
              "    [theme=dark] .colab-df-convert:hover {\n",
              "      background-color: #434B5C;\n",
              "      box-shadow: 0px 1px 3px 1px rgba(0, 0, 0, 0.15);\n",
              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
              "      fill: #FFFFFF;\n",
              "    }\n",
              "  </style>\n",
              "\n",
              "    <script>\n",
              "      const buttonEl =\n",
              "        document.querySelector('#df-32fa18cc-0546-4776-855a-1aa2f76812bd button.colab-df-convert');\n",
              "      buttonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "\n",
              "      async function convertToInteractive(key) {\n",
              "        const element = document.querySelector('#df-32fa18cc-0546-4776-855a-1aa2f76812bd');\n",
              "        const dataTable =\n",
              "          await google.colab.kernel.invokeFunction('convertToInteractive',\n",
              "                                                    [key], {});\n",
              "        if (!dataTable) return;\n",
              "\n",
              "        const docLinkHtml = 'Like what you see? Visit the ' +\n",
              "          '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
              "          + ' to learn more about interactive tables.';\n",
              "        element.innerHTML = '';\n",
              "        dataTable['output_type'] = 'display_data';\n",
              "        await google.colab.output.renderOutput(dataTable, element);\n",
              "        const docLink = document.createElement('div');\n",
              "        docLink.innerHTML = docLinkHtml;\n",
              "        element.appendChild(docLink);\n",
              "      }\n",
              "    </script>\n",
              "  </div>\n",
              "\n",
              "\n",
              "<div id=\"df-1d2f8659-8ce1-4c86-902e-a7679895ff1b\">\n",
              "  <button class=\"colab-df-quickchart\" onclick=\"quickchart('df-1d2f8659-8ce1-4c86-902e-a7679895ff1b')\"\n",
              "            title=\"Suggest charts\"\n",
              "            style=\"display:none;\">\n",
              "\n",
              "<svg xmlns=\"http://www.w3.org/2000/svg\" height=\"24px\"viewBox=\"0 0 24 24\"\n",
              "     width=\"24px\">\n",
              "    <g>\n",
              "        <path d=\"M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z\"/>\n",
              "    </g>\n",
              "</svg>\n",
              "  </button>\n",
              "\n",
              "<style>\n",
              "  .colab-df-quickchart {\n",
              "      --bg-color: #E8F0FE;\n",
              "      --fill-color: #1967D2;\n",
              "      --hover-bg-color: #E2EBFA;\n",
              "      --hover-fill-color: #174EA6;\n",
              "      --disabled-fill-color: #AAA;\n",
              "      --disabled-bg-color: #DDD;\n",
              "  }\n",
              "\n",
              "  [theme=dark] .colab-df-quickchart {\n",
              "      --bg-color: #3B4455;\n",
              "      --fill-color: #D2E3FC;\n",
              "      --hover-bg-color: #434B5C;\n",
              "      --hover-fill-color: #FFFFFF;\n",
              "      --disabled-bg-color: #3B4455;\n",
              "      --disabled-fill-color: #666;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart {\n",
              "    background-color: var(--bg-color);\n",
              "    border: none;\n",
              "    border-radius: 50%;\n",
              "    cursor: pointer;\n",
              "    display: none;\n",
              "    fill: var(--fill-color);\n",
              "    height: 32px;\n",
              "    padding: 0;\n",
              "    width: 32px;\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart:hover {\n",
              "    background-color: var(--hover-bg-color);\n",
              "    box-shadow: 0 1px 2px rgba(60, 64, 67, 0.3), 0 1px 3px 1px rgba(60, 64, 67, 0.15);\n",
              "    fill: var(--button-hover-fill-color);\n",
              "  }\n",
              "\n",
              "  .colab-df-quickchart-complete:disabled,\n",
              "  .colab-df-quickchart-complete:disabled:hover {\n",
              "    background-color: var(--disabled-bg-color);\n",
              "    fill: var(--disabled-fill-color);\n",
              "    box-shadow: none;\n",
              "  }\n",
              "\n",
              "  .colab-df-spinner {\n",
              "    border: 2px solid var(--fill-color);\n",
              "    border-color: transparent;\n",
              "    border-bottom-color: var(--fill-color);\n",
              "    animation:\n",
              "      spin 1s steps(1) infinite;\n",
              "  }\n",
              "\n",
              "  @keyframes spin {\n",
              "    0% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "      border-left-color: var(--fill-color);\n",
              "    }\n",
              "    20% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    30% {\n",
              "      border-color: transparent;\n",
              "      border-left-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    40% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-top-color: var(--fill-color);\n",
              "    }\n",
              "    60% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "    }\n",
              "    80% {\n",
              "      border-color: transparent;\n",
              "      border-right-color: var(--fill-color);\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "    90% {\n",
              "      border-color: transparent;\n",
              "      border-bottom-color: var(--fill-color);\n",
              "    }\n",
              "  }\n",
              "</style>\n",
              "\n",
              "  <script>\n",
              "    async function quickchart(key) {\n",
              "      const quickchartButtonEl =\n",
              "        document.querySelector('#' + key + ' button');\n",
              "      quickchartButtonEl.disabled = true;  // To prevent multiple clicks.\n",
              "      quickchartButtonEl.classList.add('colab-df-spinner');\n",
              "      try {\n",
              "        const charts = await google.colab.kernel.invokeFunction(\n",
              "            'suggestCharts', [key], {});\n",
              "      } catch (error) {\n",
              "        console.error('Error during call to suggestCharts:', error);\n",
              "      }\n",
              "      quickchartButtonEl.classList.remove('colab-df-spinner');\n",
              "      quickchartButtonEl.classList.add('colab-df-quickchart-complete');\n",
              "    }\n",
              "    (() => {\n",
              "      let quickchartButtonEl =\n",
              "        document.querySelector('#df-1d2f8659-8ce1-4c86-902e-a7679895ff1b button');\n",
              "      quickchartButtonEl.style.display =\n",
              "        google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
              "    })();\n",
              "  </script>\n",
              "</div>\n",
              "    </div>\n",
              "  </div>\n"
            ],
            "application/vnd.google.colaboratory.intrinsic+json": {
              "type": "dataframe",
              "repr_error": "0"
            }
          },
          "metadata": {},
          "execution_count": 158
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#Removing the row without an abstract\n",
        "scopus_all = scopus_all[scopus_all['abstract_len'] > 100]"
      ],
      "metadata": {
        "id": "jHIOr7qf2nWr"
      },
      "execution_count": null,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.iloc[26]"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Wnb03Gj-29HM",
        "outputId": "ea736015-e727-460f-f610-33f3ddf3b08b"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "title                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          Minimap2: Pairwise alignment for nucleotide sequences\n",
              "Year                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            2018\n",
              "journal                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               Bioinformatics\n",
              "citations                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       4673\n",
              "Link                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         https://www.scopus.com/inward/record.uri?eid=2-s2.0-85054132627&doi=10.1093%2fbioinformatics%2fbty191&partnerID=40&md5=703f66baee8e33ab5f70a716aa2498c7\n",
              "abstract        Motivation: Recent advances in sequencing technologies promise ultra-long reads of ~100 kb in average, full-length mRNA or cDNA reads in high throughput and genomic contigs over 100 Mb in length. Existing alignment programs are unable or inefficient to process such data at scale, which presses for the development of new alignment algorithms. Results: Minimap2 is a general-purpose alignment program to map DNA or long mRNA sequences against a large reference database. It works with accurate short reads of ≥100 bp in length, ≥1 kb genomic reads at error rate ~15%, full-length noisy Direct RNA or cDNA reads and assembly contigs or closely related full chromosomes of hundreds of megabases in length. Minimap2 does split-read alignment, employs concave gap cost for long insertions and deletions and introduces new heuristics to reduce spurious alignments. It is 3–4 times as fast as mainstream short-read mappers at comparable accuracy, and is ≥30 times faster than long-read genomic or cDNA mappers at higher accuracy, surpassing most aligners specialized in one type of alignment. © The Author(s) 2018.\n",
              "abstract_len                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    1108\n",
              "Name: 30, dtype: object"
            ]
          },
          "metadata": {},
          "execution_count": 160
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.shape"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hSszRMsg3HfU",
        "outputId": "b1fe368b-aec0-411f-ffb2-22856c5cf2a0"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(3523, 7)"
            ]
          },
          "metadata": {},
          "execution_count": 161
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "scopus_all.to_csv('scopus all 16to19 final.csv', index=False)"
      ],
      "metadata": {
        "id": "gFureZ7tosru"
      },
      "execution_count": null,
      "outputs": []
    }
  ]
}